{
 "cells": [
  {
   "cell_type": "markdown",
   "source": [
    "# Preprocessing Code and Guidelines for HOGVAX\n",
    "\n",
    "This jupyter notebook file provides code and guidelines for parts of the preprocessing to execute HOGVAX with your own data. Here, we use some data of the SARS-CoV-2 study provided by Liu et al., 2020, *Cell Systems 11, 131–144* as an example. Feel free to use and modify the provided code for your own purposes. If you like to run HOGVAX for the case study by Liu et al., download the data using the provided [download_data.sh](../download_data.sh) file."
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "6dda54709248cc69"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Creating epitope candidates and removing cleavage regions\n",
    "\n",
    "If a protein sequence contains cleavage sites, this fasta entry must be split into two separate entries in a fasta. Thereby, we guarantee that we are not creating and using peptides that cover a cleavage site and would therefore not occur naturally. Starting from the protein fastas of the SARS-CoV-2 proteins, we cut them into peptides using sliding windows of length 8 to 10 for MHC class I or 13 to 25 for MHC class II. We store the peptides in a text file and additionally create a csv file for further information about the peptides.\n"
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "112e24340f92573"
  },
  {
   "cell_type": "code",
   "execution_count": 319,
   "outputs": [],
   "source": [
    "import copy\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from Bio import SeqIO\n",
    "from collections import defaultdict"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.242007Z",
     "start_time": "2023-10-11T14:14:55.807170Z"
    }
   },
   "id": "2132084384b4f750"
  },
  {
   "cell_type": "code",
   "execution_count": 320,
   "outputs": [
    {
     "data": {
      "text/plain": "28"
     },
     "execution_count": 320,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# create a simple csv file with information of cleavage positions, if necessary\n",
    "# format: Start,End,Sub_Protein (new name for new fasta file),Protein (record.id as in fasta file)\n",
    "csv_file = 'test/ProteinCleavageSites.csv'\n",
    "protein_file = 'test/test.fasta'\n",
    "cleaved_file = 'test/cleaved_test.fasta'\n",
    "\n",
    "df_cleavage = pd.read_csv(csv_file)\n",
    "\n",
    "new_records = []\n",
    "with open(protein_file, 'r') as file:\n",
    "    for record in SeqIO.parse(file, 'fasta'):\n",
    "        if (df_cleavage['Protein'] == record.id).any():\n",
    "            for i, entry in df_cleavage[df_cleavage['Protein'] == record.id].iterrows():\n",
    "                new_record = copy.deepcopy(record)\n",
    "                new_record.id = entry.Sub_Protein\n",
    "                new_record.seq = record.seq[entry.Start:entry.End+1]\n",
    "                new_records.append(new_record)\n",
    "        else:\n",
    "            new_records.append(record)\n",
    "\n",
    "SeqIO.write(new_records, cleaved_file, 'fasta')"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.268522Z",
     "start_time": "2023-10-11T14:14:55.815569Z"
    }
   },
   "id": "7f2ead807230fda0"
  },
  {
   "cell_type": "code",
   "execution_count": 321,
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.269066Z",
     "start_time": "2023-10-11T14:14:55.848155Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": "                         Peptides Protein  Index  Length\n0                        MYSFVSEE       E      0       8\n1                        YSFVSEET       E      1       8\n2                        SFVSEETG       E      2       8\n3                        FVSEETGT       E      3       8\n4                        VSEETGTL       E      4       8\n...                           ...     ...    ...     ...\n153106  FQLTPIAVQMTKLATTEELPDEFVV   ORF9b     68      25\n153107  QLTPIAVQMTKLATTEELPDEFVVV   ORF9b     69      25\n153108  LTPIAVQMTKLATTEELPDEFVVVT   ORF9b     70      25\n153109  TPIAVQMTKLATTEELPDEFVVVTV   ORF9b     71      25\n153110  PIAVQMTKLATTEELPDEFVVVTVK   ORF9b     72      25\n\n[153111 rows x 4 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Peptides</th>\n      <th>Protein</th>\n      <th>Index</th>\n      <th>Length</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>MYSFVSEE</td>\n      <td>E</td>\n      <td>0</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>YSFVSEET</td>\n      <td>E</td>\n      <td>1</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>SFVSEETG</td>\n      <td>E</td>\n      <td>2</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>FVSEETGT</td>\n      <td>E</td>\n      <td>3</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>4</th>\n      <td>VSEETGTL</td>\n      <td>E</td>\n      <td>4</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>153106</th>\n      <td>FQLTPIAVQMTKLATTEELPDEFVV</td>\n      <td>ORF9b</td>\n      <td>68</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>153107</th>\n      <td>QLTPIAVQMTKLATTEELPDEFVVV</td>\n      <td>ORF9b</td>\n      <td>69</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>153108</th>\n      <td>LTPIAVQMTKLATTEELPDEFVVVT</td>\n      <td>ORF9b</td>\n      <td>70</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>153109</th>\n      <td>TPIAVQMTKLATTEELPDEFVVVTV</td>\n      <td>ORF9b</td>\n      <td>71</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>153110</th>\n      <td>PIAVQMTKLATTEELPDEFVVVTVK</td>\n      <td>ORF9b</td>\n      <td>72</td>\n      <td>25</td>\n    </tr>\n  </tbody>\n</table>\n<p>153111 rows × 4 columns</p>\n</div>"
     },
     "execution_count": 321,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# specify input and output file, if you split the input sequences at cleavage sites, make sure to use this modified fasta file\n",
    "input_fasta = 'test/cleaved_test.fasta'\n",
    "output_peptides = 'test/peptides.pep'\n",
    "output_csv = 'test/epitope_features.csv'\n",
    "\n",
    "# give range of peptide length as min and max value\n",
    "peptide_range = [8, 25]\n",
    "\n",
    "peptides = []\n",
    "proteins = []\n",
    "lengths = []\n",
    "indices = []\n",
    "with open(input_fasta, 'r') as file:\n",
    "    for record in SeqIO.parse(file, 'fasta'):\n",
    "        protein_seq = str(record.seq).strip('*')\n",
    "        protein = record.id\n",
    "        for r in range(peptide_range[0], peptide_range[1]+1):\n",
    "            curr_peptides = [protein_seq[i:i+r] for i in range(len(protein_seq)-r+1)]\n",
    "            proteins += [protein] * len(curr_peptides)\n",
    "            lengths += [r] * len(curr_peptides)\n",
    "            indices += [i for i in range(len(protein_seq)-r+1)]\n",
    "            peptides += curr_peptides\n",
    "\n",
    "with open(output_peptides, 'w') as file:\n",
    "    file.write('\\n'.join(peptides))\n",
    "\n",
    "df_all_features = pd.DataFrame(list(zip(peptides, proteins, indices, lengths)), columns=['Peptides', 'Protein', 'Index', 'Length'])\n",
    "df_all_features"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 322,
   "outputs": [],
   "source": [
    "df_all_features.to_csv(output_csv)"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.269355Z",
     "start_time": "2023-10-11T14:14:55.940026Z"
    }
   },
   "id": "d53fe9a625f0c08f"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Exclude self-peptides\n",
    "\n",
    "To exclude self-peptides from the input peptides, you must identify the self-peptides and provide them in a text file with one peptide per row."
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "d9423c2c0cf3409b"
  },
  {
   "cell_type": "code",
   "execution_count": 323,
   "outputs": [
    {
     "data": {
      "text/plain": "                          Protein  Index  Length\nPeptides                                        \nMYSFVSEE                        E      0       8\nYSFVSEET                        E      1       8\nSFVSEETG                        E      2       8\nFVSEETGT                        E      3       8\nVSEETGTL                        E      4       8\n...                           ...    ...     ...\nFQLTPIAVQMTKLATTEELPDEFVV   ORF9b     68      25\nQLTPIAVQMTKLATTEELPDEFVVV   ORF9b     69      25\nLTPIAVQMTKLATTEELPDEFVVVT   ORF9b     70      25\nTPIAVQMTKLATTEELPDEFVVVTV   ORF9b     71      25\nPIAVQMTKLATTEELPDEFVVVTVK   ORF9b     72      25\n\n[153066 rows x 3 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Protein</th>\n      <th>Index</th>\n      <th>Length</th>\n    </tr>\n    <tr>\n      <th>Peptides</th>\n      <th></th>\n      <th></th>\n      <th></th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>MYSFVSEE</th>\n      <td>E</td>\n      <td>0</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>YSFVSEET</th>\n      <td>E</td>\n      <td>1</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>SFVSEETG</th>\n      <td>E</td>\n      <td>2</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>FVSEETGT</th>\n      <td>E</td>\n      <td>3</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>VSEETGTL</th>\n      <td>E</td>\n      <td>4</td>\n      <td>8</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>FQLTPIAVQMTKLATTEELPDEFVV</th>\n      <td>ORF9b</td>\n      <td>68</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>QLTPIAVQMTKLATTEELPDEFVVV</th>\n      <td>ORF9b</td>\n      <td>69</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>LTPIAVQMTKLATTEELPDEFVVVT</th>\n      <td>ORF9b</td>\n      <td>70</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>TPIAVQMTKLATTEELPDEFVVVTV</th>\n      <td>ORF9b</td>\n      <td>71</td>\n      <td>25</td>\n    </tr>\n    <tr>\n      <th>PIAVQMTKLATTEELPDEFVVVTVK</th>\n      <td>ORF9b</td>\n      <td>72</td>\n      <td>25</td>\n    </tr>\n  </tbody>\n</table>\n<p>153066 rows × 3 columns</p>\n</div>"
     },
     "execution_count": 323,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "self_peptides = 'test/self_pept.pep'\n",
    "with open(self_peptides, 'r') as file:\n",
    "    for line in file:\n",
    "        self_pep = line.strip('\\n')\n",
    "        df_all_features = df_all_features.drop(df_all_features[df_all_features['Peptides'] == self_pep].index)\n",
    "df_all_features.set_index('Peptides')"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.641925Z",
     "start_time": "2023-10-11T14:14:56.104281Z"
    }
   },
   "id": "bcbd2ad7a6c992ef"
  },
  {
   "cell_type": "code",
   "execution_count": 324,
   "outputs": [],
   "source": [
    "df_all_features.to_csv(output_csv)"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.799249Z",
     "start_time": "2023-10-11T14:14:56.642725Z"
    }
   },
   "id": "f922e7c6edb7936c"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Identifying mutation probabilities of peptides\n",
    "\n",
    "This is based on the work by Liu et al. that you can find [here](https://github.com/gifford-lab/optivax/tree/master/covid_mutation_analysis_and_nextstrain_build). For the identification of peptides that mutated in different viral strains, you need *a)* the sequences of the different strains, and *b)* alignments of the different strains to a reference sequence. For SARS-CoV-2 this data was already provided by Liu et al. To get the alignments, they used the [nextstrain pipeline](https://docs.nextstrain.org/projects/ncov/en/latest/index.html). For further information on how to compute the alignments, follow the tutorial on the nextstrain website and check out the [README.md](https://github.com/gifford-lab/optivax/blob/master/covid_mutation_analysis_and_nextstrain_build/README.md) by Liu et al. \n",
    "\n",
    "The following code is based on the code provided by Liu et al. and adds the mutation probability of each of the previously computed peptides to the csv file."
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "18588df25e34f226"
  },
  {
   "cell_type": "code",
   "execution_count": 325,
   "outputs": [
    {
     "data": {
      "text/plain": "['test/aligned_protein_E.fasta']"
     },
     "execution_count": 325,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "alignment_files = !ls test/aligned_*.fasta\n",
    "alignment_files"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:14:56.830891Z",
     "start_time": "2023-10-11T14:14:56.800949Z"
    }
   },
   "id": "d3cefb64a580bde6"
  },
  {
   "cell_type": "code",
   "execution_count": 326,
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting pooling on 1 cores\n",
      "computeWindows!!!\n",
      "aa_file =  test/aligned_protein_E.fasta\n",
      "protein: E\n",
      "size of records 12789\n",
      "removing the stop codon at the end E\n",
      "window size 8\n",
      "window size 9\n",
      "window size 10\n",
      "window size 11\n",
      "window size 12\n",
      "window size 13\n",
      "window size 14\n",
      "window size 15\n",
      "window size 16\n",
      "window size 17\n",
      "window size 18\n",
      "window size 19\n",
      "window size 20\n",
      "window size 21\n",
      "window size 22\n",
      "window size 23\n",
      "window size 24\n",
      "window size 25\n"
     ]
    }
   ],
   "source": [
    "import time \n",
    "\n",
    "def computeWindows(aa_file):\n",
    "    print('computeWindows!!!')\n",
    "    print('aa_file = ', aa_file) ## Alex\n",
    "    \n",
    "    def entropy_calc(x):\n",
    "        # where x is a list of numbers. \n",
    "        summ = 0\n",
    "        nc = np.sum(x)\n",
    "        for e in x: \n",
    "            p = e/nc\n",
    "            summ += p*np.log2(p)\n",
    "        return -summ\n",
    "    \n",
    "    window_sizes = list(range(8,26)) \n",
    "    protein_res = []\n",
    "    protein = aa_file.split('_')[-1].split('.')[0]\n",
    "    print('protein:', protein)\n",
    "    with open(aa_file, \"rt\") as handle:\n",
    "        records = list(SeqIO.parse(handle, \"fasta\"))\n",
    "    \n",
    "    \n",
    "    # getting rid of the node sequences!\n",
    "    no_nodes = []\n",
    "    for r in records: \n",
    "        if 'NODE_' not in r.id:\n",
    "            no_nodes.append(r)\n",
    "    records = no_nodes\n",
    "    print('size of records', len(records))\n",
    "    # getting the reference sequence\n",
    "    for ind, r in enumerate(records): \n",
    "        if r.id == 'Wuhan-Hu-1/2019':#'Wuhan/WH01/2019':#'Wuhan/IPBCAMS-WH-01/2019':\n",
    "            ref_seq_ind = ind\n",
    "            \n",
    "    ref_seq = str(records[ref_seq_ind].seq)\n",
    "    \n",
    "    seqs = np.array(records)\n",
    "    \n",
    "    if seqs[ref_seq_ind, -1]=='*': # ignore the stop code at the end. \n",
    "        print('removing the stop codon at the end', protein)\n",
    "        ref_seq = ref_seq[:-1]\n",
    "        seqs = seqs[:, :-1]\n",
    "    \n",
    "    for window in window_sizes:\n",
    "        print('window size', window)\n",
    "        \n",
    "        window_res = []\n",
    "        \n",
    "        # get epitope based column slices. gives a list of epi columns. \n",
    "        epi_columns = [seqs[:,i:i+window] for i in range(seqs.shape[1]-window+1)]\n",
    "    \n",
    "        for col_ind, col in enumerate(epi_columns):\n",
    "            #first need to convert each of the columns into strings:\n",
    "            col = np.asarray([''.join(col[i,:]) for i in range(col.shape[0])])\n",
    "            unique, counts = np.unique(col, return_counts=True) \n",
    "            ent = entropy_calc(counts)\n",
    "            \n",
    "            # useful for percentages\n",
    "            ref_epitope = col[ref_seq_ind]\n",
    "            count_dict = dict(zip(unique, counts))\n",
    "            # -1 for no self count. percentage mutated. \n",
    "            perc = 1 - ((count_dict[ref_epitope]-1)/(np.sum(counts)-1))\n",
    "            \n",
    "            # start pos, window size, entropy\n",
    "            window_res.append([protein, ref_epitope, col_ind, window, ent, perc])\n",
    "            \n",
    "        protein_res += window_res\n",
    "        \n",
    "    df = pd.DataFrame(protein_res)\n",
    "    return df\n",
    "    \n",
    "    \n",
    "ncores = 1\n",
    "\n",
    "start_time = time.time()\n",
    "\n",
    "### uncomment this to run it, commented for safety reasons -- Alex\n",
    "# multicore generate new samples\n",
    "print('Starting pooling on %d cores' % ncores)\n",
    "df_list = list()\n",
    "for file in alignment_files:\n",
    "    df_list.append(computeWindows(file))"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:28.866980Z",
     "start_time": "2023-10-11T14:14:56.831162Z"
    }
   },
   "id": "860a56bad760ba53"
  },
  {
   "cell_type": "code",
   "execution_count": 327,
   "outputs": [
    {
     "data": {
      "text/plain": "(1070, 6)"
     },
     "execution_count": 327,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.concat(df_list)\n",
    "df.columns = ['protein', 'epitope', 'start_pos', 'epi_len', 'entropy', 'perc_mutated']\n",
    "\n",
    "# only keep epitopes in range 8-10 and 13-25 that do not cover a cleavage region and are no self-peptides\n",
    "df = df[df.epitope.isin(df_all_features.Peptides)]\n",
    "df.to_csv('test/epitope_features.csv')\n",
    "df.shape"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:28.927637Z",
     "start_time": "2023-10-11T14:15:28.884710Z"
    }
   },
   "id": "70a81d56efae015f"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Add glycosylation probabilities\n",
    "\n",
    "For glycosylation probability predictions, Liu et al. used the NetNGlyc N-glycosylation prediction server [Gupta et al., 2004](http://www.cbs.dtu.dk/services/NetNGlyc/) and validated the predictions for the spike protein using data from other publications. For details, check out [Liu et al., 2020](https://doi.org/10.1016/j.cels.2020.06.009). The following code is again taken from Liu et al. and modified for the purpose to give you an idea on how to design the epitope feature table, which is useful to filter the peptides."
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "31670cb207709b82"
  },
  {
   "cell_type": "code",
   "execution_count": 328,
   "outputs": [
    {
     "data": {
      "text/plain": "                      protein  position   seq  prob_of_glyco\n0  Wuhan_IPBCAMS-WH-01_2019_E        48  NVSL         0.6507\n1  Wuhan_IPBCAMS-WH-01_2019_E        66  NSSR         0.6339\n2  Wuhan_IPBCAMS-WH-01_2019_M         5  NGTI         0.7577\n3  Wuhan_IPBCAMS-WH-01_2019_N        47  NNTA         0.6798\n4  Wuhan_IPBCAMS-WH-01_2019_N        77  NSSP         0.2149",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>protein</th>\n      <th>position</th>\n      <th>seq</th>\n      <th>prob_of_glyco</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>Wuhan_IPBCAMS-WH-01_2019_E</td>\n      <td>48</td>\n      <td>NVSL</td>\n      <td>0.6507</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>Wuhan_IPBCAMS-WH-01_2019_E</td>\n      <td>66</td>\n      <td>NSSR</td>\n      <td>0.6339</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>Wuhan_IPBCAMS-WH-01_2019_M</td>\n      <td>5</td>\n      <td>NGTI</td>\n      <td>0.7577</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>Wuhan_IPBCAMS-WH-01_2019_N</td>\n      <td>47</td>\n      <td>NNTA</td>\n      <td>0.6798</td>\n    </tr>\n    <tr>\n      <th>4</th>\n      <td>Wuhan_IPBCAMS-WH-01_2019_N</td>\n      <td>77</td>\n      <td>NSSP</td>\n      <td>0.2149</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
     },
     "execution_count": 328,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_glyco = pd.read_csv('test/glycosolation_predictions.txt', sep=' ', header=None)\n",
    "df_glyco.columns =['protein', 'position', 'seq', 'prob_of_glyco']\n",
    "df_glyco.head()"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:28.964033Z",
     "start_time": "2023-10-11T14:15:28.893520Z"
    }
   },
   "id": "d3d5385921a231cd"
  },
  {
   "cell_type": "code",
   "execution_count": 329,
   "outputs": [
    {
     "data": {
      "text/plain": "   protein  position   seq  prob_of_glyco\n0        E        48  NVSL         0.6507\n1        E        66  NSSR         0.6339\n2        M         5  NGTI         0.7577\n3        N        47  NNTA         0.6798\n4        N        77  NSSP         0.2149\n..     ...       ...   ...            ...\n72       S      1098  NGTH         0.5496\n73       S      1134  NNTV         0.5800\n74       S      1158  NHTS         0.3730\n75       S      1173  NASV         0.3998\n76       S      1194  NESL         0.6791\n\n[77 rows x 4 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>protein</th>\n      <th>position</th>\n      <th>seq</th>\n      <th>prob_of_glyco</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>E</td>\n      <td>48</td>\n      <td>NVSL</td>\n      <td>0.6507</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>E</td>\n      <td>66</td>\n      <td>NSSR</td>\n      <td>0.6339</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>M</td>\n      <td>5</td>\n      <td>NGTI</td>\n      <td>0.7577</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>N</td>\n      <td>47</td>\n      <td>NNTA</td>\n      <td>0.6798</td>\n    </tr>\n    <tr>\n      <th>4</th>\n      <td>N</td>\n      <td>77</td>\n      <td>NSSP</td>\n      <td>0.2149</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>72</th>\n      <td>S</td>\n      <td>1098</td>\n      <td>NGTH</td>\n      <td>0.5496</td>\n    </tr>\n    <tr>\n      <th>73</th>\n      <td>S</td>\n      <td>1134</td>\n      <td>NNTV</td>\n      <td>0.5800</td>\n    </tr>\n    <tr>\n      <th>74</th>\n      <td>S</td>\n      <td>1158</td>\n      <td>NHTS</td>\n      <td>0.3730</td>\n    </tr>\n    <tr>\n      <th>75</th>\n      <td>S</td>\n      <td>1173</td>\n      <td>NASV</td>\n      <td>0.3998</td>\n    </tr>\n    <tr>\n      <th>76</th>\n      <td>S</td>\n      <td>1194</td>\n      <td>NESL</td>\n      <td>0.6791</td>\n    </tr>\n  </tbody>\n</table>\n<p>77 rows × 4 columns</p>\n</div>"
     },
     "execution_count": 329,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_glyco['protein'] = df_glyco.protein.apply(lambda x:x.split('_')[-1])\n",
    "df_glyco"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:28.964550Z",
     "start_time": "2023-10-11T14:15:28.908691Z"
    }
   },
   "id": "5bd925a6baa4b63c"
  },
  {
   "cell_type": "code",
   "execution_count": 330,
   "outputs": [
    {
     "data": {
      "text/plain": "(1070, 7)"
     },
     "execution_count": 330,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df['glyco_probs'] = np.empty((len(df), 0)).tolist()\n",
    "for i in range(len(df_glyco)):\n",
    "    protein_mask = df.protein==df_glyco.iloc[i].protein # select relevant protein\n",
    "    seq_start = df['start_pos'] # zero based\n",
    "    seq_end = df['start_pos']+(df['epi_len']-1)\n",
    "    glyco_start = df_glyco.iloc[i].position - 1 # one based, thus -1\n",
    "    glyco_end = df_glyco.iloc[i].position+(len(df_glyco.iloc[i].seq)-1)\n",
    "    in_region = np.logical_and(glyco_start >= seq_start,glyco_start <= seq_end)\n",
    "    #if all 4 matter::: \n",
    "    #.    front_in_region = np.logical_and(glyco_start >= seq_start,glyco_start <= seq_end)\n",
    "    #     end_in_region = np.logical_and(glyco_end >= seq_start,glyco_end <= seq_end)\n",
    "    #.    in_region = np.logical_or(front_in_region, end_in_region)\n",
    "    in_region_and_protein = np.logical_and(protein_mask,in_region)\n",
    "    \n",
    "    df.loc[in_region_and_protein,'glyco_probs'] = df[in_region_and_protein]['glyco_probs'].apply(lambda x: x + [df_glyco.iloc[i].prob_of_glyco])\n",
    "    #apply protein mask and then epitope mask\n",
    "df.shape"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:29.084830Z",
     "start_time": "2023-10-11T14:15:28.916094Z"
    }
   },
   "id": "be65d400f7d18915"
  },
  {
   "cell_type": "code",
   "execution_count": 331,
   "outputs": [],
   "source": [
    "df['glyco_probs'] = df.glyco_probs.apply(lambda x: 1.0 if len(x)>0 else 0.0)"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:29.085003Z",
     "start_time": "2023-10-11T14:15:29.003559Z"
    }
   },
   "id": "eb84db0e93f2065f"
  },
  {
   "cell_type": "code",
   "execution_count": 332,
   "outputs": [
    {
     "data": {
      "text/plain": "(941, 7)"
     },
     "execution_count": 332,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# optionally remove epitopes of specific lengths\n",
    "df = df[np.logical_and(df.epi_len!=11, df.epi_len!=12)]\n",
    "df.shape"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:29.085177Z",
     "start_time": "2023-10-11T14:15:29.011424Z"
    }
   },
   "id": "5505a307deb7651e"
  },
  {
   "cell_type": "code",
   "execution_count": 333,
   "outputs": [
    {
     "data": {
      "text/plain": "                          protein  start_pos  epi_len   entropy  perc_mutated  \\\nepitope                                                                         \nMYSFVSEE                        E          0        8  0.035578      0.002893   \nYSFVSEET                        E          1        8  0.042296      0.003441   \nSFVSEETG                        E          2        8  0.042296      0.003441   \nFVSEETGT                        E          3        8  0.041116      0.003363   \nVSEETGTL                        E          4        8  0.038914      0.003206   \n...                           ...        ...      ...       ...           ...   \nVNVSLVKPSFYVYSRVKNLNSSRVP       E         46       25  0.174796      0.017907   \nNVSLVKPSFYVYSRVKNLNSSRVPD       E         47       25  0.175817      0.017986   \nVSLVKPSFYVYSRVKNLNSSRVPDL       E         48       25  0.188548      0.019080   \nSLVKPSFYVYSRVKNLNSSRVPDLL       E         49       25  0.186565      0.018924   \nLVKPSFYVYSRVKNLNSSRVPDLLV       E         50       25  0.187742      0.019002   \n\n                           glyco_probs  \nepitope                                 \nMYSFVSEE                           0.0  \nYSFVSEET                           0.0  \nSFVSEETG                           0.0  \nFVSEETGT                           0.0  \nVSEETGTL                           0.0  \n...                                ...  \nVNVSLVKPSFYVYSRVKNLNSSRVP          1.0  \nNVSLVKPSFYVYSRVKNLNSSRVPD          1.0  \nVSLVKPSFYVYSRVKNLNSSRVPDL          1.0  \nSLVKPSFYVYSRVKNLNSSRVPDLL          1.0  \nLVKPSFYVYSRVKNLNSSRVPDLLV          1.0  \n\n[941 rows x 6 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>protein</th>\n      <th>start_pos</th>\n      <th>epi_len</th>\n      <th>entropy</th>\n      <th>perc_mutated</th>\n      <th>glyco_probs</th>\n    </tr>\n    <tr>\n      <th>epitope</th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>MYSFVSEE</th>\n      <td>E</td>\n      <td>0</td>\n      <td>8</td>\n      <td>0.035578</td>\n      <td>0.002893</td>\n      <td>0.0</td>\n    </tr>\n    <tr>\n      <th>YSFVSEET</th>\n      <td>E</td>\n      <td>1</td>\n      <td>8</td>\n      <td>0.042296</td>\n      <td>0.003441</td>\n      <td>0.0</td>\n    </tr>\n    <tr>\n      <th>SFVSEETG</th>\n      <td>E</td>\n      <td>2</td>\n      <td>8</td>\n      <td>0.042296</td>\n      <td>0.003441</td>\n      <td>0.0</td>\n    </tr>\n    <tr>\n      <th>FVSEETGT</th>\n      <td>E</td>\n      <td>3</td>\n      <td>8</td>\n      <td>0.041116</td>\n      <td>0.003363</td>\n      <td>0.0</td>\n    </tr>\n    <tr>\n      <th>VSEETGTL</th>\n      <td>E</td>\n      <td>4</td>\n      <td>8</td>\n      <td>0.038914</td>\n      <td>0.003206</td>\n      <td>0.0</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>VNVSLVKPSFYVYSRVKNLNSSRVP</th>\n      <td>E</td>\n      <td>46</td>\n      <td>25</td>\n      <td>0.174796</td>\n      <td>0.017907</td>\n      <td>1.0</td>\n    </tr>\n    <tr>\n      <th>NVSLVKPSFYVYSRVKNLNSSRVPD</th>\n      <td>E</td>\n      <td>47</td>\n      <td>25</td>\n      <td>0.175817</td>\n      <td>0.017986</td>\n      <td>1.0</td>\n    </tr>\n    <tr>\n      <th>VSLVKPSFYVYSRVKNLNSSRVPDL</th>\n      <td>E</td>\n      <td>48</td>\n      <td>25</td>\n      <td>0.188548</td>\n      <td>0.019080</td>\n      <td>1.0</td>\n    </tr>\n    <tr>\n      <th>SLVKPSFYVYSRVKNLNSSRVPDLL</th>\n      <td>E</td>\n      <td>49</td>\n      <td>25</td>\n      <td>0.186565</td>\n      <td>0.018924</td>\n      <td>1.0</td>\n    </tr>\n    <tr>\n      <th>LVKPSFYVYSRVKNLNSSRVPDLLV</th>\n      <td>E</td>\n      <td>50</td>\n      <td>25</td>\n      <td>0.187742</td>\n      <td>0.019002</td>\n      <td>1.0</td>\n    </tr>\n  </tbody>\n</table>\n<p>941 rows × 6 columns</p>\n</div>"
     },
     "execution_count": 333,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = df.set_index('epitope')\n",
    "df"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:29.094707Z",
     "start_time": "2023-10-11T14:15:29.018592Z"
    }
   },
   "id": "6f3c901db8a122a7"
  },
  {
   "cell_type": "code",
   "execution_count": 334,
   "outputs": [],
   "source": [
    "df.to_csv('test/epitope_features.csv')"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T14:15:29.145107Z",
     "start_time": "2023-10-11T14:15:29.021469Z"
    }
   },
   "id": "e926bcc94fccc4fe"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Filter epitopes and create input file\n",
    "\n",
    "Given the epitope feature file, we can use it to filter the peptides, e.g., exclude glycosylated, likely mutated, and self-peptides. Peptides of cleavage reagions were already filtered by splitting up the sequences at the cleavage position and thus do not exist in our input data. "
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "3031a1afcd0422da"
  },
  {
   "cell_type": "code",
   "execution_count": 351,
   "outputs": [
    {
     "data": {
      "text/plain": "['VFLLVTLAILTAL', 'LLVTLAILTALRL', 'LVTLAILTALRLC', 'LLVTLAILTALRLC']"
     },
     "execution_count": 351,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# create list of input peptides that do not contain glycosylated, self-peptides, and cleavage peptides, and no peptides with a mutation probability larger than a given threshold, e.g. 0.001\n",
    "# MHC I length < 11\n",
    "peptides_mhcI = list(df[(df['glyco_probs'] != 1.0) & (df['perc_mutated'] < 0.0001) & (df['epi_len'] < 11)].index)\n",
    "peptides_mhcI\n",
    "\n",
    "# MHC II length range 13-25\n",
    "peptides_mhcII = list(df[(df['glyco_probs'] != 1.0) & (df['perc_mutated'] < 0.001) & (df['epi_len'].isin(range(13,26)))].index)\n",
    "peptides_mhcII"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:01:42.266464Z",
     "start_time": "2023-10-11T15:01:42.250295Z"
    }
   },
   "id": "f3bfbbe013670525"
  },
  {
   "cell_type": "code",
   "execution_count": 352,
   "outputs": [],
   "source": [
    "peptides_file = 'test/filtered_peptides_mhcI.pep'\n",
    "with open(peptides_file, 'w') as pep_file:\n",
    "    pep_file.write('\\n'.join(peptides_mhcI))\n",
    "    \n",
    "peptidesII_file = 'test/filtered_peptides_mhcII.pep'\n",
    "with open(peptidesII_file, 'w') as pep2_file:\n",
    "    pep2_file.write('\\n'.join(peptides_mhcII))"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:01:45.052556Z",
     "start_time": "2023-10-11T15:01:45.036758Z"
    }
   },
   "id": "3335d0354ba4cc2"
  },
  {
   "cell_type": "markdown",
   "source": [
    "### Format netMHCpan predictions for HOGVAX\n",
    "\n",
    "Make sure to have netMHCpan and netMHCIIpan installed. Based on the code by Liu et al. the following will execute netMHCpan and netMHCIIpan for each MHC allele that you provide in a MHC class I and class II file + for the peptide files created above. The binding affinity predictions are then written to the format used by HOGVAX."
   ],
   "metadata": {
    "collapsed": false
   },
   "id": "8dbcb8a1330ac81d"
  },
  {
   "cell_type": "code",
   "execution_count": 353,
   "outputs": [
    {
     "data": {
      "text/plain": "         allele\n0    HLA-B44:04\n1    HLA-B44:05\n2    HLA-B44:07\n3    HLA-A30:10\n4    HLA-B44:02\n..          ...\n225  HLA-B55:02\n226  HLA-B67:01\n227  HLA-A24:10\n228  HLA-B15:32\n229  HLA-B56:10\n\n[230 rows x 1 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>allele</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>HLA-B44:04</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>HLA-B44:05</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>HLA-B44:07</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>HLA-A30:10</td>\n    </tr>\n    <tr>\n      <th>4</th>\n      <td>HLA-B44:02</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>225</th>\n      <td>HLA-B55:02</td>\n    </tr>\n    <tr>\n      <th>226</th>\n      <td>HLA-B67:01</td>\n    </tr>\n    <tr>\n      <th>227</th>\n      <td>HLA-A24:10</td>\n    </tr>\n    <tr>\n      <th>228</th>\n      <td>HLA-B15:32</td>\n    </tr>\n    <tr>\n      <th>229</th>\n      <td>HLA-B56:10</td>\n    </tr>\n  </tbody>\n</table>\n<p>230 rows × 1 columns</p>\n</div>"
     },
     "execution_count": 353,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load final set of HLA alleles.\n",
    "hla_alleles = pd.read_csv('test/MHC1_allele_mary_cleaned.txt', names=['allele'])\n",
    "hla_alleles"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:01:47.822774Z",
     "start_time": "2023-10-11T15:01:47.774331Z"
    }
   },
   "id": "228e1fabeedb9b27"
  },
  {
   "cell_type": "code",
   "execution_count": 354,
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:04 -xls -xlsfile test/HLA-B4404_preds.xls\r\n",
      "# Wed Oct 11 17:01:50 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4404_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4404_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan5Du3pB\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:04 : Distance to training data  0.029 (using nearest neighbor HLA-B44:28)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000120   63.750 0.018050   77.108 41129.51\r\n",
      "   1 HLA-B*44:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000560   38.250 0.026532   53.023 37522.95\r\n",
      "   1 HLA-B*44:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0000980   31.000 0.034475   37.122 34432.86\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:05 -xls -xlsfile test/HLA-B4405_preds.xls\r\n",
      "# Wed Oct 11 17:01:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4405_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4405_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanlNO84a\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:05 : Distance to training data  0.050 (using nearest neighbor HLA-B44:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000110   49.000 0.021878   50.553 39460.80\r\n",
      "   1 HLA-B*44:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000750   24.875 0.028132   37.421 36878.95\r\n",
      "   1 HLA-B*44:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0000860   23.556 0.036872   25.316 33551.31\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:07 -xls -xlsfile test/HLA-B4407_preds.xls\r\n",
      "# Wed Oct 11 17:01:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4407_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4407_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpannPWHVG\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:07 : Distance to training data  0.000 (using nearest neighbor HLA-B44:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000220   47.000 0.017051   74.150 41576.50\r\n",
      "   1 HLA-B*44:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000370   39.000 0.022276   58.599 39291.25\r\n",
      "   1 HLA-B*44:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001010   26.500 0.033275   35.423 34882.83\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A30:10 -xls -xlsfile test/HLA-A3010_preds.xls\r\n",
      "# Wed Oct 11 17:01:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A30:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3010_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A30:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3010_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanV6KaXZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A30:10 : Distance to training data  0.028 (using nearest neighbor HLA-A30:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*30:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0011200   26.122 0.060029   44.740 26115.32\r\n",
      "   1 HLA-A*30:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0005630   35.595 0.055000   49.936 27575.71\r\n",
      "   1 HLA-A*30:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0041840   12.840 0.131872   10.493 12003.49\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*30:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:02 -xls -xlsfile test/HLA-B4402_preds.xls\r\n",
      "# Wed Oct 11 17:01:52 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpangugT0l\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:02 : Distance to training data  0.000 (using nearest neighbor HLA-B44:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000340   43.000 0.023897   67.179 38608.13\r\n",
      "   1 HLA-B*44:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000640   33.800 0.029461   53.843 36352.44\r\n",
      "   1 HLA-B*44:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001250   25.583 0.040822   34.347 32147.60\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:03 -xls -xlsfile test/HLA-B4403_preds.xls\r\n",
      "# Wed Oct 11 17:01:52 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJpe2jJ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:03 : Distance to training data  0.000 (using nearest neighbor HLA-B44:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000220   47.000 0.017051   74.150 41576.50\r\n",
      "   1 HLA-B*44:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000370   39.000 0.022276   58.599 39291.25\r\n",
      "   1 HLA-B*44:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001010   26.500 0.033275   35.423 34882.83\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C03:02 -xls -xlsfile test/HLA-C0302_preds.xls\r\n",
      "# Wed Oct 11 17:01:52 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C03:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0302_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C03:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0302_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanW1VEK3\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C03:02 : Distance to training data  0.049 (using nearest neighbor HLA-C03:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*03:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007390   18.200 0.050660   35.656 28901.48\r\n",
      "   1 HLA-C*03:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0153580    5.723 0.181108    8.405  7046.10\r\n",
      "   1 HLA-C*03:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0187600    5.191 0.247328    5.019  3441.78\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*03:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C03:03 -xls -xlsfile test/HLA-C0303_preds.xls\r\n",
      "# Wed Oct 11 17:01:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C03:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0303_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C03:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0303_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanEorUcV\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C03:03 : Distance to training data  0.000 (using nearest neighbor HLA-C03:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*03:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001320   17.960 0.035632   28.433 34004.49\r\n",
      "   1 HLA-C*03:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0129270    3.107 0.187559    4.402  6571.07\r\n",
      "   1 HLA-C*03:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0275800    2.187 0.281061    2.233  2389.31\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*03:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A11:01 -xls -xlsfile test/HLA-A1101_preds.xls\r\n",
      "# Wed Oct 11 17:01:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A11:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A1101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A11:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A1101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan8UGYcc\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A11:01 : Distance to training data  0.000 (using nearest neighbor HLA-A11:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*11:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000550   31.600 0.034523   44.642 34414.97\r\n",
      "   1 HLA-A*11:01       TLIVNSVL TLIVNSVL-  0  0  0  8  1     TLIVNSVL         PEPLIST 0.0000100   56.667 0.025691   59.191 37865.95\r\n",
      "   1 HLA-A*11:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0003120   17.060 0.073111   18.644 22668.54\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*11:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C03:05 -xls -xlsfile test/HLA-C0305_preds.xls\r\n",
      "# Wed Oct 11 17:01:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C03:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0305_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C03:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0305_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanG5QOBp\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C03:05 : Distance to training data  0.038 (using nearest neighbor HLA-C03:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*03:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007960   13.888 0.043622   24.364 31188.28\r\n",
      "   1 HLA-C*03:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0246940    3.164 0.153957    5.010  9452.15\r\n",
      "   1 HLA-C*03:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0425380    2.274 0.234509    2.300  3953.84\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*03:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A69:01 -xls -xlsfile test/HLA-A6901_preds.xls\r\n",
      "# Wed Oct 11 17:01:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A69:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6901_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A69:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6901_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanwtvbM3\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A69:01 : Distance to training data  0.000 (using nearest neighbor HLA-A69:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*69:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0070310    7.572 0.130736   12.093 12151.94\r\n",
      "   1 HLA-A*69:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0015790   15.206 0.086856   20.039 19536.03\r\n",
      "   1 HLA-A*69:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0208140    4.173 0.216626    5.462  4797.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*69:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:30 -xls -xlsfile test/HLA-A0230_preds.xls\r\n",
      "# Wed Oct 11 17:01:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:30           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0230_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:30           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0230_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanOYvhFs\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:30 : Distance to training data  0.000 (using nearest neighbor HLA-A02:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:30       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0040420   10.080 0.100763   22.090 16806.90\r\n",
      "   1 HLA-A*02:30       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0084200    7.308 0.166648   12.931  8239.41\r\n",
      "   1 HLA-A*02:30      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0069210    7.969 0.189946   11.015  6403.53\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:30. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:07 -xls -xlsfile test/HLA-B2707_preds.xls\r\n",
      "# Wed Oct 11 17:01:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2707_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2707_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanQNRkz1\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:07 : Distance to training data  0.000 (using nearest neighbor HLA-B27:07)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000550   36.600 0.046050   44.693 30379.62\r\n",
      "   1 HLA-B*27:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001150   28.455 0.058910   34.639 26433.44\r\n",
      "   1 HLA-B*27:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0012660   11.129 0.137332   11.384 11314.91\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B53:01 -xls -xlsfile test/HLA-B5301_preds.xls\r\n",
      "# Wed Oct 11 17:01:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B53:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B53:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanSQyEH2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B53:01 : Distance to training data  0.000 (using nearest neighbor HLA-B53:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*53:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000640   34.833 0.015802   65.041 42142.16\r\n",
      "   1 HLA-B*53:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004350   17.639 0.031688   34.238 35486.99\r\n",
      "   1 HLA-B*53:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011300   12.253 0.047753   21.206 29824.98\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*53:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:05 -xls -xlsfile test/HLA-B2705_preds.xls\r\n",
      "# Wed Oct 11 17:01:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2705_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2705_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanqziNLK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:05 : Distance to training data  0.000 (using nearest neighbor HLA-B27:05)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000350   49.667 0.032435   62.326 35201.32\r\n",
      "   1 HLA-B*27:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   44.333 0.037181   54.798 33439.32\r\n",
      "   1 HLA-B*27:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007090   15.805 0.077087   21.387 21714.04\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:03 -xls -xlsfile test/HLA-B2703_preds.xls\r\n",
      "# Wed Oct 11 17:01:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2703_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2703_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfr0KlK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:03 : Distance to training data  0.000 (using nearest neighbor HLA-B27:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000780   43.200 0.016075   61.861 42017.87\r\n",
      "   1 HLA-B*27:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000920   40.714 0.019137   50.972 40648.61\r\n",
      "   1 HLA-B*27:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009640   15.000 0.033162   21.613 34925.51\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:02 -xls -xlsfile test/HLA-B2702_preds.xls\r\n",
      "# Wed Oct 11 17:01:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2702_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2702_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanP8Q2FQ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:02 : Distance to training data  0.000 (using nearest neighbor HLA-B27:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000280   47.500 0.008531   64.853 45591.43\r\n",
      "   1 HLA-B*27:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000610   37.600 0.009886   57.707 44927.88\r\n",
      "   1 HLA-B*27:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0006780   15.777 0.025279   19.485 38035.12\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B41:02 -xls -xlsfile test/HLA-B4102_preds.xls\r\n",
      "# Wed Oct 11 17:01:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B41:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4102_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B41:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4102_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan6OYdbH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B41:02 : Distance to training data  0.028 (using nearest neighbor HLA-B41:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*41:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004730   27.255 0.063060   34.724 25272.78\r\n",
      "   1 HLA-B*41:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0016180   17.032 0.084528   23.722 20034.37\r\n",
      "   1 HLA-B*41:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0027110   13.803 0.118561   14.420 13862.94\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*41:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:17 -xls -xlsfile test/HLA-B1517_preds.xls\r\n",
      "# Wed Oct 11 17:01:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:17           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1517_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:17           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1517_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanMR0OoN\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:17 : Distance to training data  0.000 (using nearest neighbor HLA-B15:17)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:17       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0048020   15.863 0.090981   20.294 18683.27\r\n",
      "   1 HLA-B*15:17       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0035150   17.377 0.112085   16.645 14869.15\r\n",
      "   1 HLA-B*15:17      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.5082240    1.248 0.576169    1.069    98.08 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:17. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:16 -xls -xlsfile test/HLA-B1516_preds.xls\r\n",
      "# Wed Oct 11 17:01:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:16           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1516_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:16           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1516_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJv3nQX\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:16 : Distance to training data  0.114 (using nearest neighbor HLA-B15:17)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:16       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0151730   11.518 0.093312   17.885 18217.95\r\n",
      "   1 HLA-B*15:16       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0031580   22.781 0.077833   22.413 21539.47\r\n",
      "   1 HLA-B*15:16      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.1979530    1.677 0.292653    2.168  2107.67 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:16. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:10 -xls -xlsfile test/HLA-B1510_preds.xls\r\n",
      "# Wed Oct 11 17:01:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1510_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1510_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanvY1t7g\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:10 : Distance to training data  0.000 (using nearest neighbor HLA-B15:10)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000410   37.250 0.010163   60.941 44793.43\r\n",
      "   1 HLA-B*15:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0031720    5.713 0.052328    7.262 28384.57\r\n",
      "   1 HLA-B*15:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005940   13.323 0.034210   14.969 34531.71\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C15:09 -xls -xlsfile test/HLA-C1509_preds.xls\r\n",
      "# Wed Oct 11 17:01:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C15:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1509_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C15:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1509_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan6wXd2r\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C15:09 : Distance to training data  0.031 (using nearest neighbor HLA-C15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*15:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0053000    9.362 0.113512   16.771 14641.33\r\n",
      "   1 HLA-C*15:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0021440   13.971 0.096494   20.807 17601.41\r\n",
      "   1 HLA-C*15:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0413500    2.936 0.310291    2.376  1741.49\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*15:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B56:01 -xls -xlsfile test/HLA-B5601_preds.xls\r\n",
      "# Wed Oct 11 17:01:58 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B56:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B56:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIF0o62\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B56:01 : Distance to training data  0.000 (using nearest neighbor HLA-B56:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*56:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005040   25.956 0.019161   42.662 40638.07\r\n",
      "   1 HLA-B*56:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0005940   24.060 0.020687   39.394 39972.61\r\n",
      "   1 HLA-B*56:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0013350   16.582 0.030343   25.331 36007.18\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*56:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C15:05 -xls -xlsfile test/HLA-C1505_preds.xls\r\n",
      "# Wed Oct 11 17:01:58 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C15:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1505_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C15:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1505_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpansrjiAG\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C15:05 : Distance to training data  0.000 (using nearest neighbor HLA-C15:05)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*15:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0051130    6.010 0.095242    9.805 17841.47\r\n",
      "   1 HLA-C*15:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0030450    7.730 0.084947   11.478 19943.74\r\n",
      "   1 HLA-C*15:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0514550    1.609 0.266164    1.242  2807.20 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*15:05. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C15:04 -xls -xlsfile test/HLA-C1504_preds.xls\r\n",
      "# Wed Oct 11 17:01:58 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C15:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1504_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C15:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1504_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanK5D9pC\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C15:04 : Distance to training data  0.031 (using nearest neighbor HLA-C15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*15:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0053000    9.362 0.113512   16.771 14641.33\r\n",
      "   1 HLA-C*15:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0021440   13.971 0.096494   20.807 17601.41\r\n",
      "   1 HLA-C*15:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0413500    2.936 0.310291    2.376  1741.49\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*15:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C08:12 -xls -xlsfile test/HLA-C0812_preds.xls\r\n",
      "# Wed Oct 11 17:01:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C08:12           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0812_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C08:12           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0812_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJQLGqT\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C08:12 : Distance to training data  0.055 (using nearest neighbor HLA-C08:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*08:12       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0009520   15.089 0.049265   22.781 29341.01\r\n",
      "   1 HLA-C*08:12       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0107820    5.372 0.113199    7.412 14691.00\r\n",
      "   1 HLA-C*08:12      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0099820    5.588 0.167872    3.526  8131.01\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*08:12. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:114 -xls -xlsfile test/HLA-B40114_preds.xls\r\n",
      "# Wed Oct 11 17:01:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:114          MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B40114_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:114          MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B40114_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpandmnRur\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:114 : Distance to training data  0.046 (using nearest neighbor HLA-B40:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:114       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001490   21.529 0.035057   39.575 34216.70\r\n",
      "   1 HLA-B*40:114       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0003570   15.328 0.056896   21.726 27015.78\r\n",
      "   1 HLA-B*40:114      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007260   11.643 0.103429    9.726 16329.02\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:114. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C01:02 -xls -xlsfile test/HLA-C0102_preds.xls\r\n",
      "# Wed Oct 11 17:01:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C01:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0102_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C01:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0102_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanVjlp6X\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C01:02 : Distance to training data  0.000 (using nearest neighbor HLA-C01:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*01:02       GTLIVNSV GTLI-VNSV  0  0  0  4  1     GTLIVNSV         PEPLIST 0.0002020   23.481 0.020252   40.586 40161.18\r\n",
      "   1 HLA-C*01:02       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0283210    2.498 0.139548    3.264 11046.84\r\n",
      "   1 HLA-C*01:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0051390    6.522 0.090632    7.061 18753.96\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*01:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C15:02 -xls -xlsfile test/HLA-C1502_preds.xls\r\n",
      "# Wed Oct 11 17:02:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C15:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1502_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C15:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1502_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanriYXEG\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C15:02 : Distance to training data  0.000 (using nearest neighbor HLA-C15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*15:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0212570    4.017 0.145985    7.185 10303.64\r\n",
      "   1 HLA-C*15:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0023470   11.486 0.088649   14.721 19160.69\r\n",
      "   1 HLA-C*15:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0861680    1.677 0.303241    1.498  1879.53 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*15:02. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B50:01 -xls -xlsfile test/HLA-B5001_preds.xls\r\n",
      "# Wed Oct 11 17:02:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B50:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5001_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B50:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5001_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpantoRAKw\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B50:01 : Distance to training data  0.000 (using nearest neighbor HLA-B50:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*50:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001650   33.667 0.026625   56.878 37485.22\r\n",
      "   1 HLA-B*50:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002890   26.808 0.034481   43.647 34430.62\r\n",
      "   1 HLA-B*50:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002730   27.458 0.044764   31.334 30805.30\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*50:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B82:01 -xls -xlsfile test/HLA-B8201_preds.xls\r\n",
      "# Wed Oct 11 17:02:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B82:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B8201_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B82:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B8201_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpangWbcaF\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B82:01 : Distance to training data  0.104 (using nearest neighbor HLA-B83:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*82:01       GTLIVNSV GTLIV-NSV  0  0  0  5  1     GTLIVNSV         PEPLIST 0.0005370   35.326 0.027076   54.526 37302.73\r\n",
      "   1 HLA-B*82:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0084260   10.224 0.078877   12.115 21297.54\r\n",
      "   1 HLA-B*82:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0151860    6.998 0.095205    8.262 17848.62\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*82:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:15 -xls -xlsfile test/HLA-B4415_preds.xls\r\n",
      "# Wed Oct 11 17:02:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:15           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4415_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:15           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4415_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanDJwmv6\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:15 : Distance to training data  0.054 (using nearest neighbor HLA-B45:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:15       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000910   40.333 0.019715   52.231 40395.20\r\n",
      "   1 HLA-B*44:15       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001050   38.143 0.015676   64.445 42199.65\r\n",
      "   1 HLA-B*44:15      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002490   27.050 0.026270   36.975 37629.48\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:15. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A01:23 -xls -xlsfile test/HLA-A0123_preds.xls\r\n",
      "# Wed Oct 11 17:02:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A01:23           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0123_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A01:23           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0123_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan2fCvuJ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A01:23 : Distance to training data  0.019 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*01:23       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002210   34.154 0.029842   45.206 36202.89\r\n",
      "   1 HLA-A*01:23       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0002510   32.067 0.032243   40.911 35274.52\r\n",
      "   1 HLA-A*01:23      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0006600   19.069 0.047334   22.345 29960.49\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*01:23. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C12:02 -xls -xlsfile test/HLA-C1202_preds.xls\r\n",
      "# Wed Oct 11 17:02:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C12:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1202_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C12:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1202_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpantlHIbA\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C12:02 : Distance to training data  0.000 (using nearest neighbor HLA-C12:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*12:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005480   18.197 0.060354   30.305 26023.66\r\n",
      "   1 HLA-C*12:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0031690    8.740 0.132205   10.425 11960.32\r\n",
      "   1 HLA-C*12:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0055250    6.752 0.181860    5.827  6989.01\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*12:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C12:03 -xls -xlsfile test/HLA-C1203_preds.xls\r\n",
      "# Wed Oct 11 17:02:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C12:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1203_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C12:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1203_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanyCJz2b\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C12:03 : Distance to training data  0.000 (using nearest neighbor HLA-C12:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*12:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0010380   11.550 0.070753   26.818 23254.33\r\n",
      "   1 HLA-C*12:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0027340    7.517 0.128642   12.816 12430.41\r\n",
      "   1 HLA-C*12:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0051020    5.609 0.190427    6.888  6370.29\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*12:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C02:17 -xls -xlsfile test/HLA-C0217_preds.xls\r\n",
      "# Wed Oct 11 17:02:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C02:17           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0217_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C02:17           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0217_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpansKsYMq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C02:17 : Distance to training data  0.071 (using nearest neighbor HLA-C02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*02:17       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002030   18.968 0.038663   29.480 32907.40\r\n",
      "   1 HLA-C*02:17       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0020670    7.297 0.075978   13.217 21976.16\r\n",
      "   1 HLA-C*02:17      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0043330    5.042 0.130273    5.523 12212.97\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*02:17. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A34:02 -xls -xlsfile test/HLA-A3402_preds.xls\r\n",
      "# Wed Oct 11 17:02:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A34:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A34:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanA3P6VK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A34:02 : Distance to training data  0.068 (using nearest neighbor HLA-A03:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*34:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002930   44.091 0.034094   62.243 34575.10\r\n",
      "   1 HLA-A*34:02       TLIVNSVL TLIVNSVL-  0  0  0  8  1     TLIVNSVL         PEPLIST 0.0005000   37.400 0.053775   41.549 27943.63\r\n",
      "   1 HLA-A*34:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0028210   21.056 0.072716   29.780 22765.63\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*34:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A32:01 -xls -xlsfile test/HLA-A3201_preds.xls\r\n",
      "# Wed Oct 11 17:02:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A32:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3201_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A32:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3201_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanugQB06\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A32:01 : Distance to training data  0.000 (using nearest neighbor HLA-A32:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*32:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005080   19.364 0.059752   21.395 26193.72\r\n",
      "   1 HLA-A*32:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002200   26.083 0.047641   26.894 29861.15\r\n",
      "   1 HLA-A*32:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0300720    2.822 0.277547    2.100  2481.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*32:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:03 -xls -xlsfile test/HLA-C1703_preds.xls\r\n",
      "# Wed Oct 11 17:02:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1703_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1703_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanZ1IYrD\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:03 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:03. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:01 -xls -xlsfile test/HLA-C1701_preds.xls\r\n",
      "# Wed Oct 11 17:02:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKsjTQS\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:01 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:01. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:02 -xls -xlsfile test/HLA-C1702_preds.xls\r\n",
      "# Wed Oct 11 17:02:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1702_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1702_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanqayq2Z\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:02 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:02. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:04 -xls -xlsfile test/HLA-C1704_preds.xls\r\n",
      "# Wed Oct 11 17:02:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1704_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1704_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanokbYZp\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:04 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:04. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:05 -xls -xlsfile test/HLA-C1705_preds.xls\r\n",
      "# Wed Oct 11 17:02:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1705_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1705_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanhW44Gp\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:05 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:05. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:06 -xls -xlsfile test/HLA-C1706_preds.xls\r\n",
      "# Wed Oct 11 17:02:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1706_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1706_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan7Jq8vC\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:06 : Distance to training data  0.000 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0031180   11.981 0.062446   19.647 25441.23\r\n",
      "   1 HLA-C*17:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284840    4.327 0.144229    6.468 10501.28\r\n",
      "   1 HLA-C*17:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0967770    1.957 0.278436    1.709  2458.15 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:06. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C17:07 -xls -xlsfile test/HLA-C1707_preds.xls\r\n",
      "# Wed Oct 11 17:02:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C17:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1707_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C17:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1707_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanTIcyBz\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C17:07 : Distance to training data  0.040 (using nearest neighbor HLA-C17:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*17:07       GTLIVNSV GTLIV-NSV  0  0  0  5  1     GTLIVNSV         PEPLIST 0.0018580   14.276 0.036035   26.849 33856.54\r\n",
      "   1 HLA-C*17:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0249730    4.257 0.113215    6.561 14688.46\r\n",
      "   1 HLA-C*17:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0611420    2.354 0.204802    2.180  5452.70\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*17:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A43:01 -xls -xlsfile test/HLA-A4301_preds.xls\r\n",
      "# Wed Oct 11 17:02:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A43:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A4301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A43:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A4301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanWTKIKU\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A43:01 : Distance to training data  0.059 (using nearest neighbor HLA-A26:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*43:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004660   27.468 0.036628   36.010 33640.01\r\n",
      "   1 HLA-A*43:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0013250   18.450 0.058754   20.826 26478.09\r\n",
      "   1 HLA-A*43:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0059870    9.450 0.099647    9.510 17011.08\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*43:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B78:01 -xls -xlsfile test/HLA-B7801_preds.xls\r\n",
      "# Wed Oct 11 17:02:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B78:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B7801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B78:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B7801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfY28ic\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B78:01 : Distance to training data  0.129 (using nearest neighbor HLA-B51:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*78:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0042220   18.158 0.041771   29.292 31819.21\r\n",
      "   1 HLA-B*78:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0110380   10.591 0.061289   16.701 25761.71\r\n",
      "   1 HLA-B*78:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066180   14.347 0.052764   20.965 28250.98\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*78:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A68:27 -xls -xlsfile test/HLA-A6827_preds.xls\r\n",
      "# Wed Oct 11 17:02:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A68:27           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6827_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A68:27           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6827_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanRC1eqb\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A68:27 : Distance to training data  0.000 (using nearest neighbor HLA-A68:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*68:27       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0053790    9.557 0.135262   15.263 11571.19\r\n",
      "   1 HLA-A*68:27       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0006430   23.814 0.076209   28.741 21921.29\r\n",
      "   1 HLA-A*68:27      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0195660    4.813 0.218027    7.772  4725.72\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*68:27. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B73:01 -xls -xlsfile test/HLA-B7301_preds.xls\r\n",
      "# Wed Oct 11 17:02:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B73:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B7301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B73:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B7301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanVURRkM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B73:01 : Distance to training data  0.000 (using nearest neighbor HLA-B73:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*73:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004140   41.269 0.017013   58.206 41593.59\r\n",
      "   1 HLA-B*73:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004690   39.424 0.021510   46.616 39618.23\r\n",
      "   1 HLA-B*73:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011250   27.649 0.023367   42.675 38830.16\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*73:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A26:12 -xls -xlsfile test/HLA-A2612_preds.xls\r\n",
      "# Wed Oct 11 17:02:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A26:12           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2612_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A26:12           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2612_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpandMDp4c\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A26:12 : Distance to training data  0.032 (using nearest neighbor HLA-A26:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*26:12       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002300   31.600 0.033498   42.174 34798.77\r\n",
      "   1 HLA-A*26:12       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002170   32.278 0.039491   35.469 32613.92\r\n",
      "   1 HLA-A*26:12      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0044440    8.910 0.088677   11.832 19154.88\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*26:12. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C08:04 -xls -xlsfile test/HLA-C0804_preds.xls\r\n",
      "# Wed Oct 11 17:02:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C08:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0804_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C08:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0804_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpangr5Xcu\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C08:04 : Distance to training data  0.031 (using nearest neighbor HLA-C08:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*08:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0011660   11.720 0.045824   21.067 30454.01\r\n",
      "   1 HLA-C*08:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0138540    3.774 0.115678    5.607 14302.19\r\n",
      "   1 HLA-C*08:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0206140    3.010 0.184726    2.320  6775.60\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*08:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C08:03 -xls -xlsfile test/HLA-C0803_preds.xls\r\n",
      "# Wed Oct 11 17:02:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C08:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0803_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C08:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0803_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanvLAfiq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C08:03 : Distance to training data  0.000 (using nearest neighbor HLA-C08:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*08:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003340   13.136 0.029009   23.337 36530.67\r\n",
      "   1 HLA-C*08:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0036870    4.579 0.069525    7.220 23565.36\r\n",
      "   1 HLA-C*08:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0069810    3.275 0.120531    2.741 13570.58\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*08:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C08:02 -xls -xlsfile test/HLA-C0802_preds.xls\r\n",
      "# Wed Oct 11 17:02:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C08:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C08:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanab8R6y\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C08:02 : Distance to training data  0.000 (using nearest neighbor HLA-C08:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*08:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003020   18.050 0.029649   31.053 36278.57\r\n",
      "   1 HLA-C*08:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0041010    5.871 0.077913    8.318 21520.84\r\n",
      "   1 HLA-C*08:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0032550    6.545 0.104271    5.133 16180.94\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*08:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C08:01 -xls -xlsfile test/HLA-C0801_preds.xls\r\n",
      "# Wed Oct 11 17:02:08 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C08:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C08:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanf3xDSY\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C08:01 : Distance to training data  0.000 (using nearest neighbor HLA-C08:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*08:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003340   13.136 0.029009   23.337 36530.67\r\n",
      "   1 HLA-C*08:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0036870    4.579 0.069525    7.220 23565.36\r\n",
      "   1 HLA-C*08:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0069810    3.275 0.120531    2.741 13570.58\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C03:04 -xls -xlsfile test/HLA-C0304_preds.xls\r\n",
      "# Wed Oct 11 17:02:08 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C03:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0304_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C03:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0304_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan9xaFrS\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C03:04 : Distance to training data  0.000 (using nearest neighbor HLA-C03:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*03:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001320   17.960 0.035632   28.433 34004.49\r\n",
      "   1 HLA-C*03:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0129270    3.107 0.187559    4.402  6571.07\r\n",
      "   1 HLA-C*03:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0275800    2.187 0.281061    2.233  2389.31\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*03:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B18:01 -xls -xlsfile test/HLA-B1801_preds.xls\r\n",
      "# Wed Oct 11 17:02:08 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B18:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B18:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanu0rUc7\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B18:01 : Distance to training data  0.000 (using nearest neighbor HLA-B18:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*18:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000900   36.714 0.025572   60.706 37914.73\r\n",
      "   1 HLA-B*18:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0003670   21.231 0.044680   30.521 30833.30\r\n",
      "   1 HLA-B*18:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002220   26.095 0.041690   33.694 31847.11\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*18:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B18:03 -xls -xlsfile test/HLA-B1803_preds.xls\r\n",
      "# Wed Oct 11 17:02:09 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B18:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1803_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B18:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1803_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanEGfd56\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B18:03 : Distance to training data  0.000 (using nearest neighbor HLA-B18:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*18:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003200   26.871 0.035270   58.882 34137.95\r\n",
      "   1 HLA-B*18:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0008210   17.982 0.064463   27.391 24892.02\r\n",
      "   1 HLA-B*18:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004690   22.904 0.059449   30.972 26279.73\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*18:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:11 -xls -xlsfile test/HLA-A0211_preds.xls\r\n",
      "# Wed Oct 11 17:02:09 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:11           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0211_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:11           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0211_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan262jFY\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:11 : Distance to training data  0.000 (using nearest neighbor HLA-A02:11)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:11       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0104860   12.400 0.158174   19.307  9030.57\r\n",
      "   1 HLA-A*02:11       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0211550    9.397 0.205198   14.731  5429.39\r\n",
      "   1 HLA-A*02:11      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0184080    9.938 0.240966   12.257  3687.04\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:11. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:14 -xls -xlsfile test/HLA-A0214_preds.xls\r\n",
      "# Wed Oct 11 17:02:09 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:14           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0214_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:14           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0214_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan4AwpO8\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:14 : Distance to training data  0.015 (using nearest neighbor HLA-A02:06)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:14       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0184980    6.543 0.161687   17.056  8693.76\r\n",
      "   1 HLA-A*02:14       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0091590    9.600 0.149597   18.961  9908.74\r\n",
      "   1 HLA-A*02:14      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0487020    3.457 0.315170    4.975  1651.95\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:14. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C05:09 -xls -xlsfile test/HLA-C0509_preds.xls\r\n",
      "# Wed Oct 11 17:02:10 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C05:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0509_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C05:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0509_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanoAo9D1\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C05:09 : Distance to training data  0.000 (using nearest neighbor HLA-C05:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*05:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004530   16.800 0.035616   29.215 34010.39\r\n",
      "   1 HLA-C*05:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0022560    8.382 0.066490   12.920 24352.05\r\n",
      "   1 HLA-C*05:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0025930    7.859 0.102523    6.371 16489.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*05:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B14:01 -xls -xlsfile test/HLA-B1401_preds.xls\r\n",
      "# Wed Oct 11 17:02:10 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B14:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1401_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B14:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1401_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanNbd4W5\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B14:01 : Distance to training data  0.000 (using nearest neighbor HLA-B14:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*14:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004840   27.140 0.031467   42.885 35571.93\r\n",
      "   1 HLA-B*14:01       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0079840    7.218 0.091137    8.759 18651.77\r\n",
      "   1 HLA-B*14:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0029300   12.462 0.065072   16.256 24728.55\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*14:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B14:03 -xls -xlsfile test/HLA-B1403_preds.xls\r\n",
      "# Wed Oct 11 17:02:10 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B14:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B14:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanueUfj0\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B14:03 : Distance to training data  0.031 (using nearest neighbor HLA-B14:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*14:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0008120   29.613 0.035026   46.936 34228.18\r\n",
      "   1 HLA-B*14:03       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0182920    6.525 0.112781    8.157 14757.59\r\n",
      "   1 HLA-B*14:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0065910   11.903 0.088616   13.263 19167.52\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*14:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B14:02 -xls -xlsfile test/HLA-B1402_preds.xls\r\n",
      "# Wed Oct 11 17:02:11 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B14:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B14:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanOeDjrE\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B14:02 : Distance to training data  0.000 (using nearest neighbor HLA-B14:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*14:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004840   27.140 0.031467   42.885 35571.93\r\n",
      "   1 HLA-B*14:02       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0079840    7.218 0.091137    8.759 18651.77\r\n",
      "   1 HLA-B*14:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0029300   12.462 0.065072   16.256 24728.55\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*14:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:03 -xls -xlsfile test/HLA-A7403_preds.xls\r\n",
      "# Wed Oct 11 17:02:11 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCPEfnL\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:03 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:01 -xls -xlsfile test/HLA-A7401_preds.xls\r\n",
      "# Wed Oct 11 17:02:11 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7401_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7401_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKC75Qo\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:01 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:02 -xls -xlsfile test/HLA-A7402_preds.xls\r\n",
      "# Wed Oct 11 17:02:12 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCoiFFn\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:02 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:04 -xls -xlsfile test/HLA-A7404_preds.xls\r\n",
      "# Wed Oct 11 17:02:12 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7404_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7404_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan2UFE48\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:04 : Distance to training data  0.127 (using nearest neighbor HLA-A02:04)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002890   34.375 0.039931   47.143 32459.02\r\n",
      "   1 HLA-A*74:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004570   29.050 0.057000   33.234 26985.40\r\n",
      "   1 HLA-A*74:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0028110   13.755 0.096200   17.265 17657.50\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:05 -xls -xlsfile test/HLA-A7405_preds.xls\r\n",
      "# Wed Oct 11 17:02:12 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7405_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7405_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan9sOS2C\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:05 : Distance to training data  0.065 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000940   33.000 0.030688   43.181 35873.03\r\n",
      "   1 HLA-A*74:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000550   39.600 0.027057   48.503 37310.41\r\n",
      "   1 HLA-A*74:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007160   15.496 0.076176   14.466 21929.13\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:06 -xls -xlsfile test/HLA-A7406_preds.xls\r\n",
      "# Wed Oct 11 17:02:13 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7406_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7406_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan7Uwd1D\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:06 : Distance to training data  0.108 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000910   29.500 0.025952   42.734 37759.17\r\n",
      "   1 HLA-A*74:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000620   33.833 0.025847   42.901 37802.07\r\n",
      "   1 HLA-A*74:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0006430   13.676 0.065185   13.474 24698.33\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:07 -xls -xlsfile test/HLA-A7407_preds.xls\r\n",
      "# Wed Oct 11 17:02:13 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7407_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7407_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanruaihN\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:07 : Distance to training data  0.039 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000790   36.333 0.028032   46.170 36918.88\r\n",
      "   1 HLA-A*74:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000470   43.333 0.026628   48.380 37483.98\r\n",
      "   1 HLA-A*74:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009680   13.603 0.073921   14.393 22470.74\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:08 -xls -xlsfile test/HLA-A7408_preds.xls\r\n",
      "# Wed Oct 11 17:02:13 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7408_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7408_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpansdix5G\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:08 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:09 -xls -xlsfile test/HLA-A7409_preds.xls\r\n",
      "# Wed Oct 11 17:02:14 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7409_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7409_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanYsRWPY\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:09 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:10 -xls -xlsfile test/HLA-A7410_preds.xls\r\n",
      "# Wed Oct 11 17:02:14 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7410_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7410_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanaZCnnz\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:10 : Distance to training data  0.104 (using nearest neighbor HLA-A32:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001990   22.957 0.037730   28.720 33241.30\r\n",
      "   1 HLA-A*74:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000990   29.667 0.030101   36.978 36101.59\r\n",
      "   1 HLA-A*74:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0020020    8.037 0.099264    6.408 17081.71\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:11 -xls -xlsfile test/HLA-A7411_preds.xls\r\n",
      "# Wed Oct 11 17:02:14 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:11           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7411_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:11           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7411_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanifv5q2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:11 : Distance to training data  0.067 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:11       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   35.250 0.025589   44.852 37907.76\r\n",
      "   1 HLA-A*74:11       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000500   35.000 0.024203   47.289 38480.51\r\n",
      "   1 HLA-A*74:11      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005540   14.012 0.065766   14.417 24543.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:11. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A74:13 -xls -xlsfile test/HLA-A7413_preds.xls\r\n",
      "# Wed Oct 11 17:02:15 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A74:13           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A7413_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A74:13           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A7413_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanMEftdA\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A74:13 : Distance to training data  0.073 (using nearest neighbor HLA-A03:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*74:13       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001180   29.833 0.034069   40.534 34584.44\r\n",
      "   1 HLA-A*74:13       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000550   39.000 0.030597   44.975 35908.38\r\n",
      "   1 HLA-A*74:13      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011200   12.455 0.084171   13.484 20111.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*74:13. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:37 -xls -xlsfile test/HLA-B1537_preds.xls\r\n",
      "# Wed Oct 11 17:02:15 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:37           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1537_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:37           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1537_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanK16eaZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:37 : Distance to training data  0.027 (using nearest neighbor HLA-B15:10)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:37       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001200   32.636 0.013617   56.202 43150.34\r\n",
      "   1 HLA-B*15:37       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0042170    6.863 0.061380    6.836 25736.36\r\n",
      "   1 HLA-B*15:37      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0008760   14.979 0.037059   16.906 33483.50\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:37. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A80:01 -xls -xlsfile test/HLA-A8001_preds.xls\r\n",
      "# Wed Oct 11 17:02:15 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A80:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A8001_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A80:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A8001_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanc3IMF4\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A80:01 : Distance to training data  0.000 (using nearest neighbor HLA-A80:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*80:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000500   42.000 0.016828   61.442 41676.93\r\n",
      "   1 HLA-A*80:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001430   28.500 0.027673   38.339 37062.56\r\n",
      "   1 HLA-A*80:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005960   15.500 0.044508   20.602 30890.74\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*80:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A36:01 -xls -xlsfile test/HLA-A3601_preds.xls\r\n",
      "# Wed Oct 11 17:02:16 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A36:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A36:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfWj20z\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A36:01 : Distance to training data  0.086 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*36:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004390   37.071 0.045691   43.260 30497.86\r\n",
      "   1 HLA-A*36:01       TLIVNSVL TLIVNSVL-  0  0  0  8  1     TLIVNSVL         PEPLIST 0.0003580   40.333 0.041336   48.653 31969.32\r\n",
      "   1 HLA-A*36:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0033290   13.241 0.094241   12.527 18035.75\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*36:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:31 -xls -xlsfile test/HLA-B1531_preds.xls\r\n",
      "# Wed Oct 11 17:02:16 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:31           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1531_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:31           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1531_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfCtzX6\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:31 : Distance to training data  0.044 (using nearest neighbor HLA-B15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:31       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002670   35.667 0.026884   61.083 37380.30\r\n",
      "   1 HLA-B*15:31       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0139490    6.555 0.110518   10.079 15123.40\r\n",
      "   1 HLA-B*15:31      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0051300   10.618 0.095069   12.858 17874.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:31. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A30:09 -xls -xlsfile test/HLA-A3009_preds.xls\r\n",
      "# Wed Oct 11 17:02:16 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A30:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3009_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A30:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3009_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanB5pEdo\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A30:09 : Distance to training data  0.039 (using nearest neighbor HLA-A30:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*30:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0014570   30.828 0.094167   38.241 18050.21\r\n",
      "   1 HLA-A*30:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0010210   35.569 0.077084   49.282 21714.74\r\n",
      "   1 HLA-A*30:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0122910   10.485 0.199364    8.791  5783.15\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*30:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A26:08 -xls -xlsfile test/HLA-A2608_preds.xls\r\n",
      "# Wed Oct 11 17:02:17 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A26:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2608_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A26:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2608_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanMKJkAS\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A26:08 : Distance to training data  0.050 (using nearest neighbor HLA-A26:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*26:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001480   40.000 0.026823   46.807 37405.00\r\n",
      "   1 HLA-A*26:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0006080   24.131 0.045040   25.909 30713.43\r\n",
      "   1 HLA-A*26:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0033220   11.626 0.068962   14.064 23709.36\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*26:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B42:02 -xls -xlsfile test/HLA-B4202_preds.xls\r\n",
      "# Wed Oct 11 17:02:17 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B42:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4202_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B42:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4202_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan00NEly\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B42:02 : Distance to training data  0.028 (using nearest neighbor HLA-B42:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*42:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003580   33.690 0.037499   45.780 33324.47\r\n",
      "   1 HLA-B*42:02       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0091090    7.584 0.105566   11.052 15955.79\r\n",
      "   1 HLA-B*42:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0074830    8.443 0.118499    8.895 13872.25\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*42:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B42:01 -xls -xlsfile test/HLA-B4201_preds.xls\r\n",
      "# Wed Oct 11 17:02:17 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B42:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4201_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B42:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4201_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCJfjoE\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B42:01 : Distance to training data  0.000 (using nearest neighbor HLA-B42:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*42:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004910   30.021 0.045734   42.499 30483.68\r\n",
      "   1 HLA-B*42:01       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0119620    8.423 0.139093   11.178 11101.36\r\n",
      "   1 HLA-B*42:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0191530    6.692 0.178365    7.506  7258.36\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*42:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A26:01 -xls -xlsfile test/HLA-A2601_preds.xls\r\n",
      "# Wed Oct 11 17:02:18 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A26:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A26:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJoUDEx\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A26:01 : Distance to training data  0.000 (using nearest neighbor HLA-A26:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*26:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001030   35.889 0.022485   45.598 39202.48\r\n",
      "   1 HLA-A*26:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001990   28.294 0.032560   30.257 35153.74\r\n",
      "   1 HLA-A*26:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0006710   17.300 0.048923   17.689 29449.79\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*26:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B48:01 -xls -xlsfile test/HLA-B4801_preds.xls\r\n",
      "# Wed Oct 11 17:02:18 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B48:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B48:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpancbO7GM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B48:01 : Distance to training data  0.000 (using nearest neighbor HLA-B48:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*48:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002640   20.889 0.021643   36.925 39561.27\r\n",
      "   1 HLA-B*48:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0072710    4.547 0.081719    6.134 20652.61\r\n",
      "   1 HLA-B*48:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0139720    3.005 0.118558    2.946 13863.39\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*48:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C02:14 -xls -xlsfile test/HLA-C0214_preds.xls\r\n",
      "# Wed Oct 11 17:02:18 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C02:14           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0214_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C02:14           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0214_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanDVkNfR\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C02:14 : Distance to training data  0.047 (using nearest neighbor HLA-C02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*02:14       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000740   35.429 0.019969   53.758 40284.35\r\n",
      "   1 HLA-C*02:14       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0005830   16.602 0.041613   23.006 31873.65\r\n",
      "   1 HLA-C*02:14      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007440   14.921 0.050762   16.864 28869.61\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*02:14. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C02:10 -xls -xlsfile test/HLA-C0210_preds.xls\r\n",
      "# Wed Oct 11 17:02:19 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C02:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0210_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C02:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0210_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan0RMCin\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C02:10 : Distance to training data  0.000 (using nearest neighbor HLA-C02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*02:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003990   18.984 0.051928   27.683 28507.69\r\n",
      "   1 HLA-C*02:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0013940   11.647 0.079360   16.415 21186.52\r\n",
      "   1 HLA-C*02:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066430    5.762 0.156876    5.460  9158.29\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*02:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:01 -xls -xlsfile test/HLA-B3901_preds.xls\r\n",
      "# Wed Oct 11 17:02:19 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3901_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3901_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanRDpsbi\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:01 : Distance to training data  0.000 (using nearest neighbor HLA-B39:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000660   39.800 0.023886   53.667 38612.71\r\n",
      "   1 HLA-B*39:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0056460    7.065 0.112036    8.542 14877.03\r\n",
      "   1 HLA-B*39:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0019750   11.509 0.102291    9.871 16531.33\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B57:04 -xls -xlsfile test/HLA-B5704_preds.xls\r\n",
      "# Wed Oct 11 17:02:19 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B57:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5704_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B57:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5704_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanL6dUVH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B57:04 : Distance to training data  0.060 (using nearest neighbor HLA-B57:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*57:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0038270   17.734 0.069066   21.086 23682.69\r\n",
      "   1 HLA-B*57:04       TLIVNSVL -TLIVNSVL  0  0  0  0  1     TLIVNSVL         PEPLIST 0.0016750   24.725 0.058374   26.430 26587.19\r\n",
      "   1 HLA-B*57:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.1134740    3.065 0.235939    2.436  3893.14\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*57:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B47:01 -xls -xlsfile test/HLA-B4701_preds.xls\r\n",
      "# Wed Oct 11 17:02:20 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B47:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B47:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanVx8tX3\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B47:01 : Distance to training data  0.000 (using nearest neighbor HLA-B47:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*47:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007750   28.070 0.056893   54.677 27016.65\r\n",
      "   1 HLA-B*47:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0017600   19.816 0.087457   32.137 19409.40\r\n",
      "   1 HLA-B*47:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0039180   13.270 0.152650   11.270  9586.77\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*47:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B57:01 -xls -xlsfile test/HLA-B5701_preds.xls\r\n",
      "# Wed Oct 11 17:02:20 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B57:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B57:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanzCUoez\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B57:01 : Distance to training data  0.000 (using nearest neighbor HLA-B57:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*57:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0020800   18.211 0.074232   22.341 22395.26\r\n",
      "   1 HLA-B*57:01       TLIVNSVL -TLIVNSVL  0  0  0  0  1     TLIVNSVL         PEPLIST 0.0008070   27.443 0.065213   26.456 24690.85\r\n",
      "   1 HLA-B*57:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0687210    3.498 0.234256    3.309  3964.68\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*57:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B57:02 -xls -xlsfile test/HLA-B5702_preds.xls\r\n",
      "# Wed Oct 11 17:02:20 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B57:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5702_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B57:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5702_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanQCSfUw\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B57:02 : Distance to training data  0.033 (using nearest neighbor HLA-B57:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*57:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0017050   18.151 0.050770   24.505 28867.11\r\n",
      "   1 HLA-B*57:02       TLIVNSVL -TLIVNSVL  0  0  0  0  1     TLIVNSVL         PEPLIST 0.0008450   23.737 0.048697   25.688 29521.90\r\n",
      "   1 HLA-B*57:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0821440    2.767 0.225836    2.388  4342.84\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*57:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B57:03 -xls -xlsfile test/HLA-B5703_preds.xls\r\n",
      "# Wed Oct 11 17:02:21 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B57:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5703_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B57:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5703_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpancaaLsO\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B57:03 : Distance to training data  0.000 (using nearest neighbor HLA-B57:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*57:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0028540   14.560 0.072523   19.268 22813.22\r\n",
      "   1 HLA-B*57:03       TLIVNSVL -TLIVNSVL  0  0  0  0  1     TLIVNSVL         PEPLIST 0.0011890   20.377 0.073683   18.905 22528.68\r\n",
      "   1 HLA-B*57:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.1722280    1.986 0.326344    1.687  1463.83 <= WB\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*57:03. Number of high binders 0. Number of weak binders 1. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:01 -xls -xlsfile test/HLA-B3501_preds.xls\r\n",
      "# Wed Oct 11 17:02:21 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanEZey6U\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:01 : Distance to training data  0.000 (using nearest neighbor HLA-B35:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000310   45.500 0.012508   80.537 43671.23\r\n",
      "   1 HLA-B*35:01       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0012440   12.740 0.049356   23.818 29312.15\r\n",
      "   1 HLA-B*35:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005970   16.813 0.051033   22.959 28785.08\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:02 -xls -xlsfile test/HLA-B3502_preds.xls\r\n",
      "# Wed Oct 11 17:02:21 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3502_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3502_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan8AYKQk\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:02 : Distance to training data  0.000 (using nearest neighbor HLA-B35:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001140   25.154 0.014758   51.126 42620.90\r\n",
      "   1 HLA-B*35:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0041630    5.832 0.071625    8.835 23035.96\r\n",
      "   1 HLA-B*35:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0054600    5.124 0.080142    7.493 21008.02\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:04 -xls -xlsfile test/HLA-A0204_preds.xls\r\n",
      "# Wed Oct 11 17:02:22 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0204_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0204_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIlzlBq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:04 : Distance to training data  0.000 (using nearest neighbor HLA-A02:04)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0032980    9.425 0.071215   21.067 23138.38\r\n",
      "   1 HLA-A*02:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0057580    7.181 0.107795   13.193 15575.59\r\n",
      "   1 HLA-A*02:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0077760    6.182 0.130391   10.440 12197.38\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:02 -xls -xlsfile test/HLA-A0202_preds.xls\r\n",
      "# Wed Oct 11 17:02:22 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0202_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0202_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanswvVJw\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:02 : Distance to training data  0.000 (using nearest neighbor HLA-A02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0048000   14.000 0.130891   25.784 12131.58\r\n",
      "   1 HLA-A*02:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0277210    6.175 0.261631   11.329  2948.31\r\n",
      "   1 HLA-A*02:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0169560    7.966 0.255555   11.718  3148.65\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:01 -xls -xlsfile test/HLA-A0201_preds.xls\r\n",
      "# Wed Oct 11 17:02:22 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0201_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0201_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanGrxldD\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:01 : Distance to training data  0.000 (using nearest neighbor HLA-A02:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0040420   10.080 0.100763   22.090 16806.90\r\n",
      "   1 HLA-A*02:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0084200    7.308 0.166648   12.931  8239.41\r\n",
      "   1 HLA-A*02:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0069210    7.969 0.189946   11.015  6403.53\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:08 -xls -xlsfile test/HLA-B3508_preds.xls\r\n",
      "# Wed Oct 11 17:02:23 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3508_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3508_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan8J4773\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:08 : Distance to training data  0.000 (using nearest neighbor HLA-B35:08)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000190   53.333 0.011822   82.021 43996.57\r\n",
      "   1 HLA-B*35:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0006910   14.977 0.043537   22.587 31216.98\r\n",
      "   1 HLA-B*35:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004240   18.123 0.045679   21.226 30501.83\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:18 -xls -xlsfile test/HLA-B1518_preds.xls\r\n",
      "# Wed Oct 11 17:02:23 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:18           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1518_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:18           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1518_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanWDM0q7\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:18 : Distance to training data  0.000 (using nearest neighbor HLA-B15:18)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:18       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000480   55.000 0.012744   85.460 43559.84\r\n",
      "   1 HLA-B*15:18       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0028130   11.964 0.064087   15.820 24993.49\r\n",
      "   1 HLA-B*15:18      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0012260   17.488 0.042289   29.241 31641.36\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:18. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A68:02 -xls -xlsfile test/HLA-A6802_preds.xls\r\n",
      "# Wed Oct 11 17:02:23 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A68:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A68:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanj4bGnM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A68:02 : Distance to training data  0.000 (using nearest neighbor HLA-A68:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*68:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0053790    9.557 0.135262   15.263 11571.19\r\n",
      "   1 HLA-A*68:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0006430   23.814 0.076209   28.741 21921.29\r\n",
      "   1 HLA-A*68:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0195660    4.813 0.218027    7.772  4725.72\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*68:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A68:01 -xls -xlsfile test/HLA-A6801_preds.xls\r\n",
      "# Wed Oct 11 17:02:24 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A68:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A68:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan9es9je\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A68:01 : Distance to training data  0.000 (using nearest neighbor HLA-A68:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*68:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000470   46.667 0.022948   67.842 39006.58\r\n",
      "   1 HLA-A*68:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000390   49.333 0.028020   59.689 36923.66\r\n",
      "   1 HLA-A*68:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0003770   24.348 0.064117   28.625 24985.39\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*68:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:06 -xls -xlsfile test/HLA-A0206_preds.xls\r\n",
      "# Wed Oct 11 17:02:24 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0206_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0206_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJiCVH8\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:06 : Distance to training data  0.000 (using nearest neighbor HLA-A02:06)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0258930    5.545 0.190097   15.646  6393.08\r\n",
      "   1 HLA-A*02:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0062490   11.665 0.143652   22.210 10567.04\r\n",
      "   1 HLA-A*02:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0339140    4.711 0.316196    6.728  1633.71\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B52:01 -xls -xlsfile test/HLA-B5201_preds.xls\r\n",
      "# Wed Oct 11 17:02:24 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B52:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5201_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B52:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5201_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanl13GRe\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B52:01 : Distance to training data  0.000 (using nearest neighbor HLA-B52:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*52:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0444400    4.220 0.092957   11.881 18288.06\r\n",
      "   1 HLA-B*52:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0284260    6.184 0.082117   14.923 20563.86\r\n",
      "   1 HLA-B*52:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0325660    5.518 0.099337   10.401 17068.23\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*52:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A29:01 -xls -xlsfile test/HLA-A2901_preds.xls\r\n",
      "# Wed Oct 11 17:02:25 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A29:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2901_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A29:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2901_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanj8h1Ft\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A29:01 : Distance to training data  0.000 (using nearest neighbor HLA-A29:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*29:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000330   50.556 0.020711   69.003 39962.22\r\n",
      "   1 HLA-A*29:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001570   29.154 0.042137   38.646 31693.44\r\n",
      "   1 HLA-A*29:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004410   18.947 0.062239   24.635 25498.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*29:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A29:02 -xls -xlsfile test/HLA-A2902_preds.xls\r\n",
      "# Wed Oct 11 17:02:25 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A29:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2902_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A29:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2902_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXjJ7Il\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A29:02 : Distance to training data  0.000 (using nearest neighbor HLA-A29:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*29:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000330   50.556 0.020711   69.003 39962.22\r\n",
      "   1 HLA-A*29:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001570   29.154 0.042137   38.646 31693.44\r\n",
      "   1 HLA-A*29:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004410   18.947 0.062239   24.635 25498.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*29:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:03 -xls -xlsfile test/HLA-B3503_preds.xls\r\n",
      "# Wed Oct 11 17:02:25 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3503_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3503_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKfKxAx\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:03 : Distance to training data  0.000 (using nearest neighbor HLA-B35:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001960   21.423 0.011893   45.841 43962.79\r\n",
      "   1 HLA-B*35:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0045020    6.014 0.055050    8.177 27560.79\r\n",
      "   1 HLA-B*35:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0074840    4.740 0.062234    6.887 25499.66\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:08 -xls -xlsfile test/HLA-B5108_preds.xls\r\n",
      "# Wed Oct 11 17:02:26 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5108_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5108_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanZuPjge\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:08 : Distance to training data  0.000 (using nearest neighbor HLA-B51:08)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0102020    9.015 0.067505   11.746 24086.07\r\n",
      "   1 HLA-B*51:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0047180   13.479 0.050197   18.837 29046.63\r\n",
      "   1 HLA-B*51:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0042270   14.206 0.048139   20.018 29700.68\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:24 -xls -xlsfile test/HLA-B3924_preds.xls\r\n",
      "# Wed Oct 11 17:02:26 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:24           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3924_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:24           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3924_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanSZ9RF0\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:24 : Distance to training data  0.000 (using nearest neighbor HLA-B39:24)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:24       GTLIVNSV -GTLIVNSV  0  0  0  0  1     GTLIVNSV         PEPLIST 0.0004660   31.980 0.034648   47.072 34368.46\r\n",
      "   1 HLA-B*39:24       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0247390    6.337 0.132072    8.981 11977.54\r\n",
      "   1 HLA-B*39:24      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0049710   13.565 0.098696   14.121 17187.02\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:24. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A26:30 -xls -xlsfile test/HLA-A2630_preds.xls\r\n",
      "# Wed Oct 11 17:02:26 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A26:30           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2630_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A26:30           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2630_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanMpsY9e\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A26:30 : Distance to training data  0.000 (using nearest neighbor HLA-A26:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*26:30       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001740   31.867 0.026975   50.582 37343.53\r\n",
      "   1 HLA-A*26:30       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004070   22.707 0.044216   29.233 30988.49\r\n",
      "   1 HLA-A*26:30      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007860   16.883 0.068315   15.531 23875.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*26:30. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B41:01 -xls -xlsfile test/HLA-B4101_preds.xls\r\n",
      "# Wed Oct 11 17:02:27 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B41:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B41:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanZ9T3If\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B41:01 : Distance to training data  0.000 (using nearest neighbor HLA-B41:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*41:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000880   34.333 0.041575   45.141 31886.75\r\n",
      "   1 HLA-B*41:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000940   33.429 0.042165   44.374 31683.86\r\n",
      "   1 HLA-B*41:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001260   29.700 0.057293   29.424 26899.98\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*41:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C02:02 -xls -xlsfile test/HLA-C0202_preds.xls\r\n",
      "# Wed Oct 11 17:02:27 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C02:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0202_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C02:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0202_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanNNjdWC\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C02:02 : Distance to training data  0.000 (using nearest neighbor HLA-C02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*02:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003990   18.984 0.051928   27.683 28507.69\r\n",
      "   1 HLA-C*02:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0013940   11.647 0.079360   16.415 21186.52\r\n",
      "   1 HLA-C*02:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066430    5.762 0.156876    5.460  9158.29\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*02:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B49:01 -xls -xlsfile test/HLA-B4901_preds.xls\r\n",
      "# Wed Oct 11 17:02:27 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B49:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4901_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B49:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4901_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanp10PR5\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B49:01 : Distance to training data  0.000 (using nearest neighbor HLA-B49:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*49:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0009500   21.225 0.035501   38.550 34052.72\r\n",
      "   1 HLA-B*49:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0003600   31.400 0.028729   47.840 36641.50\r\n",
      "   1 HLA-B*49:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011390   19.602 0.054878   21.963 27612.13\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*49:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B08:01 -xls -xlsfile test/HLA-B0801_preds.xls\r\n",
      "# Wed Oct 11 17:02:28 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B08:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B0801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B08:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B0801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanoBmWhM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B08:01 : Distance to training data  0.000 (using nearest neighbor HLA-B08:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*08:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006800   28.984 0.055256   43.973 27499.43\r\n",
      "   1 HLA-B*08:01       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0567430    2.581 0.236415    3.718  3873.14\r\n",
      "   1 HLA-B*08:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0019580   18.398 0.079813   28.193 21082.94\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*08:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:18 -xls -xlsfile test/HLA-C0718_preds.xls\r\n",
      "# Wed Oct 11 17:02:28 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:18           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0718_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:18           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0718_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanzed4Jb\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:18 : Distance to training data  0.000 (using nearest neighbor HLA-C07:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:18       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000160   43.500 0.028161   53.692 36867.39\r\n",
      "   1 HLA-C*07:18       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000720   26.143 0.050016   29.653 29103.58\r\n",
      "   1 HLA-C*07:18      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002480   15.737 0.085183   13.545 19892.87\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:18. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C05:01 -xls -xlsfile test/HLA-C0501_preds.xls\r\n",
      "# Wed Oct 11 17:02:28 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C05:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C05:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanuNPyGq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C05:01 : Distance to training data  0.000 (using nearest neighbor HLA-C05:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*05:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004530   16.800 0.035616   29.215 34010.39\r\n",
      "   1 HLA-C*05:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0022560    8.382 0.066490   12.920 24352.05\r\n",
      "   1 HLA-C*05:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0025930    7.859 0.102523    6.371 16489.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*05:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:12 -xls -xlsfile test/HLA-B3512_preds.xls\r\n",
      "# Wed Oct 11 17:02:29 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:12           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3512_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:12           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3512_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanFMKoPR\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:12 : Distance to training data  0.000 (using nearest neighbor HLA-B35:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:12       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001140   25.154 0.014758   51.126 42620.90\r\n",
      "   1 HLA-B*35:12       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0041630    5.832 0.071625    8.835 23035.96\r\n",
      "   1 HLA-B*35:12      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0054600    5.124 0.080142    7.493 21008.02\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:12. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A25:01 -xls -xlsfile test/HLA-A2501_preds.xls\r\n",
      "# Wed Oct 11 17:02:29 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A25:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A25:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpant9XDIX\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A25:01 : Distance to training data  0.000 (using nearest neighbor HLA-A25:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*25:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002640   26.767 0.020880   35.881 39889.22\r\n",
      "   1 HLA-A*25:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002760   26.367 0.027666   26.057 37065.38\r\n",
      "   1 HLA-A*25:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0020080   11.998 0.048744   11.931 29506.88\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*25:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A23:01 -xls -xlsfile test/HLA-A2301_preds.xls\r\n",
      "# Wed Oct 11 17:02:29 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A23:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A23:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan3aCTnK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A23:01 : Distance to training data  0.000 (using nearest neighbor HLA-A23:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*23:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000180   47.000 0.022226   55.871 39312.51\r\n",
      "   1 HLA-A*23:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001350   24.333 0.041526   32.093 31903.66\r\n",
      "   1 HLA-A*23:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004330   15.805 0.067376   19.184 24119.71\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*23:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C18:02 -xls -xlsfile test/HLA-C1802_preds.xls\r\n",
      "# Wed Oct 11 17:02:30 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C18:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C18:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanTP0Dfu\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C18:02 : Distance to training data  0.041 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*18:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003910   28.649 0.026580   44.327 37503.47\r\n",
      "   1 HLA-C*18:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0034020   11.416 0.058613   12.847 26518.52\r\n",
      "   1 HLA-C*18:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0022950   13.857 0.054873   14.794 27613.63\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*18:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C18:01 -xls -xlsfile test/HLA-C1801_preds.xls\r\n",
      "# Wed Oct 11 17:02:30 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C18:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C18:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanmF13NU\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C18:01 : Distance to training data  0.041 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*18:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003910   28.649 0.026580   44.327 37503.47\r\n",
      "   1 HLA-C*18:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0034020   11.416 0.058613   12.847 26518.52\r\n",
      "   1 HLA-C*18:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0022950   13.857 0.054873   14.794 27613.63\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*18:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C18:03 -xls -xlsfile test/HLA-C1803_preds.xls\r\n",
      "# Wed Oct 11 17:02:30 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C18:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1803_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C18:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1803_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanagk31M\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C18:03 : Distance to training data  0.126 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*18:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0019700   18.885 0.026696   39.237 37456.41\r\n",
      "   1 HLA-C*18:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0097990    8.108 0.056418   12.680 27155.86\r\n",
      "   1 HLA-C*18:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0043120   13.023 0.048858   16.682 29470.52\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*18:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A33:03 -xls -xlsfile test/HLA-A3303_preds.xls\r\n",
      "# Wed Oct 11 17:02:31 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A33:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3303_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A33:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3303_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanktCNtr\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A33:03 : Distance to training data  0.000 (using nearest neighbor HLA-A33:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*33:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001210   43.375 0.025514   63.347 37938.53\r\n",
      "   1 HLA-A*33:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001510   40.100 0.028763   58.027 36628.03\r\n",
      "   1 HLA-A*33:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007430   22.188 0.052141   32.679 28442.05\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*33:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A33:01 -xls -xlsfile test/HLA-A3301_preds.xls\r\n",
      "# Wed Oct 11 17:02:31 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A33:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A33:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfnvm4m\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A33:01 : Distance to training data  0.000 (using nearest neighbor HLA-A33:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*33:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000800   46.400 0.021132   64.320 39780.61\r\n",
      "   1 HLA-A*33:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000730   48.000 0.022399   61.687 39238.98\r\n",
      "   1 HLA-A*33:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002350   30.579 0.035125   41.221 34191.54\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*33:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A29:11 -xls -xlsfile test/HLA-A2911_preds.xls\r\n",
      "# Wed Oct 11 17:02:31 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A29:11           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2911_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A29:11           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2911_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanio0RVy\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A29:11 : Distance to training data  0.000 (using nearest neighbor HLA-A29:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*29:11       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000330   50.556 0.020711   69.003 39962.22\r\n",
      "   1 HLA-A*29:11       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001570   29.154 0.042137   38.646 31693.44\r\n",
      "   1 HLA-A*29:11      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004410   18.947 0.062239   24.635 25498.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*29:11. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A33:05 -xls -xlsfile test/HLA-A3305_preds.xls\r\n",
      "# Wed Oct 11 17:02:32 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A33:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3305_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A33:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3305_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan78A8On\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A33:05 : Distance to training data  0.000 (using nearest neighbor HLA-A33:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*33:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000800   46.400 0.021132   64.320 39780.61\r\n",
      "   1 HLA-A*33:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000730   48.000 0.022399   61.687 39238.98\r\n",
      "   1 HLA-A*33:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002350   30.579 0.035125   41.221 34191.54\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*33:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C04:07 -xls -xlsfile test/HLA-C0407_preds.xls\r\n",
      "# Wed Oct 11 17:02:32 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C04:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0407_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C04:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0407_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanv4bfAE\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C04:07 : Distance to training data  0.000 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*04:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002380   26.619 0.032121   40.658 35321.11\r\n",
      "   1 HLA-C*04:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0030830    7.516 0.069534    9.875 23563.07\r\n",
      "   1 HLA-C*04:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0016750   10.663 0.065761   11.351 24544.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*04:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A66:01 -xls -xlsfile test/HLA-A6601_preds.xls\r\n",
      "# Wed Oct 11 17:02:32 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A66:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A66:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIG0VvM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A66:01 : Distance to training data  0.000 (using nearest neighbor HLA-A66:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*66:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001750   40.857 0.039070   48.086 32762.81\r\n",
      "   1 HLA-A*66:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0003100   33.292 0.055978   31.582 27285.45\r\n",
      "   1 HLA-A*66:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0019310   14.793 0.091517   14.828 18575.24\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*66:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A66:03 -xls -xlsfile test/HLA-A6603_preds.xls\r\n",
      "# Wed Oct 11 17:02:33 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A66:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6603_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A66:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6603_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanNV6rDx\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A66:03 : Distance to training data  0.057 (using nearest neighbor HLA-A66:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*66:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002640   37.045 0.035449   49.889 34071.88\r\n",
      "   1 HLA-A*66:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004010   32.405 0.052291   34.103 28395.94\r\n",
      "   1 HLA-A*66:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0018050   18.376 0.081010   19.715 20811.65\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*66:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A66:02 -xls -xlsfile test/HLA-A6602_preds.xls\r\n",
      "# Wed Oct 11 17:02:33 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A66:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A6602_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A66:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A6602_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanHgx3UR\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A66:02 : Distance to training data  0.025 (using nearest neighbor HLA-A66:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*66:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003820   36.129 0.045031   47.172 30716.42\r\n",
      "   1 HLA-A*66:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0005270   32.468 0.063285   32.970 25211.33\r\n",
      "   1 HLA-A*66:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0038930   14.395 0.110868   14.973 15066.23\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*66:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B47:03 -xls -xlsfile test/HLA-B4703_preds.xls\r\n",
      "# Wed Oct 11 17:02:33 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B47:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4703_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B47:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4703_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan0Ia5OO\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B47:03 : Distance to training data  0.053 (using nearest neighbor HLA-B47:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*47:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006200   29.167 0.041143   55.063 32036.15\r\n",
      "   1 HLA-B*47:03       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0043410   12.418 0.092618   19.003 18355.27\r\n",
      "   1 HLA-B*47:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0045820   12.053 0.126341   10.670 12743.76\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*47:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:06 -xls -xlsfile test/HLA-C0706_preds.xls\r\n",
      "# Wed Oct 11 17:02:34 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0706_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0706_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanpCYqx7\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:06 : Distance to training data  0.000 (using nearest neighbor HLA-C07:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000160   43.500 0.028161   53.692 36867.39\r\n",
      "   1 HLA-C*07:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000720   26.143 0.050016   29.653 29103.58\r\n",
      "   1 HLA-C*07:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002480   15.737 0.085183   13.545 19892.87\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B13:02 -xls -xlsfile test/HLA-B1302_preds.xls\r\n",
      "# Wed Oct 11 17:02:34 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B13:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1302_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B13:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1302_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpant3HoeK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B13:02 : Distance to training data  0.000 (using nearest neighbor HLA-B13:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*13:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0508920    4.814 0.080392   12.110 20951.27\r\n",
      "   1 HLA-B*13:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0189580    9.569 0.053253   22.977 28101.91\r\n",
      "   1 HLA-B*13:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0638900    4.014 0.098646    8.269 17196.32\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*13:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B13:03 -xls -xlsfile test/HLA-B1303_preds.xls\r\n",
      "# Wed Oct 11 17:02:34 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B13:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1303_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B13:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1303_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan1xoAkD\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B13:03 : Distance to training data  0.037 (using nearest neighbor HLA-B13:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*13:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0428370    6.283 0.072197   14.040 22893.83\r\n",
      "   1 HLA-B*13:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0237320    9.592 0.053598   22.336 27997.21\r\n",
      "   1 HLA-B*13:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0956840    3.063 0.107012    6.363 15708.10\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*13:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C14:02 -xls -xlsfile test/HLA-C1402_preds.xls\r\n",
      "# Wed Oct 11 17:02:35 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C14:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C14:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpannKW4B4\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C14:02 : Distance to training data  0.000 (using nearest neighbor HLA-C14:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*14:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001230   30.417 0.044091   53.151 31030.43\r\n",
      "   1 HLA-C*14:02       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0061250    6.338 0.184816    9.388  6769.01\r\n",
      "   1 HLA-C*14:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0010620   13.938 0.143180   14.180 10621.15\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*14:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C14:03 -xls -xlsfile test/HLA-C1403_preds.xls\r\n",
      "# Wed Oct 11 17:02:35 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C14:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C14:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan6Qe8v2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C14:03 : Distance to training data  0.000 (using nearest neighbor HLA-C14:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*14:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001230   30.417 0.044091   53.151 31030.43\r\n",
      "   1 HLA-C*14:03       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0061250    6.338 0.184816    9.388  6769.01\r\n",
      "   1 HLA-C*14:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0010620   13.938 0.143180   14.180 10621.15\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*14:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B81:03 -xls -xlsfile test/HLA-B8103_preds.xls\r\n",
      "# Wed Oct 11 17:02:35 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B81:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B8103_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B81:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B8103_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan859SHX\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B81:03 : Distance to training data  0.146 (using nearest neighbor HLA-B42:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*81:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000430   42.333 0.013819   54.265 43056.13\r\n",
      "   1 HLA-B*81:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0022080    8.791 0.057412   10.482 26865.36\r\n",
      "   1 HLA-B*81:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0074240    4.820 0.092319    4.945 18414.75\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*81:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B81:01 -xls -xlsfile test/HLA-B8101_preds.xls\r\n",
      "# Wed Oct 11 17:02:36 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B81:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B8101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B81:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B8101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCuDEGH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B81:01 : Distance to training data  0.146 (using nearest neighbor HLA-B42:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*81:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000430   42.333 0.013819   54.265 43056.13\r\n",
      "   1 HLA-B*81:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0022080    8.791 0.057412   10.482 26865.36\r\n",
      "   1 HLA-B*81:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0074240    4.820 0.092319    4.945 18414.75\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*81:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:01 -xls -xlsfile test/HLA-B4001_preds.xls\r\n",
      "# Wed Oct 11 17:02:36 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4001_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4001_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanRbDEfK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:01 : Distance to training data  0.000 (using nearest neighbor HLA-B40:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000960   24.909 0.023777   45.318 38658.28\r\n",
      "   1 HLA-B*40:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002210   17.800 0.047422   19.654 29931.98\r\n",
      "   1 HLA-B*40:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0013940    8.131 0.101650    6.439 16646.38\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:02 -xls -xlsfile test/HLA-B4002_preds.xls\r\n",
      "# Wed Oct 11 17:02:36 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4002_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4002_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanu72XSv\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:02 : Distance to training data  0.000 (using nearest neighbor HLA-B40:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006610   15.755 0.043767   33.693 31139.40\r\n",
      "   1 HLA-B*40:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0005810   16.582 0.050993   28.001 28797.54\r\n",
      "   1 HLA-B*40:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009850   13.449 0.086786   14.016 19550.82\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:08 -xls -xlsfile test/HLA-B4008_preds.xls\r\n",
      "# Wed Oct 11 17:02:37 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4008_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4008_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanYjwNCZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:08 : Distance to training data  0.063 (using nearest neighbor HLA-B40:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0015540   20.702 0.043652   36.867 31178.17\r\n",
      "   1 HLA-B*40:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0173520    5.396 0.114003    6.742 14563.75\r\n",
      "   1 HLA-B*40:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0134400    6.436 0.105575    8.064 15954.24\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A03:05 -xls -xlsfile test/HLA-A0305_preds.xls\r\n",
      "# Wed Oct 11 17:02:37 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A03:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0305_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A03:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0305_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanE9953a\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A03:05 : Distance to training data  0.000 (using nearest neighbor HLA-A03:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*03:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000550   42.500 0.030821   55.540 35821.43\r\n",
      "   1 HLA-A*03:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000590   41.333 0.037867   45.260 33192.05\r\n",
      "   1 HLA-A*03:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004460   19.123 0.061950   24.815 25578.14\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*03:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:43 -xls -xlsfile test/HLA-B3543_preds.xls\r\n",
      "# Wed Oct 11 17:02:37 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:43           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3543_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:43           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3543_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpankKMyxn\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:43 : Distance to training data  0.047 (using nearest neighbor HLA-B15:11)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:43       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001070   39.250 0.019133   69.214 40650.39\r\n",
      "   1 HLA-B*35:43       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0023560   11.954 0.075049   14.818 22198.17\r\n",
      "   1 HLA-B*35:43      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0022130   12.307 0.068624   16.848 23796.21\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:43. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A03:01 -xls -xlsfile test/HLA-A0301_preds.xls\r\n",
      "# Wed Oct 11 17:02:38 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A03:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A03:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanet8AV5\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A03:01 : Distance to training data  0.000 (using nearest neighbor HLA-A03:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*03:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000550   42.500 0.030821   55.540 35821.43\r\n",
      "   1 HLA-A*03:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000590   41.333 0.037867   45.260 33192.05\r\n",
      "   1 HLA-A*03:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004460   19.123 0.061950   24.815 25578.14\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*03:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A03:02 -xls -xlsfile test/HLA-A0302_preds.xls\r\n",
      "# Wed Oct 11 17:02:38 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A03:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0302_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A03:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0302_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanZeAsJW\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A03:02 : Distance to training data  0.000 (using nearest neighbor HLA-A03:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*03:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001000   39.429 0.030115   54.347 36096.12\r\n",
      "   1 HLA-A*03:02       TLIVNSVL TLIVNSVL-  0  0  0  8  1     TLIVNSVL         PEPLIST 0.0000810   42.400 0.033320   49.589 34865.85\r\n",
      "   1 HLA-A*03:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007760   18.878 0.065683   23.142 24565.61\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*03:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B45:07 -xls -xlsfile test/HLA-B4507_preds.xls\r\n",
      "# Wed Oct 11 17:02:38 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B45:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4507_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B45:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4507_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpannhkgd9\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B45:07 : Distance to training data  0.000 (using nearest neighbor HLA-B45:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*45:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000260   54.375 0.018429   67.136 40961.21\r\n",
      "   1 HLA-B*45:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000270   53.750 0.017026   71.385 41587.74\r\n",
      "   1 HLA-B*45:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0000320   50.625 0.020585   60.921 40016.74\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*45:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:07 -xls -xlsfile test/HLA-B5107_preds.xls\r\n",
      "# Wed Oct 11 17:02:39 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5107_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5107_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanrlxv91\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:07 : Distance to training data  0.027 (using nearest neighbor HLA-B52:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0294160    5.016 0.061205   13.909 25785.13\r\n",
      "   1 HLA-B*51:07       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0221320    6.393 0.062575   13.317 25405.75\r\n",
      "   1 HLA-B*51:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0117980   10.031 0.049510   20.347 29263.35\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:06 -xls -xlsfile test/HLA-B3906_preds.xls\r\n",
      "# Wed Oct 11 17:02:39 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3906_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3906_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanbp1gtl\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:06 : Distance to training data  0.000 (using nearest neighbor HLA-B39:06)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005820   32.511 0.029642   46.588 36281.33\r\n",
      "   1 HLA-B*39:06       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0044600   13.472 0.047925   26.950 29769.52\r\n",
      "   1 HLA-B*39:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0024580   17.869 0.039922   33.733 32462.18\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:05 -xls -xlsfile test/HLA-B5105_preds.xls\r\n",
      "# Wed Oct 11 17:02:40 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5105_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5105_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanoboJcg\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:05 : Distance to training data  0.063 (using nearest neighbor HLA-B51:08)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0040690   13.252 0.043613   19.305 31191.32\r\n",
      "   1 HLA-B*51:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0057750   11.061 0.037308   24.054 33393.42\r\n",
      "   1 HLA-B*51:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0051710   11.726 0.051156   15.145 28746.80\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B38:01 -xls -xlsfile test/HLA-B3801_preds.xls\r\n",
      "# Wed Oct 11 17:02:40 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B38:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B38:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpansAXjw2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B38:01 : Distance to training data  0.000 (using nearest neighbor HLA-B38:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*38:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000540   45.000 0.014702   62.196 42646.72\r\n",
      "   1 HLA-B*38:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0010900   16.051 0.044743   21.096 30812.29\r\n",
      "   1 HLA-B*38:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0010390   16.378 0.051328   17.570 28693.35\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*38:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B45:01 -xls -xlsfile test/HLA-B4501_preds.xls\r\n",
      "# Wed Oct 11 17:02:40 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B45:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B45:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan1cFz66\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B45:01 : Distance to training data  0.000 (using nearest neighbor HLA-B45:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*45:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000260   54.375 0.018429   67.136 40961.21\r\n",
      "   1 HLA-B*45:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000270   53.750 0.017026   71.385 41587.74\r\n",
      "   1 HLA-B*45:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0000320   50.625 0.020585   60.921 40016.74\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*45:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:01 -xls -xlsfile test/HLA-B5101_preds.xls\r\n",
      "# Wed Oct 11 17:02:41 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanGJQk3i\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:01 : Distance to training data  0.000 (using nearest neighbor HLA-B51:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0140300    7.059 0.058525   12.869 26543.77\r\n",
      "   1 HLA-B*51:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0055990   11.545 0.047131   17.853 30026.37\r\n",
      "   1 HLA-B*51:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0032720   14.923 0.040311   22.178 32325.85\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C16:04 -xls -xlsfile test/HLA-C1604_preds.xls\r\n",
      "# Wed Oct 11 17:02:41 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C16:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1604_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C16:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1604_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanPBIjko\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C16:04 : Distance to training data  0.048 (using nearest neighbor HLA-C12:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*16:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006930   12.289 0.070731   26.072 23259.87\r\n",
      "   1 HLA-C*16:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0012200    9.685 0.114711   14.484 14452.62\r\n",
      "   1 HLA-C*16:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0057370    4.524 0.212752    5.187  5003.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*16:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C16:01 -xls -xlsfile test/HLA-C1601_preds.xls\r\n",
      "# Wed Oct 11 17:02:41 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C16:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C16:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpansJxhlU\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C16:01 : Distance to training data  0.000 (using nearest neighbor HLA-C16:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*16:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0010380   13.681 0.086265   24.324 19661.36\r\n",
      "   1 HLA-C*16:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0030740    8.977 0.158705   11.686  8978.83\r\n",
      "   1 HLA-C*16:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0094620    5.531 0.284210    4.343  2309.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*16:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C02:29 -xls -xlsfile test/HLA-C0229_preds.xls\r\n",
      "# Wed Oct 11 17:02:42 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C02:29           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0229_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C02:29           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0229_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCGgMA0\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C02:29 : Distance to training data  0.000 (using nearest neighbor HLA-C02:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*02:29       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003990   18.984 0.051928   27.683 28507.69\r\n",
      "   1 HLA-C*02:29       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0013940   11.647 0.079360   16.415 21186.52\r\n",
      "   1 HLA-C*02:29      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066430    5.762 0.156876    5.460  9158.29\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*02:29. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:09 -xls -xlsfile test/HLA-B5109_preds.xls\r\n",
      "# Wed Oct 11 17:02:42 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5109_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5109_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan2EmFdN\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:09 : Distance to training data  0.032 (using nearest neighbor HLA-B51:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0091950    7.844 0.050442   14.166 28969.74\r\n",
      "   1 HLA-B*51:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0044860   11.450 0.040736   19.428 32177.53\r\n",
      "   1 HLA-B*51:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0035630   12.769 0.038452   21.029 32982.61\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B82:02 -xls -xlsfile test/HLA-B8202_preds.xls\r\n",
      "# Wed Oct 11 17:02:42 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B82:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B8202_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B82:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B8202_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanbbk3Id\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B82:02 : Distance to training data  0.104 (using nearest neighbor HLA-B83:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*82:02       GTLIVNSV GTLIV-NSV  0  0  0  5  1     GTLIVNSV         PEPLIST 0.0005370   35.326 0.027076   54.526 37302.73\r\n",
      "   1 HLA-B*82:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0084260   10.224 0.078877   12.115 21297.54\r\n",
      "   1 HLA-B*82:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0151860    6.998 0.095205    8.262 17848.62\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*82:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:27 -xls -xlsfile test/HLA-B4427_preds.xls\r\n",
      "# Wed Oct 11 17:02:43 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:27           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4427_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:27           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4427_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanWv4UJy\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:27 : Distance to training data  0.000 (using nearest neighbor HLA-B44:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:27       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000340   43.000 0.023897   67.179 38608.13\r\n",
      "   1 HLA-B*44:27       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000640   33.800 0.029461   53.843 36352.44\r\n",
      "   1 HLA-B*44:27      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001250   25.583 0.040822   34.347 32147.60\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:27. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B44:10 -xls -xlsfile test/HLA-B4410_preds.xls\r\n",
      "# Wed Oct 11 17:02:43 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B44:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4410_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B44:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4410_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpankUNrkx\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B44:10 : Distance to training data  0.094 (using nearest neighbor HLA-B44:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*44:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000650   38.667 0.014152   60.257 42901.28\r\n",
      "   1 HLA-B*44:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002000   26.818 0.018932   46.461 40738.89\r\n",
      "   1 HLA-B*44:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004220   20.418 0.034466   22.848 34436.20\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*44:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B58:01 -xls -xlsfile test/HLA-B5801_preds.xls\r\n",
      "# Wed Oct 11 17:02:43 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B58:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5801_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B58:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5801_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpangqE36C\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B58:01 : Distance to training data  0.000 (using nearest neighbor HLA-B58:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*58:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0011300   17.379 0.075273   30.796 22144.43\r\n",
      "   1 HLA-B*58:01       TLIVNSVL -TLIVNSVL  0  0  0  0  1     TLIVNSVL         PEPLIST 0.0002980   30.880 0.071408   32.761 23090.10\r\n",
      "   1 HLA-B*58:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0380370    3.278 0.283720    3.709  2321.55\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*58:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B58:02 -xls -xlsfile test/HLA-B5802_preds.xls\r\n",
      "# Wed Oct 11 17:02:44 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B58:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B58:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanxO4pbM\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B58:02 : Distance to training data  0.067 (using nearest neighbor HLA-B58:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*58:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001520   16.700 0.021187   35.966 39756.93\r\n",
      "   1 HLA-B*58:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000230   39.000 0.017910   44.658 41191.86\r\n",
      "   1 HLA-B*58:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0015710    4.761 0.072427    4.204 22836.94\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*58:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:16 -xls -xlsfile test/HLA-B4016_preds.xls\r\n",
      "# Wed Oct 11 17:02:44 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:16           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4016_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:16           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4016_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanFQgweh\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:16 : Distance to training data  0.066 (using nearest neighbor HLA-B40:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:16       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0008310   25.224 0.060206   39.208 26065.37\r\n",
      "   1 HLA-B*40:16       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0061510   10.531 0.101980   18.808 16587.05\r\n",
      "   1 HLA-B*40:16      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0046730   12.066 0.135610   11.518 11527.70\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:16. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B55:01 -xls -xlsfile test/HLA-B5501_preds.xls\r\n",
      "# Wed Oct 11 17:02:44 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B55:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B55:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanN8N5GQ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B55:01 : Distance to training data  0.000 (using nearest neighbor HLA-B55:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*55:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007490   28.259 0.023467   42.178 38788.17\r\n",
      "   1 HLA-B*55:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0010820   23.560 0.026933   35.724 37360.49\r\n",
      "   1 HLA-B*55:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0020240   16.956 0.037866   22.489 33192.41\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*55:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:12 -xls -xlsfile test/HLA-B4012_preds.xls\r\n",
      "# Wed Oct 11 17:02:45 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:12           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4012_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:12           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4012_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan4pZRwm\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:12 : Distance to training data  0.030 (using nearest neighbor HLA-B48:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:12       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000890   26.700 0.021932   42.378 39437.75\r\n",
      "   1 HLA-B*40:12       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0049430    4.603 0.101284    5.416 16712.42\r\n",
      "   1 HLA-B*40:12      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066890    3.931 0.143261    2.758 10611.84\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:12. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B37:01 -xls -xlsfile test/HLA-B3701_preds.xls\r\n",
      "# Wed Oct 11 17:02:45 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B37:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B37:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCMgfF6\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B37:01 : Distance to training data  0.000 (using nearest neighbor HLA-B37:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*37:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005480   24.391 0.042763   35.523 31479.51\r\n",
      "   1 HLA-B*37:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0021890   13.884 0.068855   18.972 23736.82\r\n",
      "   1 HLA-B*37:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0018830   14.846 0.084739   13.578 19988.68\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*37:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:10 -xls -xlsfile test/HLA-B3910_preds.xls\r\n",
      "# Wed Oct 11 17:02:46 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3910_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3910_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan2HDeSI\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:10 : Distance to training data  0.050 (using nearest neighbor HLA-B39:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006380   23.646 0.044987   40.132 30731.04\r\n",
      "   1 HLA-B*39:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0293290    4.328 0.185184    5.990  6742.11\r\n",
      "   1 HLA-B*39:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0154720    6.219 0.193197    5.524  6182.20\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C06:02 -xls -xlsfile test/HLA-C0602_preds.xls\r\n",
      "# Wed Oct 11 17:02:46 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C06:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0602_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C06:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0602_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanchqPaQ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C06:02 : Distance to training data  0.000 (using nearest neighbor HLA-C06:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*06:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000880   31.889 0.027173   45.843 37263.61\r\n",
      "   1 HLA-C*06:02       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0004660   17.306 0.042069   29.209 31716.77\r\n",
      "   1 HLA-C*06:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004730   17.194 0.053123   21.768 28141.46\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*06:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:32 -xls -xlsfile test/HLA-B3532_preds.xls\r\n",
      "# Wed Oct 11 17:02:47 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:32           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3532_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:32           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3532_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanD2tbFS\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:32 : Distance to training data  0.010 (using nearest neighbor HLA-B35:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:32       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000370   44.333 0.012562   78.770 43645.72\r\n",
      "   1 HLA-B*35:32       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0011740   13.636 0.047640   23.487 29861.46\r\n",
      "   1 HLA-B*35:32      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007970   15.738 0.050212   22.090 29041.91\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:32. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A01:02 -xls -xlsfile test/HLA-A0102_preds.xls\r\n",
      "# Wed Oct 11 17:02:47 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A01:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0102_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A01:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0102_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanETfQJ6\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A01:02 : Distance to training data  0.037 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*01:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005830   49.727 0.049228   54.764 29352.77\r\n",
      "   1 HLA-A*01:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0009080   41.778 0.048343   55.963 29635.19\r\n",
      "   1 HLA-A*01:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0034880   22.077 0.078798   25.634 21315.75\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*01:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A01:03 -xls -xlsfile test/HLA-A0103_preds.xls\r\n",
      "# Wed Oct 11 17:02:47 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A01:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0103_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A01:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0103_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan3lRRol\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A01:03 : Distance to training data  0.018 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*01:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002040   32.333 0.029971   44.048 36152.40\r\n",
      "   1 HLA-A*01:03       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0003060   26.095 0.031779   40.849 35452.06\r\n",
      "   1 HLA-A*01:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007050   16.319 0.052145   18.650 28440.83\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*01:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A01:01 -xls -xlsfile test/HLA-A0101_preds.xls\r\n",
      "# Wed Oct 11 17:02:48 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A01:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A01:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanyflfNH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A01:01 : Distance to training data  0.000 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*01:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003720   32.364 0.033504   44.808 34796.51\r\n",
      "   1 HLA-A*01:01       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0004590   29.069 0.033539   44.748 34783.34\r\n",
      "   1 HLA-A*01:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011040   17.636 0.052704   22.147 28269.34\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*01:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A30:04 -xls -xlsfile test/HLA-A3004_preds.xls\r\n",
      "# Wed Oct 11 17:02:48 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A30:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3004_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A30:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3004_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanfFTeZK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A30:04 : Distance to training data  0.086 (using nearest neighbor HLA-A30:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*30:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0015790   32.793 0.091842   45.011 18510.04\r\n",
      "   1 HLA-A*30:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0011890   36.656 0.092484   44.630 18381.90\r\n",
      "   1 HLA-A*30:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0145360   10.675 0.206786    9.088  5336.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*30:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A01:09 -xls -xlsfile test/HLA-A0109_preds.xls\r\n",
      "# Wed Oct 11 17:02:48 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A01:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0109_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A01:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0109_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan4wqvQq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A01:09 : Distance to training data  0.000 (using nearest neighbor HLA-A01:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*01:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003720   32.364 0.033504   44.808 34796.51\r\n",
      "   1 HLA-A*01:09       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0004590   29.069 0.033539   44.748 34783.34\r\n",
      "   1 HLA-A*01:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011040   17.636 0.052704   22.147 28269.34\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*01:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A30:02 -xls -xlsfile test/HLA-A3002_preds.xls\r\n",
      "# Wed Oct 11 17:02:49 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A30:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3002_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A30:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3002_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIMlkET\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A30:02 : Distance to training data  0.000 (using nearest neighbor HLA-A30:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*30:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007200   35.458 0.078370   46.243 21414.69\r\n",
      "   1 HLA-A*30:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0007270   35.312 0.069222   53.766 23642.75\r\n",
      "   1 HLA-A*30:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0044960   13.877 0.159550   12.305  8897.12\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*30:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A30:01 -xls -xlsfile test/HLA-A3001_preds.xls\r\n",
      "# Wed Oct 11 17:02:49 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A30:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3001_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A30:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3001_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanPJedOO\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A30:01 : Distance to training data  0.000 (using nearest neighbor HLA-A30:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*30:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0021460   24.110 0.123681   30.397 13115.86\r\n",
      "   1 HLA-A*30:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0002210   57.500 0.059141   64.361 26367.46\r\n",
      "   1 HLA-A*30:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0214690    6.747 0.279486    7.008  2430.38\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*30:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C16:02 -xls -xlsfile test/HLA-C1602_preds.xls\r\n",
      "# Wed Oct 11 17:02:49 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C16:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C1602_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C16:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C1602_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanvZC9CZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C16:02 : Distance to training data  0.047 (using nearest neighbor HLA-C16:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*16:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0017070   12.327 0.090040   19.163 18874.47\r\n",
      "   1 HLA-C*16:02       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0028800    9.904 0.107488   15.334 15627.42\r\n",
      "   1 HLA-C*16:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0126030    5.045 0.242507    3.867  3626.08\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*16:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C04:03 -xls -xlsfile test/HLA-C0403_preds.xls\r\n",
      "# Wed Oct 11 17:02:50 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C04:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C04:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan1orJKZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C04:03 : Distance to training data  0.038 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*04:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005930   17.235 0.033831   28.702 34673.62\r\n",
      "   1 HLA-C*04:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0048240    6.829 0.072677    9.417 22775.24\r\n",
      "   1 HLA-C*04:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0037670    7.672 0.094706    5.683 17945.24\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*04:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C04:01 -xls -xlsfile test/HLA-C0401_preds.xls\r\n",
      "# Wed Oct 11 17:02:50 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C04:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0401_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C04:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0401_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpannufRRI\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C04:01 : Distance to training data  0.000 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*04:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002380   26.619 0.032121   40.658 35321.11\r\n",
      "   1 HLA-C*04:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0030830    7.516 0.069534    9.875 23563.07\r\n",
      "   1 HLA-C*04:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0016750   10.663 0.065761   11.351 24544.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*04:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B07:02 -xls -xlsfile test/HLA-B0702_preds.xls\r\n",
      "# Wed Oct 11 17:02:50 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B07:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B0702_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B07:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B0702_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanvMc1HE\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B07:02 : Distance to training data  0.000 (using nearest neighbor HLA-B07:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*07:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001570   32.846 0.029043   53.248 36517.22\r\n",
      "   1 HLA-B*07:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0015680   12.338 0.084098   14.654 20127.79\r\n",
      "   1 HLA-B*07:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0069170    6.112 0.129769    7.744 12279.75\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B07:05 -xls -xlsfile test/HLA-B0705_preds.xls\r\n",
      "# Wed Oct 11 17:02:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B07:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B0705_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B07:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B0705_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanliSSMr\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B07:05 : Distance to training data  0.025 (using nearest neighbor HLA-B07:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*07:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002570   34.739 0.039867   51.609 32481.51\r\n",
      "   1 HLA-B*07:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0041320   11.372 0.119068   13.928 13787.10\r\n",
      "   1 HLA-B*07:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0188140    5.541 0.186904    6.921  6617.80\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*07:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C04:04 -xls -xlsfile test/HLA-C0404_preds.xls\r\n",
      "# Wed Oct 11 17:02:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C04:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0404_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C04:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0404_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXnY3S7\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C04:04 : Distance to training data  0.034 (using nearest neighbor HLA-C04:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*04:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003850   21.545 0.034906   34.479 34272.64\r\n",
      "   1 HLA-C*04:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0039270    6.943 0.071387   10.160 23095.36\r\n",
      "   1 HLA-C*04:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0019760   10.087 0.071370   10.166 23099.60\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*04:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A24:03 -xls -xlsfile test/HLA-A2403_preds.xls\r\n",
      "# Wed Oct 11 17:02:51 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A24:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2403_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A24:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2403_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanzghtss\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A24:03 : Distance to training data  0.000 (using nearest neighbor HLA-A24:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*24:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000180   50.833 0.025046   53.558 38131.13\r\n",
      "   1 HLA-A*24:03       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0002510   23.750 0.047015   31.499 30064.08\r\n",
      "   1 HLA-A*24:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0014720   12.908 0.086572   17.049 19596.15\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*24:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A24:02 -xls -xlsfile test/HLA-A2402_preds.xls\r\n",
      "# Wed Oct 11 17:02:52 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A24:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2402_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A24:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2402_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanueT3Hm\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A24:02 : Distance to training data  0.000 (using nearest neighbor HLA-A24:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*24:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000130   50.000 0.017896   56.828 41198.11\r\n",
      "   1 HLA-A*24:02       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0001120   25.250 0.030122   35.684 36093.39\r\n",
      "   1 HLA-A*24:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0003750   16.127 0.051032   20.433 28785.39\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*24:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:04 -xls -xlsfile test/HLA-C0704_preds.xls\r\n",
      "# Wed Oct 11 17:02:52 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0704_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0704_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan8kJcDH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:04 : Distance to training data  0.140 (using nearest neighbor HLA-C07:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001730   22.619 0.031679   32.695 35490.45\r\n",
      "   1 HLA-C*07:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0014120    9.034 0.064305   11.738 24934.62\r\n",
      "   1 HLA-C*07:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005710   13.948 0.059754   13.386 26193.15\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:05 -xls -xlsfile test/HLA-C0705_preds.xls\r\n",
      "# Wed Oct 11 17:02:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0705_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0705_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanQNDTJ2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:05 : Distance to training data  0.059 (using nearest neighbor HLA-C07:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000660   42.500 0.034408   56.305 34457.83\r\n",
      "   1 HLA-C*07:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0008340   17.810 0.084692   21.146 19998.84\r\n",
      "   1 HLA-C*07:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009120   17.165 0.101392   16.006 16692.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:02 -xls -xlsfile test/HLA-C0702_preds.xls\r\n",
      "# Wed Oct 11 17:02:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0702_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0702_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanjvXEXF\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:02 : Distance to training data  0.000 (using nearest neighbor HLA-C07:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000090   53.333 0.017635   71.893 41314.62\r\n",
      "   1 HLA-C*07:02       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0002850   16.932 0.069960   18.821 23454.71\r\n",
      "   1 HLA-C*07:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0001560   21.526 0.062424   22.092 25447.29\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:41 -xls -xlsfile test/HLA-B3541_preds.xls\r\n",
      "# Wed Oct 11 17:02:53 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:41           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3541_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:41           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3541_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanLmx07c\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:41 : Distance to training data  0.051 (using nearest neighbor HLA-B35:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:41       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000490   46.750 0.014142   82.281 42905.93\r\n",
      "   1 HLA-B*35:41       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0018400   14.342 0.061369   24.691 25739.43\r\n",
      "   1 HLA-B*35:41      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0010060   17.744 0.066257   22.707 24413.52\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:41. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:08 -xls -xlsfile test/HLA-B1508_preds.xls\r\n",
      "# Wed Oct 11 17:02:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:08           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1508_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:08           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1508_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpannfF5bF\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:08 : Distance to training data  0.017 (using nearest neighbor HLA-B15:11)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:08       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001350   36.700 0.017702   62.721 41284.68\r\n",
      "   1 HLA-B*15:08       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0028490   10.755 0.063756   12.991 25083.18\r\n",
      "   1 HLA-B*15:08      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0022940   11.851 0.059015   14.595 26403.43\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:08. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C07:01 -xls -xlsfile test/HLA-C0701_preds.xls\r\n",
      "# Wed Oct 11 17:02:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C07:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C07:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCItSi0\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C07:01 : Distance to training data  0.000 (using nearest neighbor HLA-C07:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*07:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000160   43.500 0.028161   53.692 36867.39\r\n",
      "   1 HLA-C*07:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000720   26.143 0.050016   29.653 29103.58\r\n",
      "   1 HLA-C*07:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002480   15.737 0.085183   13.545 19892.87\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*07:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:05 -xls -xlsfile test/HLA-A0205_preds.xls\r\n",
      "# Wed Oct 11 17:02:54 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0205_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0205_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanp9hVki\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:05 : Distance to training data  0.000 (using nearest neighbor HLA-A02:05)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0084770   10.648 0.160066   20.841  8847.58\r\n",
      "   1 HLA-A*02:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0081480   10.854 0.163930   20.211  8485.31\r\n",
      "   1 HLA-A*02:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0300160    5.192 0.290021    7.857  2168.55\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:24 -xls -xlsfile test/HLA-B1524_preds.xls\r\n",
      "# Wed Oct 11 17:02:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:24           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1524_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:24           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1524_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan97jsXG\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:24 : Distance to training data  0.085 (using nearest neighbor HLA-B15:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:24       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007340   26.391 0.055057   38.992 27558.71\r\n",
      "   1 HLA-B*15:24       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0043730   12.184 0.102098   16.956 16565.88\r\n",
      "   1 HLA-B*15:24      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0123030    7.154 0.152966    8.060  9554.04\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:24. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:03 -xls -xlsfile test/HLA-B1503_preds.xls\r\n",
      "# Wed Oct 11 17:02:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1503_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1503_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan298KNK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:03 : Distance to training data  0.000 (using nearest neighbor HLA-B15:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001050   42.778 0.051316   60.631 28697.07\r\n",
      "   1 HLA-B*15:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0051880   11.379 0.202636   14.604  5582.00\r\n",
      "   1 HLA-B*15:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0041600   12.430 0.236267   11.567  3879.35\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:01 -xls -xlsfile test/HLA-B1501_preds.xls\r\n",
      "# Wed Oct 11 17:02:55 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1501_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1501_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXYZy0l\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:01 : Distance to training data  0.000 (using nearest neighbor HLA-B15:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002720   30.167 0.052236   47.358 28412.83\r\n",
      "   1 HLA-B*15:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0082760    7.874 0.186508    9.604  6646.22\r\n",
      "   1 HLA-B*15:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0060780    8.935 0.196247    8.807  5981.52\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A31:01 -xls -xlsfile test/HLA-A3101_preds.xls\r\n",
      "# Wed Oct 11 17:02:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A31:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3101_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A31:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3101_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan5rM3Eu\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A31:01 : Distance to training data  0.000 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*31:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001250   39.222 0.043008   52.231 31396.18\r\n",
      "   1 HLA-A*31:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000570   50.588 0.032376   64.485 35223.80\r\n",
      "   1 HLA-A*31:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0008880   19.129 0.102695   20.475 16459.22\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*31:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A31:04 -xls -xlsfile test/HLA-A3104_preds.xls\r\n",
      "# Wed Oct 11 17:02:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A31:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3104_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A31:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3104_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan0Mh1qT\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A31:04 : Distance to training data  0.039 (using nearest neighbor HLA-A31:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*31:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002020   47.429 0.043606   62.643 31193.69\r\n",
      "   1 HLA-A*31:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001330   53.673 0.041477   64.714 31920.58\r\n",
      "   1 HLA-A*31:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0018580   23.483 0.096415   30.125 17616.47\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*31:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A11:02 -xls -xlsfile test/HLA-A1102_preds.xls\r\n",
      "# Wed Oct 11 17:02:56 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A11:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A1102_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A11:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A1102_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanJDgmE2\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A11:02 : Distance to training data  0.000 (using nearest neighbor HLA-A11:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*11:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000550   31.600 0.034523   44.642 34414.97\r\n",
      "   1 HLA-A*11:02       TLIVNSVL TLIVNSVL-  0  0  0  8  1     TLIVNSVL         PEPLIST 0.0000100   56.667 0.025691   59.191 37865.95\r\n",
      "   1 HLA-A*11:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0003120   17.060 0.073111   18.644 22668.54\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*11:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:04 -xls -xlsfile test/HLA-B2704_preds.xls\r\n",
      "# Wed Oct 11 17:02:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2704_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2704_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanS6QJEw\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:04 : Distance to training data  0.000 (using nearest neighbor HLA-B27:04)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:04       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000320   47.667 0.009611   68.805 45061.76\r\n",
      "   1 HLA-B*27:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0000470   42.333 0.013053   52.811 43414.45\r\n",
      "   1 HLA-B*27:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0004450   18.190 0.023035   25.833 38969.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:11 -xls -xlsfile test/HLA-B1511_preds.xls\r\n",
      "# Wed Oct 11 17:02:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:11           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1511_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:11           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1511_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIvd2v9\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:11 : Distance to training data  0.000 (using nearest neighbor HLA-B15:11)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:11       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000770   41.833 0.018080   62.165 41116.17\r\n",
      "   1 HLA-B*15:11       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0024550   10.106 0.062356   12.999 25466.02\r\n",
      "   1 HLA-B*15:11      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0012830   13.608 0.056973   14.989 26993.28\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:11. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:12 -xls -xlsfile test/HLA-B1512_preds.xls\r\n",
      "# Wed Oct 11 17:02:57 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:12           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1512_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:12           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1512_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanucnVbW\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:12 : Distance to training data  0.053 (using nearest neighbor HLA-B15:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:12       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003390   31.690 0.044133   48.825 31016.32\r\n",
      "   1 HLA-B*15:12       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0047040   10.593 0.123620   11.883 13124.51\r\n",
      "   1 HLA-B*15:12      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0042580   11.039 0.132974   10.440 11861.21\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:12. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B56:04 -xls -xlsfile test/HLA-B5604_preds.xls\r\n",
      "# Wed Oct 11 17:02:58 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B56:04           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5604_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B56:04           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5604_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanSKwRaS\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B56:04 : Distance to training data  0.079 (using nearest neighbor HLA-B56:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*56:04       GTLIVNSV GTLIV-NSV  0  0  0  5  1     GTLIVNSV         PEPLIST 0.0002100   30.913 0.020954   51.753 39857.29\r\n",
      "   1 HLA-B*56:04       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0039560   10.089 0.063879   16.137 25049.81\r\n",
      "   1 HLA-B*56:04      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0103400    6.491 0.102662    8.365 16465.11\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*56:04. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C03:17 -xls -xlsfile test/HLA-C0317_preds.xls\r\n",
      "# Wed Oct 11 17:02:58 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C03:17           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0317_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C03:17           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0317_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanmaz7AU\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C03:17 : Distance to training data  0.065 (using nearest neighbor HLA-C03:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*03:17       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001680   16.484 0.038950   25.867 32805.38\r\n",
      "   1 HLA-C*03:17       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0079770    3.514 0.133581    6.158 11783.57\r\n",
      "   1 HLA-C*03:17      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0156310    2.415 0.216251    2.625  4817.40\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*03:17. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A34:01 -xls -xlsfile test/HLA-A3401_preds.xls\r\n",
      "# Wed Oct 11 17:02:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A34:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A3401_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A34:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A3401_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanqkaMUK\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A34:01 : Distance to training data  0.057 (using nearest neighbor HLA-A66:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*34:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0012380   33.427 0.059824   47.280 26173.32\r\n",
      "   1 HLA-A*34:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0030930   23.624 0.099019   26.370 17127.06\r\n",
      "   1 HLA-A*34:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0107870   12.957 0.134816   16.171 11627.16\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*34:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A26:03 -xls -xlsfile test/HLA-A2603_preds.xls\r\n",
      "# Wed Oct 11 17:02:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A26:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2603_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A26:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2603_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKcGDid\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A26:03 : Distance to training data  0.000 (using nearest neighbor HLA-A26:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*26:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001740   31.867 0.026975   50.582 37343.53\r\n",
      "   1 HLA-A*26:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004070   22.707 0.044216   29.233 30988.49\r\n",
      "   1 HLA-A*26:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007860   16.883 0.068315   15.531 23875.90\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*26:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:21 -xls -xlsfile test/HLA-B1521_preds.xls\r\n",
      "# Wed Oct 11 17:02:59 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:21           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1521_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:21           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1521_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanW0CW2g\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:21 : Distance to training data  0.038 (using nearest neighbor HLA-B15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:21       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002330   41.688 0.016108   69.114 42002.88\r\n",
      "   1 HLA-B*15:21       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0132640    7.327 0.073369    9.853 22605.35\r\n",
      "   1 HLA-B*15:21      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0050790   11.969 0.050068   18.291 29087.20\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:21. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:25 -xls -xlsfile test/HLA-B1525_preds.xls\r\n",
      "# Wed Oct 11 17:03:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:25           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1525_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:25           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1525_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKWL70k\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:25 : Distance to training data  0.029 (using nearest neighbor HLA-B15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:25       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0006510   27.514 0.064698   44.111 24828.82\r\n",
      "   1 HLA-B*15:25       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0278060    6.325 0.237032    8.783  3847.37\r\n",
      "   1 HLA-B*15:25      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0233170    6.876 0.266345    7.161  2801.71\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:25. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:27 -xls -xlsfile test/HLA-B1527_preds.xls\r\n",
      "# Wed Oct 11 17:03:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:27           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1527_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:27           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1527_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpankDHWfg\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:27 : Distance to training data  0.018 (using nearest neighbor HLA-B15:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:27       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0002400   29.455 0.039216   50.190 32711.11\r\n",
      "   1 HLA-B*15:27       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0073550    7.406 0.146730    9.465 10220.92\r\n",
      "   1 HLA-B*15:27      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0048020    8.928 0.143658    9.801 10566.36\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:27. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A24:64 -xls -xlsfile test/HLA-A2464_preds.xls\r\n",
      "# Wed Oct 11 17:03:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A24:64           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2464_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A24:64           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2464_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanCzYjiY\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A24:64 : Distance to training data  0.073 (using nearest neighbor HLA-A24:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*24:64       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000330   47.000 0.025526   53.410 37933.61\r\n",
      "   1 HLA-A*24:64       TLIVNSVL TLIVN-SVL  0  0  0  5  1     TLIVNSVL         PEPLIST 0.0006060   17.800 0.050990   25.193 28798.48\r\n",
      "   1 HLA-A*24:64      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009300   14.855 0.069507   16.323 23569.95\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*24:64. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B18:02 -xls -xlsfile test/HLA-B1802_preds.xls\r\n",
      "# Wed Oct 11 17:03:00 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B18:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B18:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanAuiBeN\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B18:02 : Distance to training data  0.028 (using nearest neighbor HLA-B18:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*18:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0005390   35.227 0.033194   61.942 34913.42\r\n",
      "   1 HLA-B*18:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0042780   15.494 0.063062   28.661 25272.23\r\n",
      "   1 HLA-B*18:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0011500   26.817 0.050321   39.151 29007.70\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*18:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B48:03 -xls -xlsfile test/HLA-B4803_preds.xls\r\n",
      "# Wed Oct 11 17:03:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B48:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4803_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B48:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4803_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKRp0U5\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B48:03 : Distance to training data  0.030 (using nearest neighbor HLA-B48:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*48:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000890   26.700 0.021932   42.378 39437.75\r\n",
      "   1 HLA-B*48:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0049430    4.603 0.101284    5.416 16712.42\r\n",
      "   1 HLA-B*48:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066890    3.931 0.143261    2.758 10611.84\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*48:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B27:06 -xls -xlsfile test/HLA-B2706_preds.xls\r\n",
      "# Wed Oct 11 17:03:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B27:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B2706_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B27:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B2706_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpano518dL\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B27:06 : Distance to training data  0.000 (using nearest neighbor HLA-B27:06)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*27:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000320   38.000 0.015428   50.995 42313.04\r\n",
      "   1 HLA-B*27:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001320   24.143 0.029577   27.393 36306.84\r\n",
      "   1 HLA-B*27:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0007730   12.706 0.049630   14.412 29225.38\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*27:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:07 -xls -xlsfile test/HLA-A0207_preds.xls\r\n",
      "# Wed Oct 11 17:03:01 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0207_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0207_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanpetBIw\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:07 : Distance to training data  0.000 (using nearest neighbor HLA-A02:07)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0014130   13.635 0.020162   28.428 40200.31\r\n",
      "   1 HLA-A*02:07       TLIVNSVL TLIVNSV-L  0  0  0  7  1     TLIVNSVL         PEPLIST 0.0041860    8.172 0.035998   14.087 33870.11\r\n",
      "   1 HLA-A*02:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0036230    8.771 0.035524   14.346 34044.25\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:03 -xls -xlsfile test/HLA-A0203_preds.xls\r\n",
      "# Wed Oct 11 17:03:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A0203_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A0203_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan8dmwlT\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:03 : Distance to training data  0.000 (using nearest neighbor HLA-A02:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:03       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0034920   11.981 0.165189   18.988  8370.51\r\n",
      "   1 HLA-A*02:03       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0142870    6.344 0.255931   11.005  3135.87\r\n",
      "   1 HLA-A*02:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0030840   12.638 0.194167   15.822  6117.66\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B46:01 -xls -xlsfile test/HLA-B4601_preds.xls\r\n",
      "# Wed Oct 11 17:03:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B46:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4601_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B46:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4601_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanznY4UZ\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B46:01 : Distance to training data  0.000 (using nearest neighbor HLA-B46:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*46:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0007490   22.259 0.018539   34.325 40912.47\r\n",
      "   1 HLA-B*46:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0144810    6.037 0.052650    7.892 28285.86\r\n",
      "   1 HLA-B*46:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0179070    5.382 0.064584    5.538 24859.46\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*46:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A02:264 -xls -xlsfile test/HLA-A02264_preds.xls\r\n",
      "# Wed Oct 11 17:03:02 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A02:264          MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A02264_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A02:264          MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A02264_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanEYGLc9\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A02:264 : Distance to training data  0.000 (using nearest neighbor HLA-A02:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*02:264       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0034920   11.981 0.165189   18.988  8370.51\r\n",
      "   1 HLA-A*02:264       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0142870    6.344 0.255931   11.005  3135.87\r\n",
      "   1 HLA-A*02:264      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0030840   12.638 0.194167   15.822  6117.66\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*02:264. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B35:05 -xls -xlsfile test/HLA-B3505_preds.xls\r\n",
      "# Wed Oct 11 17:03:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B35:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3505_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B35:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3505_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanSoDtOD\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B35:05 : Distance to training data  0.039 (using nearest neighbor HLA-B35:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*35:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0004450   31.310 0.020069   64.894 40240.78\r\n",
      "   1 HLA-B*35:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0065310   11.690 0.066490   19.527 24352.05\r\n",
      "   1 HLA-B*35:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0066420   11.608 0.066937   19.368 24234.56\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*35:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B13:01 -xls -xlsfile test/HLA-B1301_preds.xls\r\n",
      "# Wed Oct 11 17:03:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B13:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1301_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B13:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1301_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan59JKHk\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B13:01 : Distance to training data  0.000 (using nearest neighbor HLA-B13:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*13:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0109150   12.712 0.082120   24.431 20563.20\r\n",
      "   1 HLA-B*13:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0192370    9.384 0.114981   15.551 14410.46\r\n",
      "   1 HLA-B*13:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0746110    3.562 0.218869    4.122  4682.86\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*13:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A24:07 -xls -xlsfile test/HLA-A2407_preds.xls\r\n",
      "# Wed Oct 11 17:03:03 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A24:07           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2407_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A24:07           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2407_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanA8VKNH\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A24:07 : Distance to training data  0.034 (using nearest neighbor HLA-A24:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*24:07       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000950   31.000 0.022620   49.531 39145.26\r\n",
      "   1 HLA-A*24:07       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0003720   19.577 0.039984   29.270 32440.42\r\n",
      "   1 HLA-A*24:07      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0016360   10.826 0.070276   15.447 23374.65\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*24:07. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B40:06 -xls -xlsfile test/HLA-B4006_preds.xls\r\n",
      "# Wed Oct 11 17:03:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B40:06           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B4006_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B40:06           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B4006_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanxCrmFL\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B40:06 : Distance to training data  0.065 (using nearest neighbor HLA-B40:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*40:06       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000780   34.667 0.023368   48.000 38829.74\r\n",
      "   1 HLA-B*40:06       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0001070   30.667 0.022186   50.814 39329.53\r\n",
      "   1 HLA-B*40:06      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0002480   21.750 0.035784   28.270 33948.61\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*40:06. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B38:02 -xls -xlsfile test/HLA-B3802_preds.xls\r\n",
      "# Wed Oct 11 17:03:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B38:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3802_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B38:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3802_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanIPILFg\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B38:02 : Distance to training data  0.027 (using nearest neighbor HLA-B38:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*38:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000420   49.000 0.014384   63.443 42793.73\r\n",
      "   1 HLA-B*38:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0015740   14.945 0.040432   23.082 32283.55\r\n",
      "   1 HLA-B*38:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009530   18.291 0.044647   20.248 30844.32\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*38:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:05 -xls -xlsfile test/HLA-B3905_preds.xls\r\n",
      "# Wed Oct 11 17:03:04 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:05           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3905_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:05           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3905_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanvyxm1A\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:05 : Distance to training data  0.047 (using nearest neighbor HLA-B39:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:05       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001080   37.222 0.018332   55.534 41004.21\r\n",
      "   1 HLA-B*39:05       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0034840    9.544 0.066246   11.426 24416.41\r\n",
      "   1 HLA-B*39:05      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0013740   14.568 0.059171   13.631 26358.89\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:05. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B51:02 -xls -xlsfile test/HLA-B5102_preds.xls\r\n",
      "# Wed Oct 11 17:03:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B51:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5102_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B51:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5102_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanjk1nhq\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B51:02 : Distance to training data  0.027 (using nearest neighbor HLA-B51:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*51:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0098710    9.579 0.051287   16.366 28706.08\r\n",
      "   1 HLA-B*51:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0064120   11.874 0.046190   18.905 30333.64\r\n",
      "   1 HLA-B*51:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0092560    9.899 0.051438   16.297 28659.23\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*51:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-C01:03 -xls -xlsfile test/HLA-C0103_preds.xls\r\n",
      "# Wed Oct 11 17:03:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-C01:03           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-C0103_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-C01:03           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-C0103_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanKldS8x\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-C01:03 : Distance to training data  0.041 (using nearest neighbor HLA-C01:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-C*01:03       GTLIVNSV GTLI-VNSV  0  0  0  4  1     GTLIVNSV         PEPLIST 0.0007580   19.793 0.018987   37.011 40714.64\r\n",
      "   1 HLA-C*01:03       TLIVNSVL TLIVNS-VL  0  0  0  6  1     TLIVNSVL         PEPLIST 0.0352740    2.898 0.109349    3.116 15315.89\r\n",
      "   1 HLA-C*01:03      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0065830    8.208 0.071437    7.091 23082.86\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-C*01:03. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B39:09 -xls -xlsfile test/HLA-B3909_preds.xls\r\n",
      "# Wed Oct 11 17:03:05 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B39:09           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B3909_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B39:09           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B3909_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanHj2TWt\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B39:09 : Distance to training data  0.038 (using nearest neighbor HLA-B39:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*39:09       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000380   43.750 0.010667   62.386 44549.82\r\n",
      "   1 HLA-B*39:09       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0041270    6.459 0.051849    8.208 28532.06\r\n",
      "   1 HLA-B*39:09      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0009190   13.750 0.037747   14.000 33235.17\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*39:09. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:02 -xls -xlsfile test/HLA-B1502_preds.xls\r\n",
      "# Wed Oct 11 17:03:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1502_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1502_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanP2E0u6\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:02 : Distance to training data  0.000 (using nearest neighbor HLA-B15:02)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0001650   41.250 0.028364   60.318 36786.50\r\n",
      "   1 HLA-B*15:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0210630    5.814 0.154079    7.537  9439.68\r\n",
      "   1 HLA-B*15:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0034120   12.939 0.099110   14.549 17110.20\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B54:01 -xls -xlsfile test/HLA-B5401_preds.xls\r\n",
      "# Wed Oct 11 17:03:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B54:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5401_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B54:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5401_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanlsMcTg\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B54:01 : Distance to training data  0.000 (using nearest neighbor HLA-B54:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*54:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0011120   16.821 0.032925   32.197 35015.18\r\n",
      "   1 HLA-B*54:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004270   25.049 0.022816   45.984 39062.34\r\n",
      "   1 HLA-B*54:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0005530   22.627 0.030851   34.475 35809.81\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*54:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B55:02 -xls -xlsfile test/HLA-B5502_preds.xls\r\n",
      "# Wed Oct 11 17:03:06 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B55:02           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5502_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B55:02           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5502_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanbSjUpI\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B55:02 : Distance to training data  0.028 (using nearest neighbor HLA-B56:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*55:02       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0010210   24.154 0.027220   38.360 37244.66\r\n",
      "   1 HLA-B*55:02       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0006910   28.719 0.023528   44.806 38762.59\r\n",
      "   1 HLA-B*55:02      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0017490   18.891 0.034362   29.397 34474.97\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*55:02. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B67:01 -xls -xlsfile test/HLA-B6701_preds.xls\r\n",
      "# Wed Oct 11 17:03:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B67:01           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B6701_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B67:01           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B6701_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXhEBJn\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B67:01 : Distance to training data  0.000 (using nearest neighbor HLA-B67:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*67:01       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003210   27.806 0.031868   45.693 35417.94\r\n",
      "   1 HLA-B*67:01       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0075540    7.858 0.113278   10.747 14678.44\r\n",
      "   1 HLA-B*67:01      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0175890    5.166 0.177058    5.327  7361.73\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*67:01. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-A24:10 -xls -xlsfile test/HLA-A2410_preds.xls\r\n",
      "# Wed Oct 11 17:03:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-A24:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-A2410_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-A24:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-A2410_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpan3lsMXh\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-A24:10 : Distance to training data  0.031 (using nearest neighbor HLA-A24:03)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-A*24:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0000700   42.333 0.033472   47.858 34808.56\r\n",
      "   1 HLA-A*24:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0004780   24.103 0.061935   27.899 25582.28\r\n",
      "   1 HLA-A*24:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0064280    9.338 0.113650   14.257 14619.48\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-A*24:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B15:32 -xls -xlsfile test/HLA-B1532_preds.xls\r\n",
      "# Wed Oct 11 17:03:07 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B15:32           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B1532_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B15:32           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B1532_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanSb4jp3\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B15:32 : Distance to training data  0.038 (using nearest neighbor HLA-B15:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*15:32       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0003320   32.536 0.038525   49.228 32956.59\r\n",
      "   1 HLA-B*15:32       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0201460    5.487 0.144649    8.159 10453.66\r\n",
      "   1 HLA-B*15:32      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0116680    7.375 0.143044    8.337 10636.79\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*15:32. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n",
      "# /Users/sara/netMHCpan-4.1/Darwin_arm64/bin/netMHCpan -BA -p test/filtered_peptides_mhcI.pep -a HLA-B56:10 -xls -xlsfile test/HLA-B5610_preds.xls\r\n",
      "# Wed Oct 11 17:03:08 2023\r\n",
      "# User: sara\r\n",
      "# PWD : /Users/sara/Documents/VaccinesProject/ivp/Code/HOGVAX\r\n",
      "# Host: Darwin bison-skater.local 22.6.0 x86_64\r\n",
      "# -BA      1                    Include Binding affinity prediction\r\n",
      "# -p       1                    Use peptide input\r\n",
      "# -a       HLA-B56:10           MHC allele\r\n",
      "# -xls     1                    Save output to xls file\r\n",
      "# -xlsfile test/HLA-B5610_preds.xls Filename for xls dump\r\n",
      "# Command line parameters set to:\r\n",
      "#\t[-rdir filename]     /Users/sara/netMHCpan-4.1/Darwin_arm64 Home directory for NetMHpan\r\n",
      "#\t[-syn filename]      /Users/sara/netMHCpan-4.1/Darwin_arm64/data/synlist.bin Synaps file\r\n",
      "#\t[-v]                 0                    Verbose mode\r\n",
      "#\t[-dirty]             0                    Dirty mode, leave tmp dir+files\r\n",
      "#\t[-tdir filename]     /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanXXXXXX Temporary directory (made with mkdtemp)\r\n",
      "#\t[-hlapseudo filename] /Users/sara/netMHCpan-4.1/Darwin_arm64/data/MHC_pseudo.dat File with MHC pseudo sequences\r\n",
      "#\t[-hlaseq filename]                        File with full length MHC sequences\r\n",
      "#\t[-a line]            HLA-B56:10           MHC allele\r\n",
      "#\t[-f filename]                             File name with input\r\n",
      "#\t[-w]                 0                    w option for webface\r\n",
      "#\t[-s]                 0                    Sort output on descending affinity\r\n",
      "#\t[-p]                 1                    Use peptide input\r\n",
      "#\t[-rth float]         0.500000             Rank Threshold for high binding peptides\r\n",
      "#\t[-rlt float]         2.000000             Rank Threshold for low binding peptides\r\n",
      "#\t[-l string]          8,9,10,11            Peptide length [8-11] (multiple length with ,)\r\n",
      "#\t[-xls]               1                    Save output to xls file\r\n",
      "#\t[-xlsfile filename]  test/HLA-B5610_preds.xls Filename for xls dump\r\n",
      "#\t[-t float]           -99.900002           Threshold for output (%rank) [<0 print all]\r\n",
      "#\t[-thrfmt filename]   /Users/sara/netMHCpan-4.1/Darwin_arm64/data/threshold/%s.thr.%s Format for threshold filenames\r\n",
      "#\t[-expfix]            0                    Exclude prefix from synlist\r\n",
      "#\t[-version filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/version File with version information\r\n",
      "#\t[-inptype int]       0                    Input type [0] FASTA [1] Peptide\r\n",
      "#\t[-listMHC]           0                    Print list of alleles included in netMHCpan\r\n",
      "#\t[-allname filename]  /Users/sara/netMHCpan-4.1/Darwin_arm64/data/allelenames File with print names for alleles\r\n",
      "#\t[-BA]                1                    Include Binding affinity prediction\r\n",
      "\r\n",
      "# NetMHCpan version 4.1b\r\n",
      "\r\n",
      "# Tmpdir made /var/folders/jz/006cgcp100n00j3zvx6_0wvm0000gn/T//netMHCpanNjIayp\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Make both EL and BA predictions\r\n",
      "\r\n",
      "HLA-B56:10 : Distance to training data  0.064 (using nearest neighbor HLA-B67:01)\r\n",
      "\r\n",
      "# Rank Threshold for Strong binding peptides   0.500\r\n",
      "# Rank Threshold for Weak binding peptides   2.000\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos         MHC        Peptide      Core Of Gp Gl Ip Il        Icore        Identity  Score_EL %Rank_EL Score_BA %Rank_BA  Aff(nM) BindLevel\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1 HLA-B*56:10       GTLIVNSV GTL-IVNSV  0  0  0  3  1     GTLIVNSV         PEPLIST 0.0008830   22.886 0.036177   43.169 33804.57\r\n",
      "   1 HLA-B*56:10       TLIVNSVL TLI-VNSVL  0  0  0  3  1     TLIVNSVL         PEPLIST 0.0079300    9.606 0.097067   14.051 17492.63\r\n",
      "   1 HLA-B*56:10      GTLIVNSVL GTLIVNSVL  0  0  0  0  0    GTLIVNSVL         PEPLIST 0.0221630    5.886 0.150941    7.395  9765.68\r\n",
      "---------------------------------------------------------------------------------------------------------------------------\r\n",
      "\r\n",
      "Protein PEPLIST. Allele HLA-B*56:10. Number of high binders 0. Number of weak binders 0. Number of peptides 3\r\n",
      "\r\n",
      "-----------------------------------------------------------------------------------\r\n"
     ]
    }
   ],
   "source": [
    "# for MHC class I predictions\n",
    "for allele in hla_alleles['allele']:\n",
    "    outfile = allele.replace(':', '') + '_preds.xls' \n",
    "    ! ~/netMHCpan-4.1/netMHCpan -BA -p {peptides_file} -a {allele} -xls -xlsfile test/{outfile}"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:03:08.471138Z",
     "start_time": "2023-10-11T15:01:50.645781Z"
    }
   },
   "id": "c4e5233012683c15"
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "outputs": [
    {
     "data": {
      "text/plain": "Locus          HLA-A                                                         \\\nAllele    HLA-A01:01 HLA-A01:02 HLA-A01:03 HLA-A01:09 HLA-A01:23 HLA-A02:01   \nPeptide                                                                       \nGTLIVNSV      0.0335     0.0492     0.0300     0.0335     0.0298     0.1008   \nGTLIVNSVL     0.0527     0.0788     0.0521     0.0527     0.0473     0.1899   \nTLIVNSVL      0.0335     0.0483     0.0318     0.0335     0.0322     0.1666   \n\nLocus                                                  ...      HLA-C  \\\nAllele    HLA-A02:02 HLA-A02:03 HLA-A02:04 HLA-A02:05  ... HLA-C17:01   \nPeptide                                                ...              \nGTLIVNSV      0.1309     0.1652     0.0712     0.1601  ...     0.0624   \nGTLIVNSVL     0.2556     0.1942     0.1304     0.2900  ...     0.2784   \nTLIVNSVL      0.2616     0.2559     0.1078     0.1639  ...     0.1442   \n\nLocus                                                                        \\\nAllele    HLA-C17:02 HLA-C17:03 HLA-C17:04 HLA-C17:05 HLA-C17:06 HLA-C17:07   \nPeptide                                                                       \nGTLIVNSV      0.0624     0.0624     0.0624     0.0624     0.0624     0.0360   \nGTLIVNSVL     0.2784     0.2784     0.2784     0.2784     0.2784     0.2048   \nTLIVNSVL      0.1442     0.1442     0.1442     0.1442     0.1442     0.1132   \n\nLocus                                       \nAllele    HLA-C18:01 HLA-C18:02 HLA-C18:03  \nPeptide                                     \nGTLIVNSV      0.0266     0.0266     0.0267  \nGTLIVNSVL     0.0549     0.0549     0.0489  \nTLIVNSVL      0.0586     0.0586     0.0564  \n\n[3 rows x 230 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead tr th {\n        text-align: left;\n    }\n\n    .dataframe thead tr:last-of-type th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr>\n      <th>Locus</th>\n      <th colspan=\"10\" halign=\"left\">HLA-A</th>\n      <th>...</th>\n      <th colspan=\"10\" halign=\"left\">HLA-C</th>\n    </tr>\n    <tr>\n      <th>Allele</th>\n      <th>HLA-A01:01</th>\n      <th>HLA-A01:02</th>\n      <th>HLA-A01:03</th>\n      <th>HLA-A01:09</th>\n      <th>HLA-A01:23</th>\n      <th>HLA-A02:01</th>\n      <th>HLA-A02:02</th>\n      <th>HLA-A02:03</th>\n      <th>HLA-A02:04</th>\n      <th>HLA-A02:05</th>\n      <th>...</th>\n      <th>HLA-C17:01</th>\n      <th>HLA-C17:02</th>\n      <th>HLA-C17:03</th>\n      <th>HLA-C17:04</th>\n      <th>HLA-C17:05</th>\n      <th>HLA-C17:06</th>\n      <th>HLA-C17:07</th>\n      <th>HLA-C18:01</th>\n      <th>HLA-C18:02</th>\n      <th>HLA-C18:03</th>\n    </tr>\n    <tr>\n      <th>Peptide</th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>GTLIVNSV</th>\n      <td>0.0335</td>\n      <td>0.0492</td>\n      <td>0.0300</td>\n      <td>0.0335</td>\n      <td>0.0298</td>\n      <td>0.1008</td>\n      <td>0.1309</td>\n      <td>0.1652</td>\n      <td>0.0712</td>\n      <td>0.1601</td>\n      <td>...</td>\n      <td>0.0624</td>\n      <td>0.0624</td>\n      <td>0.0624</td>\n      <td>0.0624</td>\n      <td>0.0624</td>\n      <td>0.0624</td>\n      <td>0.0360</td>\n      <td>0.0266</td>\n      <td>0.0266</td>\n      <td>0.0267</td>\n    </tr>\n    <tr>\n      <th>GTLIVNSVL</th>\n      <td>0.0527</td>\n      <td>0.0788</td>\n      <td>0.0521</td>\n      <td>0.0527</td>\n      <td>0.0473</td>\n      <td>0.1899</td>\n      <td>0.2556</td>\n      <td>0.1942</td>\n      <td>0.1304</td>\n      <td>0.2900</td>\n      <td>...</td>\n      <td>0.2784</td>\n      <td>0.2784</td>\n      <td>0.2784</td>\n      <td>0.2784</td>\n      <td>0.2784</td>\n      <td>0.2784</td>\n      <td>0.2048</td>\n      <td>0.0549</td>\n      <td>0.0549</td>\n      <td>0.0489</td>\n    </tr>\n    <tr>\n      <th>TLIVNSVL</th>\n      <td>0.0335</td>\n      <td>0.0483</td>\n      <td>0.0318</td>\n      <td>0.0335</td>\n      <td>0.0322</td>\n      <td>0.1666</td>\n      <td>0.2616</td>\n      <td>0.2559</td>\n      <td>0.1078</td>\n      <td>0.1639</td>\n      <td>...</td>\n      <td>0.1442</td>\n      <td>0.1442</td>\n      <td>0.1442</td>\n      <td>0.1442</td>\n      <td>0.1442</td>\n      <td>0.1442</td>\n      <td>0.1132</td>\n      <td>0.0586</td>\n      <td>0.0586</td>\n      <td>0.0564</td>\n    </tr>\n  </tbody>\n</table>\n<p>3 rows × 230 columns</p>\n</div>"
     },
     "execution_count": 364,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfs = []\n",
    "for allele in hla_alleles['allele']:\n",
    "    df = pd.read_csv(\n",
    "        'test/' + allele.replace(':', '') + '_preds.xls',\n",
    "        delimiter='\\t',\n",
    "        skiprows=[0],\n",
    "    )\n",
    "    df['Allele'] = allele\n",
    "    df = df.drop(columns=['Pos', 'ID', 'core', 'icore', 'Ave', 'NB'])\n",
    "    dfs.append(df)\n",
    "\n",
    "netmhc41_data = pd.concat(dfs)\n",
    "netmhc41_data['sequence_length'] = [len(x) for x in netmhc41_data['Peptide'].values]\n",
    "netmhc41_data['BA_nM'] = 50000 ** (1 - netmhc41_data['BA-score'])\n",
    "netmhc41_data['Locus'] = [x[:5] for x in netmhc41_data['Allele'].values]\n",
    "\n",
    "data_pivot = netmhc41_data.pivot_table(\n",
    "    index='Peptide',\n",
    "    columns=['Locus', 'Allele'],\n",
    "    values='BA-score',\n",
    ")\n",
    "\n",
    "data_pivot"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:20:45.511523Z",
     "start_time": "2023-10-11T15:20:45.177626Z"
    }
   },
   "id": "54ace2bb4f487545"
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "outputs": [],
   "source": [
    "data_pivot.to_pickle('test/netmhcpan_pred.pkl.gz', protocol=2)"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:21:37.753743Z",
     "start_time": "2023-10-11T15:21:37.676825Z"
    }
   },
   "id": "8a58f1694d81db81"
  },
  {
   "cell_type": "code",
   "execution_count": 357,
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB11301        VFLLVTLAILTAL    3   LVTLAILTA     0.287        Sequence      0.017498    88.34       NA      0.318113       1600.17    26.37       \r\n",
      "   2   HLA-DPA10301-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.040657    63.76       NA      0.357508       1044.84    16.43       \r\n",
      "   3   HLA-DPA10301-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.760        Sequence      0.027952    77.24       NA      0.306902       1806.54    29.68       \r\n",
      "   4   HLA-DPA10301-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.026192    79.17       NA      0.341725       1239.41    20.10       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB155801 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0701\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0701        VFLLVTLAILTAL    2   LLVTLAILT     0.853        Sequence      0.001012    90.98       NA      0.236782       3857.79    58.44       \r\n",
      "   2     DRB1_0701        LLVTLAILTALRL    4   LAILTALRL     0.973        Sequence      0.012275    44.85       NA      0.330650       1397.19    34.72       \r\n",
      "   3     DRB1_0701        LVTLAILTALRLC    3   LAILTALRL     0.973        Sequence      0.003722    70.05       NA      0.265097       2839.80    50.79       \r\n",
      "   4     DRB1_0701       LLVTLAILTALRLC    4   LAILTALRL     0.960        Sequence      0.003085    73.93       NA      0.295045       2053.82    43.13       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB116201 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB15501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB15501        VFLLVTLAILTAL    1   FLLVTLAIL     0.273        Sequence      0.024040    88.39       NA      0.364339        970.40    22.91       \r\n",
      "   2   HLA-DPA10301-DPB15501        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.055215    63.59       NA      0.389846        736.37    16.98       \r\n",
      "   3   HLA-DPA10301-DPB15501        LVTLAILTALRLC    3   LAILTALRL     0.747        Sequence      0.040160    75.49       NA      0.336729       1308.25    30.32       \r\n",
      "   4   HLA-DPA10301-DPB15501       LLVTLAILTALRLC    4   LAILTALRL     0.720        Sequence      0.035666    79.24       NA      0.373620        877.69    20.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10202-DPB116201 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10601-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10601-DQB10301        VFLLVTLAILTAL    2   LLVTLAILT     0.140        Sequence      0.004026    89.08       NA      0.154648       9381.75    88.46       \r\n",
      "   2   HLA-DQA10601-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.500        Sequence      0.003279    91.35       NA      0.187643       6565.10    77.45       \r\n",
      "   3   HLA-DQA10601-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.473        Sequence      0.001763    95.00       NA      0.175945       7450.92    81.68       \r\n",
      "   4   HLA-DQA10601-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.407        Sequence      0.001934    95.00       NA      0.189232       6453.19    76.86       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10106-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10106-DPB10101        VFLLVTLAILTAL    3   LVTLAILTA     0.447        Sequence      0.001635    93.53       NA      0.238754       3776.35    55.05       \r\n",
      "   2   HLA-DPA10106-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.003970    81.86       NA      0.308834       1769.16    35.22       \r\n",
      "   3   HLA-DPA10106-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.720        Sequence      0.001198    95.00       NA      0.251967       3273.29    51.13       \r\n",
      "   4   HLA-DPA10106-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.673        Sequence      0.001823    92.56       NA      0.270553       2677.01    45.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB13001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB13001        VFLLVTLAILTAL    1   FLLVTLAIL     0.160        Sequence      0.002822    93.36       NA      0.207627       5288.55    35.95       \r\n",
      "   2   HLA-DPA10103-DPB13001        LLVTLAILTALRL    4   LAILTALRL     0.580        Sequence      0.005516    83.47       NA      0.229386       4179.19    28.16       \r\n",
      "   3   HLA-DPA10103-DPB13001        LVTLAILTALRLC    3   LAILTALRL     0.613        Sequence      0.003673    90.32       NA      0.188203       6525.44    43.89       \r\n",
      "   4   HLA-DPA10103-DPB13001       LLVTLAILTALRLC    4   LAILTALRL     0.560        Sequence      0.003262    91.79       NA      0.219529       4649.54    31.60       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1001        VFLLVTLAILTAL    3   LVTLAILTA     0.653        Sequence      0.002186    81.43       NA      0.312159       1706.65    62.01       \r\n",
      "   2     DRB1_1001        LLVTLAILTALRL    4   LAILTALRL     0.913        Sequence      0.011269    51.76       NA      0.432695        463.18    35.16       \r\n",
      "   3     DRB1_1001        LVTLAILTALRLC    3   LAILTALRL     0.987        Sequence      0.013167    48.60       NA      0.397551        677.47    42.77       \r\n",
      "   4     DRB1_1001       LLVTLAILTALRLC    4   LAILTALRL     0.947        Sequence      0.009734    54.51       NA      0.422349        518.04    37.39       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB14901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB14901        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000950    85.82       NA      0.185947       6686.68    45.75       \r\n",
      "   2   HLA-DPA10103-DPB14901        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.000977    85.35       NA      0.216799       4788.92    35.03       \r\n",
      "   3   HLA-DPA10103-DPB14901        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.000397    95.00       NA      0.183982       6830.37    46.48       \r\n",
      "   4   HLA-DPA10103-DPB14901       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000554    92.82       NA      0.203954       5502.96    39.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10202        VFLLVTLAILTAL    1   FLLVTLAIL     0.300        Sequence      0.020627    79.76       NA      0.427752        488.63    29.21       \r\n",
      "   2   HLA-DPA10301-DPB10202        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.047296    56.20       NA      0.483131        268.38    19.27       \r\n",
      "   3   HLA-DPA10301-DPB10202        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.034028    66.89       NA      0.414231        565.60    31.99       \r\n",
      "   4   HLA-DPA10301-DPB10202       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.034105    66.83       NA      0.452472        373.95    24.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.012784    82.91       NA      0.418281        541.35    31.88       \r\n",
      "   2   HLA-DPA10301-DPB10201        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.023536    69.84       NA      0.466260        322.13    23.46       \r\n",
      "   3   HLA-DPA10301-DPB10201        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.015934    78.80       NA      0.398109        673.39    35.79       \r\n",
      "   4   HLA-DPA10301-DPB10201       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.016243    78.41       NA      0.427557        489.66    30.18       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10401        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000538    88.66       NA      0.233761       3985.97    38.43       \r\n",
      "   2   HLA-DPA10103-DPB10401        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.001728    64.59       NA      0.282016       2364.75    27.21       \r\n",
      "   3   HLA-DPA10103-DPB10401        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000766    83.00       NA      0.223330       4462.20    41.24       \r\n",
      "   4   HLA-DPA10103-DPB10401       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000700    84.53       NA      0.244981       3530.30    35.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000950    85.82       NA      0.185947       6686.68    45.75       \r\n",
      "   2   HLA-DPA10103-DPB10402        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.000977    85.35       NA      0.216799       4788.92    35.03       \r\n",
      "   3   HLA-DPA10103-DPB10402        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.000397    95.00       NA      0.183982       6830.37    46.48       \r\n",
      "   4   HLA-DPA10103-DPB10402       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000554    92.82       NA      0.203954       5502.96    39.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000851    93.63       NA      0.179439       7174.50    85.71       \r\n",
      "   2   HLA-DQA10505-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.640        Sequence      0.002548    74.46       NA      0.233057       4016.45    69.17       \r\n",
      "   3   HLA-DQA10505-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.553        Sequence      0.001358    87.65       NA      0.215915       4834.95    75.24       \r\n",
      "   4   HLA-DQA10505-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.540        Sequence      0.001271    88.68       NA      0.208564       5235.21    77.54       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB15501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB15501        VFLLVTLAILTAL    2   LLVTLAILT     0.493        Sequence      0.003677    82.99       NA      0.263645       2884.76    28.58       \r\n",
      "   2   HLA-DPA10201-DPB15501        LLVTLAILTALRL    4   LAILTALRL     0.753        Sequence      0.014286    43.77       NA      0.298132       1986.35    19.36       \r\n",
      "   3   HLA-DPA10201-DPB15501        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.008924    59.78       NA      0.247666       3429.22    33.54       \r\n",
      "   4   HLA-DPA10201-DPB15501       LLVTLAILTALRLC    4   LAILTALRL     0.740        Sequence      0.009156    58.97       NA      0.277889       2472.74    24.42       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB135001 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1401        VFLLVTLAILTAL    3   LVTLAILTA     0.347        Sequence      0.000857    94.18       NA      0.197446       5904.42    67.89       \r\n",
      "   2     DRB1_1401        LLVTLAILTALRL    4   LAILTALRL     0.927        Sequence      0.011933    55.20       NA      0.305204       1840.03    30.76       \r\n",
      "   3     DRB1_1401        LVTLAILTALRLC    3   LAILTALRL     0.940        Sequence      0.008102    62.95       NA      0.262874       2908.93    44.40       \r\n",
      "   4     DRB1_1401       LLVTLAILTALRLC    4   LAILTALRL     0.940        Sequence      0.006363    67.65       NA      0.278915       2445.44    38.94       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1406\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1406        VFLLVTLAILTAL    3   LVTLAILTA     0.767        Sequence      0.005870    80.52       NA      0.218636       4694.68    75.65       \r\n",
      "   2     DRB1_1406        LLVTLAILTALRL    4   LAILTALRL     0.740        Sequence      0.023426    56.96       NA      0.334261       1343.65    43.80       \r\n",
      "   3     DRB1_1406        LVTLAILTALRLC    3   LAILTALRL     0.960        Sequence      0.026704    54.30       NA      0.307025       1804.13    51.46       \r\n",
      "   4     DRB1_1406       LLVTLAILTALRLC    4   LAILTALRL     0.880        Sequence      0.020917    59.13       NA      0.341131       1247.40    41.92       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1407\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1407        VFLLVTLAILTAL    2   LLVTLAILT     0.380        Sequence      0.000736    95.00       NA      0.184396       6799.84    58.13       \r\n",
      "   2     DRB1_1407        LLVTLAILTALRL    4   LAILTALRL     0.847        Sequence      0.006375    68.69       NA      0.254147       3196.99    29.28       \r\n",
      "   3     DRB1_1407        LVTLAILTALRLC    3   LAILTALRL     0.833        Sequence      0.003407    80.33       NA      0.213471       4964.51    45.28       \r\n",
      "   4     DRB1_1407       LLVTLAILTALRLC    4   LAILTALRL     0.840        Sequence      0.002763    83.66       NA      0.225699       4349.28    40.14       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1404\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1404        VFLLVTLAILTAL    3   LVTLAILTA     0.340        Sequence      0.001046    94.43       NA      0.209554       5179.43    67.47       \r\n",
      "   2     DRB1_1404        LLVTLAILTALRL    4   LAILTALRL     0.933        Sequence      0.026605    47.13       NA      0.326028       1468.84    28.30       \r\n",
      "   3     DRB1_1404        LVTLAILTALRLC    3   LAILTALRL     0.967        Sequence      0.017331    56.16       NA      0.279685       2425.15    43.12       \r\n",
      "   4     DRB1_1404       LLVTLAILTALRLC    4   LAILTALRL     0.947        Sequence      0.013359    61.26       NA      0.292720       2106.14    38.82       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB15001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB15001        VFLLVTLAILTAL    3   LVTLAILTA     0.300        Sequence      0.006559    95.00       NA      0.230086       4147.66    76.19       \r\n",
      "   2   HLA-DPA10103-DPB15001        LLVTLAILTALRL    3   TLAILTALR     0.287        Sequence      0.011890    90.87       NA      0.287454       2229.63    59.41       \r\n",
      "   3   HLA-DPA10103-DPB15001        LVTLAILTALRLC    2   TLAILTALR     0.260        Sequence      0.007424    95.00       NA      0.256775       3107.36    68.76       \r\n",
      "   4   HLA-DPA10103-DPB15001       LLVTLAILTALRLC    3   TLAILTALR     0.193        Sequence      0.009362    93.39       NA      0.289500       2180.81    58.76       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10309\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10309        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10505-DQB10309        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10505-DQB10309        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10505-DQB10309       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10301        VFLLVTLAILTAL    4   VTLAILTAL     0.380        Sequence      0.000467    92.35       NA      0.138834      11132.51    93.30       \r\n",
      "   2   HLA-DQA10301-DQB10301        LLVTLAILTALRL    2   VTLAILTAL     0.360        Sequence      0.000733    84.09       NA      0.164948       8392.36    86.62       \r\n",
      "   3   HLA-DQA10301-DQB10301        LVTLAILTALRLC    1   VTLAILTAL     0.253        Sequence      0.000390    94.64       NA      0.157508       9095.88    88.87       \r\n",
      "   4   HLA-DQA10301-DQB10301       LLVTLAILTALRLC    2   VTLAILTAL     0.307        Sequence      0.000558    89.62       NA      0.162200       8645.64    87.45       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.000257    90.85       NA      0.129250      12348.90    86.43       \r\n",
      "   2   HLA-DQA10301-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.567        Sequence      0.000692    70.34       NA      0.157801       9067.09    76.24       \r\n",
      "   3   HLA-DQA10301-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.493        Sequence      0.000360    85.57       NA      0.143130      10626.90    81.82       \r\n",
      "   4   HLA-DQA10301-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.487        Sequence      0.000363    85.41       NA      0.137816      11255.81    83.69       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10303        VFLLVTLAILTAL    4   VTLAILTAL     0.253        Sequence      0.000351    90.83       NA      0.132050      11980.39    90.30       \r\n",
      "   2   HLA-DQA10301-DQB10303        LLVTLAILTALRL    2   VTLAILTAL     0.180        Sequence      0.000479    85.98       NA      0.150053       9859.97    85.26       \r\n",
      "   3   HLA-DQA10301-DQB10303        LVTLAILTALRLC    1   VTLAILTAL     0.147        Sequence      0.000324    91.86       NA      0.142978      10644.39    87.29       \r\n",
      "   4   HLA-DQA10301-DQB10303       LLVTLAILTALRLC    4   LAILTALRL     0.333        Sequence      0.000297    92.88       NA      0.142647      10682.58    87.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10304\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10304        VFLLVTLAILTAL    3   LVTLAILTA     0.313        Sequence      0.000286    95.00       NA      0.142769      10668.49    94.59       \r\n",
      "   2   HLA-DQA10301-DQB10304        LLVTLAILTALRL    4   LAILTALRL     0.513        Sequence      0.000648    91.00       NA      0.172357       7745.86    87.88       \r\n",
      "   3   HLA-DQA10301-DQB10304        LVTLAILTALRLC    3   LAILTALRL     0.500        Sequence      0.000348    95.00       NA      0.158292       9019.05    91.37       \r\n",
      "   4   HLA-DQA10301-DQB10304       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000379    95.00       NA      0.153343       9515.15    92.40       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10305\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10305        VFLLVTLAILTAL    3   LVTLAILTA     0.420        Sequence      0.001218    61.71       NA      0.128148      12497.02    91.29       \r\n",
      "   2   HLA-DQA10301-DQB10305        LLVTLAILTALRL    4   LAILTALRL     0.587        Sequence      0.001384    57.16       NA      0.161593       8702.61    81.81       \r\n",
      "   3   HLA-DQA10301-DQB10305        LVTLAILTALRLC    3   LAILTALRL     0.547        Sequence      0.000577    83.49       NA      0.146467      10250.05    86.65       \r\n",
      "   4   HLA-DQA10301-DQB10305       LLVTLAILTALRLC    4   LAILTALRL     0.520        Sequence      0.000669    79.93       NA      0.149801       9886.89    85.72       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB110601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10301-DPB110601        VFLLVTLAILTAL    2   LLVTLAILT     0.340        Sequence      0.031503    92.81       NA      0.343623       1214.22    46.30       \r\n",
      "   2  HLA-DPA10301-DPB110601        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.083116    69.51       NA      0.392247        717.49    32.39       \r\n",
      "   3  HLA-DPA10301-DPB110601        LVTLAILTALRLC    3   LAILTALRL     0.720        Sequence      0.055287    82.98       NA      0.340963       1249.67    47.05       \r\n",
      "   4  HLA-DPA10301-DPB110601       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.053186    83.93       NA      0.375030        864.40    37.23       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10105-DPB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10105-DPB10401        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000538    88.66       NA      0.233761       3985.97    38.43       \r\n",
      "   2   HLA-DPA10105-DPB10401        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.001728    64.59       NA      0.282016       2364.75    27.21       \r\n",
      "   3   HLA-DPA10105-DPB10401        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000766    83.00       NA      0.223330       4462.20    41.24       \r\n",
      "   4   HLA-DPA10105-DPB10401       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000700    84.53       NA      0.244981       3530.30    35.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB15501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB15501        VFLLVTLAILTAL    1   FLLVTLAIL     0.280        Sequence      0.005663    90.24       NA      0.356605       1055.10    20.20       \r\n",
      "   2   HLA-DPA10202-DPB15501        LLVTLAILTALRL    4   LAILTALRL     0.767        Sequence      0.036936    33.31       NA      0.415234        559.50     8.21       \r\n",
      "   3   HLA-DPA10202-DPB15501        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.023671    51.99       NA      0.350681       1124.94    21.76       \r\n",
      "   4   HLA-DPA10202-DPB15501       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.023708    51.93       NA      0.381437        806.51    14.28       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB110601 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10203\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10203        VFLLVTLAILTAL    3   LVTLAILTA     0.627        Sequence      0.015362    77.81       NA      0.187311       6588.72    79.98       \r\n",
      "   2   HLA-DQA10501-DQB10203        LLVTLAILTALRL    4   LAILTALRL     0.640        Sequence      0.015101    78.12       NA      0.233266       4007.38    63.66       \r\n",
      "   3   HLA-DQA10501-DQB10203        LVTLAILTALRLC    3   LAILTALRL     0.720        Sequence      0.008285    87.21       NA      0.210753       5112.67    72.11       \r\n",
      "   4   HLA-DQA10501-DQB10203       LLVTLAILTALRLC    4   LAILTALRL     0.647        Sequence      0.007562    88.31       NA      0.203999       5500.28    74.47       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001974    89.58       NA      0.173620       7640.73    74.29       \r\n",
      "   2   HLA-DQA10501-DQB10201        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.007163    68.86       NA      0.238041       3805.59    51.46       \r\n",
      "   3   HLA-DQA10501-DQB10201        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.004627    77.22       NA      0.193753       6145.12    67.41       \r\n",
      "   4   HLA-DQA10501-DQB10201       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.003865    80.13       NA      0.187206       6596.21    69.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10107-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10107-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.593        Sequence      0.000338    95.00       NA      0.118246      13910.27    75.08       \r\n",
      "   2   HLA-DQA10107-DQB10503        LLVTLAILTALRL    4   LAILTALRL     0.433        Sequence      0.000682    85.71       NA      0.152074       9646.70    59.35       \r\n",
      "   3   HLA-DQA10107-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.540        Sequence      0.000424    93.01       NA      0.133710      11767.13    67.91       \r\n",
      "   4   HLA-DQA10107-DQB10503       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000451    92.29       NA      0.144329      10489.92    62.93       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB11101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB11101        VFLLVTLAILTAL    1   FLLVTLAIL     0.280        Sequence      0.005302    91.11       NA      0.318888       1586.81    27.71       \r\n",
      "   2   HLA-DPA10301-DPB11101        LLVTLAILTALRL    4   LAILTALRL     0.720        Sequence      0.020182    58.02       NA      0.368889        923.79    15.66       \r\n",
      "   3   HLA-DPA10301-DPB11101        LVTLAILTALRLC    3   LAILTALRL     0.773        Sequence      0.017052    64.03       NA      0.318935       1586.00    27.70       \r\n",
      "   4   HLA-DPA10301-DPB11101       LLVTLAILTALRLC    4   LAILTALRL     0.727        Sequence      0.015554    67.14       NA      0.342001       1235.71    21.69       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11701\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11701        VFLLVTLAILTAL    2   LLVTLAILT     0.640        Sequence      0.002130    93.90       NA      0.201243       5666.76    38.11       \r\n",
      "   2   HLA-DPA10103-DPB11701        LLVTLAILTALRL    4   LAILTALRL     0.527        Sequence      0.004486    83.26       NA      0.209474       5183.92    34.96       \r\n",
      "   3   HLA-DPA10103-DPB11701        LVTLAILTALRLC    3   LAILTALRL     0.587        Sequence      0.003066    89.72       NA      0.174291       7585.46    49.76       \r\n",
      "   4   HLA-DPA10103-DPB11701       LLVTLAILTALRLC    4   LAILTALRL     0.507        Sequence      0.002360    92.87       NA      0.207620       5288.95    35.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.260        Sequence      0.000745    93.31       NA      0.203264       5544.20    52.73       \r\n",
      "   2   HLA-DPA10202-DPB10301        LLVTLAILTALRL    4   LAILTALRL     0.620        Sequence      0.001729    81.73       NA      0.239743       3736.15    37.97       \r\n",
      "   3   HLA-DPA10202-DPB10301        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.001118    88.84       NA      0.210374       5133.68    49.75       \r\n",
      "   4   HLA-DPA10202-DPB10301       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.001332    86.33       NA      0.234673       3946.83    39.95       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11501        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.001318    58.92       NA      0.269620       2704.17    56.40       \r\n",
      "   2   HLA-DPA10201-DPB11501        LLVTLAILTALRL    4   LAILTALRL     0.827        Sequence      0.003799    28.24       NA      0.351540       1114.54    32.03       \r\n",
      "   3   HLA-DPA10201-DPB11501        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.003784    28.34       NA      0.297692       1995.83    47.74       \r\n",
      "   4   HLA-DPA10201-DPB11501       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.003820    28.11       NA      0.317491       1610.98    41.73       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1312\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1312        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.000469    95.00       NA      0.162691       8599.83    78.75       \r\n",
      "   2     DRB1_1312        LLVTLAILTALRL    4   LAILTALRL     0.800        Sequence      0.002246    83.51       NA      0.242372       3631.38    51.26       \r\n",
      "   3     DRB1_1312        LVTLAILTALRLC    3   LAILTALRL     0.847        Sequence      0.001353    89.49       NA      0.204013       5499.45    65.05       \r\n",
      "   4     DRB1_1312       LLVTLAILTALRLC    4   LAILTALRL     0.813        Sequence      0.001142    91.15       NA      0.219936       4629.11    59.29       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11701\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11701        VFLLVTLAILTAL    3   LVTLAILTA     0.200        Sequence      0.000755    95.00       NA      0.188696       6490.72    37.57       \r\n",
      "   2   HLA-DPA10201-DPB11701        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.004175    65.22       NA      0.209120       5203.81    29.46       \r\n",
      "   3   HLA-DPA10201-DPB11701        LVTLAILTALRLC    3   LAILTALRL     0.747        Sequence      0.001963    83.18       NA      0.170009       7945.16    45.89       \r\n",
      "   4   HLA-DPA10201-DPB11701       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.002659    76.84       NA      0.199285       5788.10    33.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB13901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB13901        VFLLVTLAILTAL    1   FLLVTLAIL     0.313        Sequence      0.006801    79.06       NA      0.386223        765.81    29.62       \r\n",
      "   2   HLA-DPA10301-DPB13901        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.024710    44.05       NA      0.454982        363.93    18.41       \r\n",
      "   3   HLA-DPA10301-DPB13901        LVTLAILTALRLC    3   LAILTALRL     0.833        Sequence      0.011187    67.53       NA      0.380633        813.56    30.66       \r\n",
      "   4   HLA-DPA10301-DPB13901       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.011799    66.08       NA      0.411267        584.03    25.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10303        VFLLVTLAILTAL    3   LVTLAILTA     0.573        Sequence      0.001748    92.85       NA      0.197890       5876.12    78.21       \r\n",
      "   2   HLA-DQA10201-DQB10303        LLVTLAILTALRL    4   LAILTALRL     0.460        Sequence      0.002626    87.93       NA      0.244536       3547.34    63.79       \r\n",
      "   3   HLA-DQA10201-DQB10303        LVTLAILTALRLC    3   LAILTALRL     0.467        Sequence      0.002441    88.95       NA      0.230421       4132.65    68.37       \r\n",
      "   4   HLA-DQA10201-DQB10303       LLVTLAILTALRLC    4   LAILTALRL     0.347        Sequence      0.002462    88.84       NA      0.234402       3958.42    67.10       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.001195    93.82       NA      0.181579       7010.29    80.52       \r\n",
      "   2   HLA-DQA10201-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.002551    83.55       NA      0.245636       3505.37    56.83       \r\n",
      "   3   HLA-DQA10201-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001153    94.14       NA      0.222479       4503.48    66.18       \r\n",
      "   4   HLA-DQA10201-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.567        Sequence      0.001139    94.25       NA      0.210567       5122.97    70.69       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.002099    95.00       NA      0.203741       5515.66    82.00       \r\n",
      "   2   HLA-DQA10201-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.460        Sequence      0.005326    86.88       NA      0.241551       3663.78    71.48       \r\n",
      "   3   HLA-DQA10201-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.520        Sequence      0.002313    95.00       NA      0.225459       4360.59    76.16       \r\n",
      "   4   HLA-DQA10201-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.413        Sequence      0.002085    95.00       NA      0.239580       3742.75    72.06       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1104\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1104        VFLLVTLAILTAL    3   LVTLAILTA     0.887        Sequence      0.001473    74.21       NA      0.206929       5328.65    63.39       \r\n",
      "   2     DRB1_1104        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.001551    73.47       NA      0.294034       2076.41    40.79       \r\n",
      "   3     DRB1_1104        LVTLAILTALRLC    3   LAILTALRL     0.920        Sequence      0.001323    75.80       NA      0.265362       2831.67    47.81       \r\n",
      "   4     DRB1_1104       LLVTLAILTALRLC    4   LAILTALRL     0.873        Sequence      0.001107    78.43       NA      0.303997       1864.22    38.59       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1103\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1103        VFLLVTLAILTAL    2   LLVTLAILT     0.220        Sequence      0.000524    91.25       NA      0.174637       7557.11    77.33       \r\n",
      "   2     DRB1_1103        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.002015    74.10       NA      0.236959       3850.41    58.88       \r\n",
      "   3     DRB1_1103        LVTLAILTALRLC    3   LAILTALRL     0.940        Sequence      0.002253    72.38       NA      0.222852       4485.34    63.09       \r\n",
      "   4     DRB1_1103       LLVTLAILTALRLC    4   LAILTALRL     0.807        Sequence      0.002009    74.15       NA      0.248037       3415.48    55.61       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1102\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1102        VFLLVTLAILTAL    2   LLVTLAILT     0.320        Sequence      0.000339    95.00       NA      0.172065       7770.37    77.21       \r\n",
      "   2     DRB1_1102        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.001473    84.05       NA      0.278972       2443.93    48.77       \r\n",
      "   3     DRB1_1102        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.001370    84.93       NA      0.247780       3424.99    56.95       \r\n",
      "   4     DRB1_1102       LLVTLAILTALRLC    4   LAILTALRL     0.827        Sequence      0.001267    86.00       NA      0.273817       2584.12    50.03       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1101        VFLLVTLAILTAL    3   LVTLAILTA     0.753        Sequence      0.000312    92.99       NA      0.155653       9280.28    71.89       \r\n",
      "   2     DRB1_1101        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.000529    87.89       NA      0.239756       3735.63    46.80       \r\n",
      "   3     DRB1_1101        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000654    85.19       NA      0.210519       5125.63    54.99       \r\n",
      "   4     DRB1_1101       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.000541    87.63       NA      0.233903       3979.85    48.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB141401 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10105-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10105-DPB10301        VFLLVTLAILTAL    1   FLLVTLAIL     0.120        Sequence      0.004526    74.61       NA      0.186242       6665.37    49.30       \r\n",
      "   2   HLA-DPA10105-DPB10301        LLVTLAILTALRL    3   TLAILTALR     0.293        Sequence      0.003365    80.65       NA      0.203123       5552.66    42.87       \r\n",
      "   3   HLA-DPA10105-DPB10301        LVTLAILTALRLC    2   TLAILTALR     0.293        Sequence      0.002383    86.46       NA      0.174597       7560.39    53.99       \r\n",
      "   4   HLA-DPA10105-DPB10301       LLVTLAILTALRLC    3   TLAILTALR     0.233        Sequence      0.002574    85.27       NA      0.198681       5826.05    44.49       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB12101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB12101        VFLLVTLAILTAL    1   FLLVTLAIL     0.160        Sequence      0.002822    93.36       NA      0.207627       5288.55    35.95       \r\n",
      "   2   HLA-DPA10103-DPB12101        LLVTLAILTALRL    4   LAILTALRL     0.580        Sequence      0.005516    83.47       NA      0.229386       4179.19    28.16       \r\n",
      "   3   HLA-DPA10103-DPB12101        LVTLAILTALRLC    3   LAILTALRL     0.613        Sequence      0.003673    90.32       NA      0.188203       6525.44    43.89       \r\n",
      "   4   HLA-DPA10103-DPB12101       LLVTLAILTALRLC    4   LAILTALRL     0.560        Sequence      0.003262    91.79       NA      0.219529       4649.54    31.60       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10104-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10104-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.593        Sequence      0.000338    95.00       NA      0.118246      13910.27    75.08       \r\n",
      "   2   HLA-DQA10104-DQB10503        LLVTLAILTALRL    4   LAILTALRL     0.433        Sequence      0.000682    85.71       NA      0.152074       9646.70    59.35       \r\n",
      "   3   HLA-DQA10104-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.540        Sequence      0.000424    93.01       NA      0.133710      11767.13    67.91       \r\n",
      "   4   HLA-DQA10104-DQB10503       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000451    92.29       NA      0.144329      10489.92    62.93       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10503-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10503-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10503-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10503-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10503-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10104-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10104-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.000397    90.04       NA      0.177268       7345.02    54.97       \r\n",
      "   2   HLA-DQA10104-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.001629    53.06       NA      0.236905       3852.66    35.66       \r\n",
      "   3   HLA-DQA10104-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.000984    69.53       NA      0.196010       5996.87    48.46       \r\n",
      "   4   HLA-DQA10104-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.760        Sequence      0.000943    70.79       NA      0.202618       5583.08    46.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11801        VFLLVTLAILTAL    3   LVTLAILTA     0.560        Sequence      0.003368    30.07       NA      0.203683       5519.12    53.49       \r\n",
      "   2   HLA-DPA10201-DPB11801        LLVTLAILTALRL    4   LAILTALRL     0.727        Sequence      0.001918    47.78       NA      0.256799       3106.56    33.50       \r\n",
      "   3   HLA-DPA10201-DPB11801        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000757    76.48       NA      0.218863       4683.16    47.46       \r\n",
      "   4   HLA-DPA10201-DPB11801       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.000808    74.81       NA      0.247832       3423.06    36.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.260        Sequence      0.004287    84.33       NA      0.317754       1606.40    45.67       \r\n",
      "   2   HLA-DPA10401-DPB10202        LLVTLAILTALRL    4   LAILTALRL     0.753        Sequence      0.009150    67.22       NA      0.351384       1116.42    36.76       \r\n",
      "   3   HLA-DPA10401-DPB10202        LVTLAILTALRLC    3   LAILTALRL     0.773        Sequence      0.003740    86.60       NA      0.301372       1917.93    50.08       \r\n",
      "   4   HLA-DPA10401-DPB10202       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.004699    82.68       NA      0.333401       1356.22    41.45       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11401        VFLLVTLAILTAL    2   LLVTLAILT     0.607        Sequence      0.000978    94.99       NA      0.219700       4640.94    45.44       \r\n",
      "   2   HLA-DPA10202-DPB11401        LLVTLAILTALRL    4   LAILTALRL     0.627        Sequence      0.005291    70.10       NA      0.261796       2943.06    30.46       \r\n",
      "   3   HLA-DPA10202-DPB11401        LVTLAILTALRLC    3   LAILTALRL     0.727        Sequence      0.003164    81.23       NA      0.222250       4514.65    44.47       \r\n",
      "   4   HLA-DPA10202-DPB11401       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.002146    87.39       NA      0.248094       3413.37    35.09       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.293        Sequence      0.002956    88.16       NA      0.317506       1610.72    44.97       \r\n",
      "   2   HLA-DPA10401-DPB10201        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.006124    74.36       NA      0.322654       1523.45    43.59       \r\n",
      "   3   HLA-DPA10401-DPB10201        LVTLAILTALRLC    3   LAILTALRL     0.727        Sequence      0.002307    91.29       NA      0.285714       2272.00    53.73       \r\n",
      "   4   HLA-DPA10401-DPB10201       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.002747    89.15       NA      0.321850       1536.76    43.80       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.560        Sequence      0.000832    95.00       NA      0.212039       5042.03    57.61       \r\n",
      "   2   HLA-DQA10102-DQB10202        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.002478    76.22       NA      0.232515       4040.07    43.71       \r\n",
      "   3   HLA-DQA10102-DQB10202        LVTLAILTALRLC    3   LAILTALRL     0.713        Sequence      0.002717    73.57       NA      0.215885       4836.52    55.06       \r\n",
      "   4   HLA-DQA10102-DQB10202       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.002464    76.38       NA      0.228304       4228.40    46.65       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1201        VFLLVTLAILTAL    3   LVTLAILTA     0.793        Sequence      0.003243    86.18       NA      0.186276       6662.92    55.74       \r\n",
      "   2     DRB1_1201        LLVTLAILTALRL    4   LAILTALRL     0.987        Sequence      0.090386    17.77       NA      0.362348        991.53    12.31       \r\n",
      "   3     DRB1_1201        LVTLAILTALRLC    3   LAILTALRL     0.993        Sequence      0.081883    19.43       NA      0.318876       1587.02    19.62       \r\n",
      "   4     DRB1_1201       LLVTLAILTALRLC    4   LAILTALRL     0.993        Sequence      0.067205    22.88       NA      0.334667       1337.76    16.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.000355    95.00       NA      0.150214       9842.81    59.08       \r\n",
      "   2   HLA-DQA10101-DQB10201        LLVTLAILTALRL    4   LAILTALRL     0.620        Sequence      0.001135    84.81       NA      0.161665       8695.83    48.84       \r\n",
      "   3   HLA-DQA10101-DQB10201        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.000852    90.10       NA      0.146240      10275.26    62.53       \r\n",
      "   4   HLA-DQA10101-DQB10201       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000758    91.74       NA      0.160091       8845.19    50.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10402        VFLLVTLAILTAL    1   FLLVTLAIL     0.287        Sequence      0.000455    69.76       NA      0.171374       7828.68    38.17       \r\n",
      "   2   HLA-DPA10201-DPB10402        LLVTLAILTALRL    4   LAILTALRL     0.767        Sequence      0.000479    68.33       NA      0.201256       5665.97    26.46       \r\n",
      "   3   HLA-DPA10201-DPB10402        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000319    79.32       NA      0.167221       8188.48    40.01       \r\n",
      "   4   HLA-DPA10201-DPB10402       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.000338    77.88       NA      0.180280       7109.51    34.36       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10401        VFLLVTLAILTAL    1   FLLVTLAIL     0.367        Sequence      0.000296    82.12       NA      0.216790       4789.39    34.44       \r\n",
      "   2   HLA-DPA10201-DPB10401        LLVTLAILTALRL    4   LAILTALRL     0.800        Sequence      0.000868    51.84       NA      0.261201       2962.06    21.93       \r\n",
      "   3   HLA-DPA10201-DPB10401        LVTLAILTALRLC    3   LAILTALRL     0.873        Sequence      0.000453    71.38       NA      0.212038       5042.08    36.01       \r\n",
      "   4   HLA-DPA10201-DPB10401       LLVTLAILTALRLC    4   LAILTALRL     0.807        Sequence      0.000503    68.41       NA      0.230801       4115.69    30.10       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10202        VFLLVTLAILTAL    2   LLVTLAILT     0.320        Sequence      0.001564    91.31       NA      0.268448       2738.68    41.40       \r\n",
      "   2   HLA-DPA10103-DPB10202        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.003763    76.79       NA      0.302773       1889.07    32.96       \r\n",
      "   3   HLA-DPA10103-DPB10202        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.001881    89.02       NA      0.252909       3240.10    45.52       \r\n",
      "   4   HLA-DPA10103-DPB10202       LLVTLAILTALRLC    4   LAILTALRL     0.687        Sequence      0.001957    88.48       NA      0.284275       2307.65    37.39       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10201        VFLLVTLAILTAL    2   LLVTLAILT     0.313        Sequence      0.001408    90.19       NA      0.266507       2796.80    40.49       \r\n",
      "   2   HLA-DPA10103-DPB10201        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.001737    87.32       NA      0.271345       2654.17    39.24       \r\n",
      "   3   HLA-DPA10103-DPB10201        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.001255    91.58       NA      0.231716       4075.15    49.85       \r\n",
      "   4   HLA-DPA10103-DPB10201       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.001199    92.08       NA      0.261556       2950.71    41.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB13801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB13801        VFLLVTLAILTAL    3   LVTLAILTA     0.393        Sequence      0.002272    94.16       NA      0.247809       3423.91    52.88       \r\n",
      "   2   HLA-DPA10202-DPB13801        LLVTLAILTALRL    4   LAILTALRL     0.667        Sequence      0.008926    74.62       NA      0.322359       1528.32    29.92       \r\n",
      "   3   HLA-DPA10202-DPB13801        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.008125    76.73       NA      0.282201       2360.02    41.80       \r\n",
      "   4   HLA-DPA10202-DPB13801       LLVTLAILTALRLC    4   LAILTALRL     0.720        Sequence      0.006192    82.01       NA      0.301335       1918.69    36.03       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0803\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0803        VFLLVTLAILTAL    2   LLVTLAILT     0.400        Sequence      0.001795    95.00       NA      0.236718       3860.46    52.22       \r\n",
      "   2     DRB1_0803        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.017111    71.86       NA      0.327696       1442.57    21.27       \r\n",
      "   3     DRB1_0803        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.010932    79.86       NA      0.281855       2368.87    35.99       \r\n",
      "   4     DRB1_0803       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.008638    83.57       NA      0.306765       1809.22    27.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10609\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10609        VFLLVTLAILTAL    3   LVTLAILTA     0.407        Sequence      0.008344    92.50       NA      0.322566       1524.90    54.41       \r\n",
      "   2   HLA-DQA10102-DQB10609        LLVTLAILTALRL    4   LAILTALRL     0.600        Sequence      0.009914    90.67       NA      0.314513       1663.73    57.18       \r\n",
      "   3   HLA-DQA10102-DQB10609        LVTLAILTALRLC    3   LAILTALRL     0.640        Sequence      0.007105    93.95       NA      0.287341       2232.36    66.16       \r\n",
      "   4   HLA-DQA10102-DQB10609       LLVTLAILTALRLC    4   LAILTALRL     0.593        Sequence      0.006869    94.22       NA      0.306283       1818.68    60.01       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11401        VFLLVTLAILTAL    2   LLVTLAILT     0.607        Sequence      0.005053    91.55       NA      0.200472       5714.23    47.77       \r\n",
      "   2   HLA-DPA10103-DPB11401        LLVTLAILTALRL    4   LAILTALRL     0.493        Sequence      0.007353    87.10       NA      0.227987       4242.93    37.54       \r\n",
      "   3   HLA-DPA10103-DPB11401        LVTLAILTALRLC    2   TLAILTALR     0.293        Sequence      0.004276    93.05       NA      0.190564       6360.85    51.71       \r\n",
      "   4   HLA-DPA10103-DPB11401       LLVTLAILTALRLC    3   TLAILTALR     0.227        Sequence      0.004196    93.20       NA      0.222106       4521.69    39.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10104-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10104-DPB10301        VFLLVTLAILTAL    1   FLLVTLAIL     0.120        Sequence      0.004526    74.61       NA      0.186242       6665.37    49.30       \r\n",
      "   2   HLA-DPA10104-DPB10301        LLVTLAILTALRL    3   TLAILTALR     0.293        Sequence      0.003365    80.65       NA      0.203123       5552.66    42.87       \r\n",
      "   3   HLA-DPA10104-DPB10301        LVTLAILTALRLC    2   TLAILTALR     0.293        Sequence      0.002383    86.46       NA      0.174597       7560.39    53.99       \r\n",
      "   4   HLA-DPA10104-DPB10301       LLVTLAILTALRLC    3   TLAILTALR     0.233        Sequence      0.002574    85.27       NA      0.198681       5826.05    44.49       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB15501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB15501        VFLLVTLAILTAL    1   FLLVTLAIL     0.240        Sequence      0.009804    79.97       NA      0.276079       2521.64    28.58       \r\n",
      "   2   HLA-DPA10103-DPB15501        LLVTLAILTALRL    4   LAILTALRL     0.680        Sequence      0.008568    82.94       NA      0.297761       1994.34    22.74       \r\n",
      "   3   HLA-DPA10103-DPB15501        LVTLAILTALRLC    3   LAILTALRL     0.747        Sequence      0.005099    91.33       NA      0.244424       3551.64    38.46       \r\n",
      "   4   HLA-DPA10103-DPB15501       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.005299    90.87       NA      0.279207       2437.72    27.68       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB13001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB13001        VFLLVTLAILTAL    1   FLLVTLAIL     0.220        Sequence      0.002851    94.31       NA      0.289439       2182.25    24.11       \r\n",
      "   2   HLA-DPA10202-DPB13001        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.021527    44.58       NA      0.340973       1249.53    11.35       \r\n",
      "   3   HLA-DPA10202-DPB13001        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.016959    54.10       NA      0.294691       2061.70    22.55       \r\n",
      "   4   HLA-DPA10202-DPB13001       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.018384    50.87       NA      0.321341       1545.25    15.58       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10209-DPB10101 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DQA10102-DQB10504 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB146301 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.002117    88.98       NA      0.376193        853.59    52.62       \r\n",
      "   2   HLA-DQA10102-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.740        Sequence      0.006819    53.38       NA      0.499391        225.09    21.99       \r\n",
      "   3   HLA-DQA10102-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.005408    63.05       NA      0.456892        356.49    32.05       \r\n",
      "   4   HLA-DQA10102-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.673        Sequence      0.005691    60.94       NA      0.462324        336.14    30.71       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.687        Sequence      0.001452    93.62       NA      0.239412       3749.56    72.36       \r\n",
      "   2   HLA-DQA10102-DQB10503        LLVTLAILTALRL    4   LAILTALRL     0.420        Sequence      0.002968    82.13       NA      0.309255       1761.12    51.48       \r\n",
      "   3   HLA-DQA10102-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.513        Sequence      0.001280    94.84       NA      0.284284       2307.43    59.43       \r\n",
      "   4   HLA-DQA10102-DQB10503       LLVTLAILTALRLC    1   LVTLAILTA     0.267        Sequence      0.001696    91.90       NA      0.294078       2075.42    56.35       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10502        VFLLVTLAILTAL    3   LVTLAILTA     0.633        Sequence      0.000380    95.00       NA      0.150879       9772.24    70.45       \r\n",
      "   2   HLA-DQA10102-DQB10502        LLVTLAILTALRL    4   LAILTALRL     0.553        Sequence      0.000678    91.59       NA      0.174128       7598.85    61.66       \r\n",
      "   3   HLA-DQA10102-DQB10502        LVTLAILTALRLC    3   LAILTALRL     0.680        Sequence      0.000462    95.00       NA      0.147847      10098.14    71.58       \r\n",
      "   4   HLA-DQA10102-DQB10502       LLVTLAILTALRLC    4   LAILTALRL     0.607        Sequence      0.000475    95.00       NA      0.159810       8872.12    67.10       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11101        VFLLVTLAILTAL    2   LLVTLAILT     0.307        Sequence      0.001944    83.24       NA      0.312869       1693.59    27.52       \r\n",
      "   2   HLA-DPA10202-DPB11101        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.008759    40.80       NA      0.394131        703.01     8.45       \r\n",
      "   3   HLA-DPA10202-DPB11101        LVTLAILTALRLC    3   LAILTALRL     0.867        Sequence      0.006092    52.94       NA      0.342299       1231.73    19.12       \r\n",
      "   4   HLA-DPA10202-DPB11101       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.005581    55.73       NA      0.367865        934.08    13.21       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.473        Sequence      0.000267    95.00       NA      0.126197      12763.63    82.01       \r\n",
      "   2   HLA-DQA10303-DQB10202        LLVTLAILTALRL    4   LAILTALRL     0.620        Sequence      0.000440    91.94       NA      0.150158       9848.77    69.83       \r\n",
      "   3   HLA-DQA10303-DQB10202        LVTLAILTALRLC    3   LAILTALRL     0.660        Sequence      0.000264    95.00       NA      0.127979      12519.89    81.23       \r\n",
      "   4   HLA-DQA10303-DQB10202       LLVTLAILTALRLC    4   LAILTALRL     0.553        Sequence      0.000252    95.00       NA      0.124177      13045.66    82.89       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10501-DQB12010 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DQA10403N-DQB10402 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.480        Sequence      0.002032    50.88       NA      0.267939       2753.80    39.13       \r\n",
      "   2   HLA-DQA10501-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.002400    43.39       NA      0.289216       2187.52    28.86       \r\n",
      "   3   HLA-DQA10501-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.793        Sequence      0.002292    45.40       NA      0.266871       2785.81    39.67       \r\n",
      "   4   HLA-DQA10501-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.693        Sequence      0.002354    44.22       NA      0.287084       2238.57    29.85       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.002466    52.51       NA      0.198457       5840.18    59.19       \r\n",
      "   2   HLA-DQA10501-DQB10503        LLVTLAILTALRL    1   LVTLAILTA     0.200        Sequence      0.001631    69.52       NA      0.211452       5074.15    51.45       \r\n",
      "   3   HLA-DQA10501-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.727        Sequence      0.001050    83.70       NA      0.192758       6211.64    62.47       \r\n",
      "   4   HLA-DQA10501-DQB10503       LLVTLAILTALRLC    4   LAILTALRL     0.587        Sequence      0.001434    74.19       NA      0.221674       4542.87    45.29       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11101        VFLLVTLAILTAL    1   FLLVTLAIL     0.173        Sequence      0.004107    80.81       NA      0.245407       3514.07    38.58       \r\n",
      "   2   HLA-DPA10103-DPB11101        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.004034    81.20       NA      0.296141       2029.61    23.61       \r\n",
      "   3   HLA-DPA10103-DPB11101        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.003517    83.94       NA      0.259284       3024.14    34.09       \r\n",
      "   4   HLA-DPA10103-DPB11101       LLVTLAILTALRLC    4   LAILTALRL     0.680        Sequence      0.003085    86.29       NA      0.280773       2396.77    27.74       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.393        Sequence      0.002272    94.16       NA      0.247809       3423.91    52.88       \r\n",
      "   2   HLA-DPA10202-DPB10501        LLVTLAILTALRL    4   LAILTALRL     0.667        Sequence      0.008926    74.62       NA      0.322359       1528.32    29.92       \r\n",
      "   3   HLA-DPA10202-DPB10501        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.008125    76.73       NA      0.282201       2360.02    41.80       \r\n",
      "   4   HLA-DPA10202-DPB10501       LLVTLAILTALRLC    4   LAILTALRL     0.720        Sequence      0.006192    82.01       NA      0.301335       1918.69    36.03       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB14901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB14901        VFLLVTLAILTAL    2   LLVTLAILT     0.353        Sequence      0.001072    91.78       NA      0.228190       4233.62    54.00       \r\n",
      "   2   HLA-DPA10401-DPB14901        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.001227    90.23       NA      0.275519       2536.97    37.60       \r\n",
      "   3   HLA-DPA10401-DPB14901        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000718    95.00       NA      0.241902       3649.89    49.10       \r\n",
      "   4   HLA-DPA10401-DPB14901       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000688    95.00       NA      0.266381       2800.62    40.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB12901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB12901        VFLLVTLAILTAL    1   FLLVTLAIL     0.127        Sequence      0.001162    95.00       NA      0.178480       7249.33    53.70       \r\n",
      "   2   HLA-DPA10103-DPB12901        LLVTLAILTALRL    4   LAILTALRL     0.500        Sequence      0.000698    95.00       NA      0.185942       6687.04    50.52       \r\n",
      "   3   HLA-DPA10103-DPB12901        LVTLAILTALRLC    3   LAILTALRL     0.607        Sequence      0.000449    95.00       NA      0.157403       9106.22    62.85       \r\n",
      "   4   HLA-DPA10103-DPB12901       LLVTLAILTALRLC    4   LAILTALRL     0.507        Sequence      0.000452    95.00       NA      0.183596       6858.95    51.52       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB12301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB12301        VFLLVTLAILTAL    1   FLLVTLAIL     0.313        Sequence      0.006801    79.06       NA      0.386223        765.81    29.62       \r\n",
      "   2   HLA-DPA10301-DPB12301        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.024710    44.05       NA      0.454982        363.93    18.41       \r\n",
      "   3   HLA-DPA10301-DPB12301        LVTLAILTALRLC    3   LAILTALRL     0.833        Sequence      0.011187    67.53       NA      0.380633        813.56    30.66       \r\n",
      "   4   HLA-DPA10301-DPB12301       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.011799    66.08       NA      0.411267        584.03    25.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10105-DPB15001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10105-DPB15001        VFLLVTLAILTAL    3   LVTLAILTA     0.300        Sequence      0.006559    95.00       NA      0.230086       4147.66    76.19       \r\n",
      "   2   HLA-DPA10105-DPB15001        LLVTLAILTALRL    3   TLAILTALR     0.287        Sequence      0.011890    90.87       NA      0.287454       2229.63    59.41       \r\n",
      "   3   HLA-DPA10105-DPB15001        LVTLAILTALRLC    2   TLAILTALR     0.260        Sequence      0.007424    95.00       NA      0.256775       3107.36    68.76       \r\n",
      "   4   HLA-DPA10105-DPB15001       LLVTLAILTALRLC    3   TLAILTALR     0.193        Sequence      0.009362    93.39       NA      0.289500       2180.81    58.76       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11901        VFLLVTLAILTAL    3   LVTLAILTA     0.360        Sequence      0.007686    95.00       NA      0.235574       3908.54    60.65       \r\n",
      "   2   HLA-DPA10103-DPB11901        LLVTLAILTALRL    4   LAILTALRL     0.587        Sequence      0.011976    90.83       NA      0.279283       2435.72    46.61       \r\n",
      "   3   HLA-DPA10103-DPB11901        LVTLAILTALRLC    3   LAILTALRL     0.633        Sequence      0.006921    95.00       NA      0.244386       3553.10    57.81       \r\n",
      "   4   HLA-DPA10103-DPB11901       LLVTLAILTALRLC    4   LAILTALRL     0.507        Sequence      0.005708    95.00       NA      0.273861       2582.89    48.32       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB141601 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.005529    60.69       NA      0.195344       6040.24    54.54       \r\n",
      "   2   HLA-DQA10201-DQB10602        LLVTLAILTALRL    2   VTLAILTAL     0.260        Sequence      0.006436    55.29       NA      0.206261       5367.30    48.44       \r\n",
      "   3   HLA-DQA10201-DQB10602        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.002151    86.40       NA      0.184381       6800.94    60.42       \r\n",
      "   4   HLA-DQA10201-DQB10602       LLVTLAILTALRLC    4   LAILTALRL     0.407        Sequence      0.002711    81.60       NA      0.213518       4961.98    44.44       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10301-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10301-DQB10501        VFLLVTLAILTAL    2   LLVTLAILT     0.200        Sequence      0.000580    47.07       NA      0.194019       6127.46    58.94       \r\n",
      "   2   HLA-DQA10301-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.680        Sequence      0.000472    55.37       NA      0.209738       5169.13    49.71       \r\n",
      "   3   HLA-DQA10301-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.720        Sequence      0.000452    57.22       NA      0.194753       6078.99    58.51       \r\n",
      "   4   HLA-DQA10301-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.000450    57.41       NA      0.204898       5447.04    52.59       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10101        VFLLVTLAILTAL    3   LVTLAILTA     0.360        Sequence      0.027363    93.49       NA      0.401381        649.97    40.68       \r\n",
      "   2   HLA-DPA10301-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.066520    74.56       NA      0.468481        314.48    23.66       \r\n",
      "   3   HLA-DPA10301-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.043968    86.18       NA      0.418931        537.56    35.95       \r\n",
      "   4   HLA-DPA10301-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.700        Sequence      0.042547    86.85       NA      0.442677        415.76    29.84       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1601        VFLLVTLAILTAL    3   LVTLAILTA     0.233        Sequence      0.000370    94.96       NA      0.171939       7780.97    73.32       \r\n",
      "   2     DRB1_1601        LLVTLAILTALRL    4   LAILTALRL     0.973        Sequence      0.008843    45.22       NA      0.250945       3309.69    48.22       \r\n",
      "   3     DRB1_1601        LVTLAILTALRLC    3   LAILTALRL     1.000        Sequence      0.013568    37.95       NA      0.237883       3812.11    52.24       \r\n",
      "   4     DRB1_1601       LLVTLAILTALRLC    4   LAILTALRL     0.993        Sequence      0.008289    46.32       NA      0.264209       2867.21    44.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1602        VFLLVTLAILTAL    2   LLVTLAILT     0.567        Sequence      0.000586    94.25       NA      0.194020       6127.39    74.86       \r\n",
      "   2     DRB1_1602        LLVTLAILTALRL    4   LAILTALRL     0.947        Sequence      0.015885    39.67       NA      0.336481       1311.76    34.47       \r\n",
      "   3     DRB1_1602        LVTLAILTALRLC    3   LAILTALRL     0.980        Sequence      0.024488    32.17       NA      0.302891       1886.66    43.39       \r\n",
      "   4     DRB1_1602       LLVTLAILTALRLC    4   LAILTALRL     0.973        Sequence      0.014906    40.81       NA      0.319971       1568.32    38.72       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10609\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10609        VFLLVTLAILTAL    3   LVTLAILTA     0.473        Sequence      0.012089    92.71       NA      0.274554       2563.59    69.77       \r\n",
      "   2   HLA-DQA10103-DQB10609        LLVTLAILTALRL    4   LAILTALRL     0.560        Sequence      0.017011    88.60       NA      0.264876       2846.59    72.81       \r\n",
      "   3   HLA-DQA10103-DQB10609        LVTLAILTALRLC    3   LAILTALRL     0.647        Sequence      0.011884    92.88       NA      0.244151       3562.15    78.88       \r\n",
      "   4   HLA-DQA10103-DQB10609       LLVTLAILTALRLC    4   LAILTALRL     0.600        Sequence      0.010929    93.68       NA      0.263741       2881.77    73.16       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10608\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10608        VFLLVTLAILTAL    3   LVTLAILTA     0.607        Sequence      0.004959    91.82       NA      0.258079       3063.83    45.26       \r\n",
      "   2   HLA-DQA10103-DQB10608        LLVTLAILTALRL    4   LAILTALRL     0.573        Sequence      0.005586    90.50       NA      0.297037       2010.03    30.85       \r\n",
      "   3   HLA-DQA10103-DQB10608        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.004304    93.20       NA      0.268445       2738.77    41.20       \r\n",
      "   4   HLA-DQA10103-DQB10608       LLVTLAILTALRLC    4   LAILTALRL     0.533        Sequence      0.003633    94.62       NA      0.285170       2285.41    35.03       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10604\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10604        VFLLVTLAILTAL    3   LVTLAILTA     0.460        Sequence      0.005526    95.00       NA      0.234932       3935.79    65.21       \r\n",
      "   2   HLA-DQA10103-DQB10604        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.008414    91.67       NA      0.230865       4112.84    66.81       \r\n",
      "   3   HLA-DQA10103-DQB10604        LVTLAILTALRLC    3   LAILTALRL     0.800        Sequence      0.005927    94.88       NA      0.207577       5291.42    75.40       \r\n",
      "   4   HLA-DQA10103-DQB10604       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.004833    95.00       NA      0.230384       4134.30    67.00       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10601        VFLLVTLAILTAL    3   LVTLAILTA     0.600        Sequence      0.009334    95.00       NA      0.184688       6778.39    92.45       \r\n",
      "   2   HLA-DQA10103-DQB10601        LLVTLAILTALRL    2   VTLAILTAL     0.400        Sequence      0.010663    95.00       NA      0.195309       6042.53    90.73       \r\n",
      "   3   HLA-DQA10103-DQB10601        LVTLAILTALRLC    4   AILTALRLC     0.260        Sequence      0.006700    95.00       NA      0.181014       7053.27    93.05       \r\n",
      "   4   HLA-DQA10103-DQB10601       LLVTLAILTALRLC    5   AILTALRLC     0.153        Sequence      0.007178    95.00       NA      0.212805       5000.41    87.14       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10603\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10603        VFLLVTLAILTAL    3   LVTLAILTA     0.807        Sequence      0.001877    94.01       NA      0.167823       8135.32    61.57       \r\n",
      "   2   HLA-DQA10103-DQB10603        LLVTLAILTALRL    1   LVTLAILTA     0.300        Sequence      0.001885    93.98       NA      0.187703       6560.84    52.36       \r\n",
      "   3   HLA-DQA10103-DQB10603        LVTLAILTALRLC    2   TLAILTALR     0.267        Sequence      0.000922    95.00       NA      0.166463       8255.92    62.19       \r\n",
      "   4   HLA-DQA10103-DQB10603       LLVTLAILTALRLC    1   LVTLAILTA     0.340        Sequence      0.001152    95.00       NA      0.189003       6469.20    51.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.847        Sequence      0.003979    91.84       NA      0.169476       7991.11    79.56       \r\n",
      "   2   HLA-DQA10103-DQB10602        LLVTLAILTALRL    1   LVTLAILTA     0.300        Sequence      0.003035    94.11       NA      0.187656       6564.17    73.92       \r\n",
      "   3   HLA-DQA10103-DQB10602        LVTLAILTALRLC    4   AILTALRLC     0.293        Sequence      0.001207    95.00       NA      0.174437       7573.49    78.04       \r\n",
      "   4   HLA-DQA10103-DQB10602       LLVTLAILTALRLC    1   LVTLAILTA     0.280        Sequence      0.002086    95.00       NA      0.194679       6083.86    71.65       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11901        VFLLVTLAILTAL    1   FLLVTLAIL     0.200        Sequence      0.005015    93.15       NA      0.298754       1973.03    49.46       \r\n",
      "   2   HLA-DPA10202-DPB11901        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.014857    74.85       NA      0.347618       1162.85    34.60       \r\n",
      "   3   HLA-DPA10202-DPB11901        LVTLAILTALRLC    3   LAILTALRL     0.760        Sequence      0.011128    81.63       NA      0.308597       1773.71    46.44       \r\n",
      "   4   HLA-DPA10202-DPB11901       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.011757    80.47       NA      0.344157       1207.22    35.66       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10104-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10104-DPB11301        VFLLVTLAILTAL    1   FLLVTLAIL     0.233        Sequence      0.003490    92.68       NA      0.234270       3964.08    35.07       \r\n",
      "   2   HLA-DPA10104-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.687        Sequence      0.006689    82.27       NA      0.266248       2804.65    24.87       \r\n",
      "   3   HLA-DPA10104-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.003279    93.34       NA      0.217884       4733.03    41.12       \r\n",
      "   4   HLA-DPA10104-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.003535    92.54       NA      0.249316       3368.54    29.99       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB11901 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.480        Sequence      0.002032    50.88       NA      0.267939       2753.80    39.13       \r\n",
      "   2   HLA-DQA10505-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.002400    43.39       NA      0.289216       2187.52    28.86       \r\n",
      "   3   HLA-DQA10505-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.793        Sequence      0.002292    45.40       NA      0.266871       2785.81    39.67       \r\n",
      "   4   HLA-DQA10505-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.693        Sequence      0.002354    44.22       NA      0.287084       2238.57    29.85       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10303-DQB10342 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10401-DPB146301 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0804\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0804        VFLLVTLAILTAL    3   LVTLAILTA     0.847        Sequence      0.001323    85.08       NA      0.210202       5143.24    62.21       \r\n",
      "   2     DRB1_0804        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.001513    83.58       NA      0.276148       2519.76    42.93       \r\n",
      "   3     DRB1_0804        LVTLAILTALRLC    3   LAILTALRL     0.880        Sequence      0.001321    85.10       NA      0.243088       3603.35    52.45       \r\n",
      "   4     DRB1_0804       LLVTLAILTALRLC    4   LAILTALRL     0.727        Sequence      0.001128    87.04       NA      0.271753       2642.47    44.15       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB113101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10103-DPB113101        VFLLVTLAILTAL    2   LLVTLAILT     0.640        Sequence      0.002130    93.90       NA      0.201243       5666.76    38.11       \r\n",
      "   2  HLA-DPA10103-DPB113101        LLVTLAILTALRL    4   LAILTALRL     0.527        Sequence      0.004486    83.26       NA      0.209474       5183.92    34.96       \r\n",
      "   3  HLA-DPA10103-DPB113101        LVTLAILTALRLC    3   LAILTALRL     0.587        Sequence      0.003066    89.72       NA      0.174291       7585.46    49.76       \r\n",
      "   4  HLA-DPA10103-DPB113101       LLVTLAILTALRLC    4   LAILTALRL     0.507        Sequence      0.002360    92.87       NA      0.207620       5288.95    35.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0806\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0806        VFLLVTLAILTAL    3   LVTLAILTA     0.747        Sequence      0.002325    89.46       NA      0.216300       4814.85    68.99       \r\n",
      "   2     DRB1_0806        LLVTLAILTALRL    4   LAILTALRL     0.787        Sequence      0.005581    77.98       NA      0.269554       2706.10    53.47       \r\n",
      "   3     DRB1_0806        LVTLAILTALRLC    3   LAILTALRL     0.913        Sequence      0.004566    81.01       NA      0.248771       3388.46    59.58       \r\n",
      "   4     DRB1_0806       LLVTLAILTALRLC    4   LAILTALRL     0.867        Sequence      0.003456    84.83       NA      0.272527       2620.44    52.60       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0801        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.000945    95.00       NA      0.181881       6987.42    74.76       \r\n",
      "   2     DRB1_0801        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.001945    89.90       NA      0.217940       4730.17    63.99       \r\n",
      "   3     DRB1_0801        LVTLAILTALRLC    3   LAILTALRL     0.827        Sequence      0.001377    93.11       NA      0.195913       6003.17    70.65       \r\n",
      "   4     DRB1_0801       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.001227    93.96       NA      0.210227       5141.85    66.35       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.000397    90.04       NA      0.177268       7345.02    54.97       \r\n",
      "   2   HLA-DQA10101-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.001629    53.06       NA      0.236905       3852.66    35.66       \r\n",
      "   3   HLA-DQA10101-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.000984    69.53       NA      0.196010       5996.87    48.46       \r\n",
      "   4   HLA-DQA10101-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.760        Sequence      0.000943    70.79       NA      0.202618       5583.08    46.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10502        VFLLVTLAILTAL    3   LVTLAILTA     0.553        Sequence      0.000189    95.00       NA      0.084717      19993.43    77.11       \r\n",
      "   2   HLA-DQA10101-DQB10502        LLVTLAILTALRL    4   LAILTALRL     0.587        Sequence      0.000287    93.57       NA      0.096132      17670.49    69.27       \r\n",
      "   3   HLA-DQA10101-DQB10502        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000219    95.00       NA      0.086433      19625.64    75.97       \r\n",
      "   4   HLA-DQA10101-DQB10502       LLVTLAILTALRLC    4   LAILTALRL     0.607        Sequence      0.000234    95.00       NA      0.093966      18089.50    70.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.593        Sequence      0.000338    95.00       NA      0.118246      13910.27    75.08       \r\n",
      "   2   HLA-DQA10101-DQB10503        LLVTLAILTALRL    4   LAILTALRL     0.433        Sequence      0.000682    85.71       NA      0.152074       9646.70    59.35       \r\n",
      "   3   HLA-DQA10101-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.540        Sequence      0.000424    93.01       NA      0.133710      11767.13    67.91       \r\n",
      "   4   HLA-DQA10101-DQB10503       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000451    92.29       NA      0.144329      10489.92    62.93       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10401-DQB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10401-DQB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001869    95.00       NA      0.205264       5425.51    65.81       \r\n",
      "   2   HLA-DQA10401-DQB10201        LLVTLAILTALRL    4   LAILTALRL     0.767        Sequence      0.008516    81.59       NA      0.270953       2665.45    31.54       \r\n",
      "   3   HLA-DQA10401-DQB10201        LVTLAILTALRLC    3   LAILTALRL     0.760        Sequence      0.005250    88.91       NA      0.238014       3806.71    48.65       \r\n",
      "   4   HLA-DQA10401-DQB10201       LLVTLAILTALRLC    4   LAILTALRL     0.693        Sequence      0.004475    90.78       NA      0.229152       4189.78    53.36       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10604\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10604        VFLLVTLAILTAL    3   LVTLAILTA     0.387        Sequence      0.005481    90.59       NA      0.284930       2291.36    44.52       \r\n",
      "   2   HLA-DQA10102-DQB10604        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.005267    91.06       NA      0.272503       2621.12    49.38       \r\n",
      "   3   HLA-DQA10102-DQB10604        LVTLAILTALRLC    3   LAILTALRL     0.773        Sequence      0.003473    95.00       NA      0.242289       3634.64    61.25       \r\n",
      "   4   HLA-DQA10102-DQB10604       LLVTLAILTALRLC    4   LAILTALRL     0.680        Sequence      0.003130    95.00       NA      0.270679       2673.36    50.08       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.827        Sequence      0.002987    90.97       NA      0.192387       6236.62    74.27       \r\n",
      "   2   HLA-DQA10102-DQB10602        LLVTLAILTALRL    1   LVTLAILTA     0.307        Sequence      0.001691    95.00       NA      0.220562       4597.86    65.02       \r\n",
      "   3   HLA-DQA10102-DQB10602        LVTLAILTALRLC    2   TLAILTALR     0.160        Sequence      0.000689    95.00       NA      0.219918       4630.01    65.23       \r\n",
      "   4   HLA-DQA10102-DQB10602       LLVTLAILTALRLC    1   LVTLAILTA     0.287        Sequence      0.001238    95.00       NA      0.234321       3961.89    60.29       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10603\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10603        VFLLVTLAILTAL    3   LVTLAILTA     0.773        Sequence      0.001650    92.20       NA      0.183450       6869.80    64.62       \r\n",
      "   2   HLA-DQA10102-DQB10603        LLVTLAILTALRL    1   LVTLAILTA     0.253        Sequence      0.000983    95.00       NA      0.213791       4947.35    50.37       \r\n",
      "   3   HLA-DQA10102-DQB10603        LVTLAILTALRLC    3   LAILTALRL     0.413        Sequence      0.000543    95.00       NA      0.195212       6048.88    59.15       \r\n",
      "   4   HLA-DQA10102-DQB10603       LLVTLAILTALRLC    1   LVTLAILTA     0.253        Sequence      0.000755    95.00       NA      0.210911       5103.94    51.74       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10601        VFLLVTLAILTAL    4   VTLAILTAL     0.273        Sequence      0.004423    95.00       NA      0.222276       4513.38    85.89       \r\n",
      "   2   HLA-DQA10102-DQB10601        LLVTLAILTALRL    2   VTLAILTAL     0.413        Sequence      0.006151    95.00       NA      0.237779       3816.40    81.93       \r\n",
      "   3   HLA-DQA10102-DQB10601        LVTLAILTALRLC    4   AILTALRLC     0.273        Sequence      0.004902    95.00       NA      0.220907       4580.73    86.20       \r\n",
      "   4   HLA-DQA10102-DQB10601       LLVTLAILTALRLC    5   AILTALRLC     0.193        Sequence      0.005002    95.00       NA      0.247045       3452.34    79.40       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB113301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10103-DPB113301        VFLLVTLAILTAL    1   FLLVTLAIL     0.233        Sequence      0.003490    92.68       NA      0.234270       3964.08    35.07       \r\n",
      "   2  HLA-DPA10103-DPB113301        LLVTLAILTALRL    4   LAILTALRL     0.687        Sequence      0.006689    82.27       NA      0.266248       2804.65    24.87       \r\n",
      "   3  HLA-DPA10103-DPB113301        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.003279    93.34       NA      0.217884       4733.03    41.12       \r\n",
      "   4  HLA-DPA10103-DPB113301       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.003535    92.54       NA      0.249316       3368.54    29.99       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1403\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1403        VFLLVTLAILTAL    3   LVTLAILTA     0.387        Sequence      0.003028    89.08       NA      0.209478       5183.69    49.66       \r\n",
      "   2     DRB1_1403        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.006704    79.09       NA      0.251331       3295.89    34.81       \r\n",
      "   3     DRB1_1403        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.007498    77.38       NA      0.216740       4791.98    46.88       \r\n",
      "   4     DRB1_1403       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.005676    81.54       NA      0.249427       3364.50    35.42       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10505-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10505-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10505-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10502        VFLLVTLAILTAL    3   LVTLAILTA     0.633        Sequence      0.000303    95.00       NA      0.119383      13740.19    80.72       \r\n",
      "   2   HLA-DQA10103-DQB10502        LLVTLAILTALRL    4   LAILTALRL     0.580        Sequence      0.000620    94.25       NA      0.133617      11778.98    74.50       \r\n",
      "   3   HLA-DQA10103-DQB10502        LVTLAILTALRLC    3   LAILTALRL     0.680        Sequence      0.000461    95.00       NA      0.109215      15338.12    84.86       \r\n",
      "   4   HLA-DQA10103-DQB10502       LLVTLAILTALRLC    4   LAILTALRL     0.600        Sequence      0.000408    95.00       NA      0.120858      13522.65    80.12       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10503        VFLLVTLAILTAL    3   LVTLAILTA     0.720        Sequence      0.001624    90.65       NA      0.185707       6704.07    76.39       \r\n",
      "   2   HLA-DQA10103-DQB10503        LLVTLAILTALRL    4   LAILTALRL     0.480        Sequence      0.002691    81.64       NA      0.234872       3938.34    58.17       \r\n",
      "   3   HLA-DQA10103-DQB10503        LVTLAILTALRLC    3   LAILTALRL     0.587        Sequence      0.001749    89.62       NA      0.207384       5302.48    68.79       \r\n",
      "   4   HLA-DQA10103-DQB10503       LLVTLAILTALRLC    4   LAILTALRL     0.467        Sequence      0.001903    88.31       NA      0.230000       4151.52    60.13       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.500        Sequence      0.001272    95.00       NA      0.302291       1898.95    56.85       \r\n",
      "   2   HLA-DQA10103-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.003944    76.46       NA      0.391313        724.77    30.40       \r\n",
      "   3   HLA-DQA10103-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.003287    81.54       NA      0.351732       1112.22    41.88       \r\n",
      "   4   HLA-DQA10103-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.687        Sequence      0.002723    85.85       NA      0.372793        885.58    35.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0403\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0403        VFLLVTLAILTAL    3   LVTLAILTA     0.867        Sequence      0.001867    82.27       NA      0.145262      10384.56    73.23       \r\n",
      "   2     DRB1_0403        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.002993    75.61       NA      0.193876       6136.95    49.71       \r\n",
      "   3     DRB1_0403        LVTLAILTALRLC    3   LAILTALRL     0.947        Sequence      0.001512    84.72       NA      0.166872       8219.46    62.86       \r\n",
      "   4     DRB1_0403       LLVTLAILTALRLC    4   LAILTALRL     0.827        Sequence      0.001276    86.72       NA      0.189954       6402.98    51.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10304\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10304        VFLLVTLAILTAL    3   LVTLAILTA     0.313        Sequence      0.000286    95.00       NA      0.142769      10668.49    94.59       \r\n",
      "   2   HLA-DQA10302-DQB10304        LLVTLAILTALRL    4   LAILTALRL     0.513        Sequence      0.000648    91.00       NA      0.172357       7745.86    87.88       \r\n",
      "   3   HLA-DQA10302-DQB10304        LVTLAILTALRLC    3   LAILTALRL     0.500        Sequence      0.000348    95.00       NA      0.158292       9019.05    91.37       \r\n",
      "   4   HLA-DQA10302-DQB10304       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000379    95.00       NA      0.153343       9515.15    92.40       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1405\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1405        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001106    94.32       NA      0.159374       8914.08    73.57       \r\n",
      "   2     DRB1_1405        LLVTLAILTALRL    4   LAILTALRL     0.827        Sequence      0.013421    63.18       NA      0.230292       4138.42    44.80       \r\n",
      "   3     DRB1_1405        LVTLAILTALRLC    3   LAILTALRL     0.913        Sequence      0.009661    69.15       NA      0.207046       5321.90    54.17       \r\n",
      "   4     DRB1_1405       LLVTLAILTALRLC    4   LAILTALRL     0.880        Sequence      0.008219    72.00       NA      0.222624       4496.42    47.89       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10301        VFLLVTLAILTAL    4   VTLAILTAL     0.380        Sequence      0.000467    92.35       NA      0.138834      11132.51    93.30       \r\n",
      "   2   HLA-DQA10302-DQB10301        LLVTLAILTALRL    2   VTLAILTAL     0.360        Sequence      0.000733    84.09       NA      0.164948       8392.36    86.62       \r\n",
      "   3   HLA-DQA10302-DQB10301        LVTLAILTALRLC    1   VTLAILTAL     0.253        Sequence      0.000390    94.64       NA      0.157508       9095.88    88.87       \r\n",
      "   4   HLA-DQA10302-DQB10301       LLVTLAILTALRLC    2   VTLAILTAL     0.307        Sequence      0.000558    89.62       NA      0.162200       8645.64    87.45       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10601        VFLLVTLAILTAL    2   LLVTLAILT     0.467        Sequence      0.000579    95.00       NA      0.194229       6113.55    44.75       \r\n",
      "   2   HLA-DPA10201-DPB10601        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.001638    83.47       NA      0.202222       5607.06    41.40       \r\n",
      "   3   HLA-DPA10201-DPB10601        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.001049    90.48       NA      0.169990       7946.80    55.35       \r\n",
      "   4   HLA-DPA10201-DPB10601       LLVTLAILTALRLC    4   LAILTALRL     0.713        Sequence      0.001140    89.38       NA      0.201700       5638.81    41.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10303        VFLLVTLAILTAL    4   VTLAILTAL     0.253        Sequence      0.000351    90.83       NA      0.132050      11980.39    90.30       \r\n",
      "   2   HLA-DQA10302-DQB10303        LLVTLAILTALRL    2   VTLAILTAL     0.180        Sequence      0.000479    85.98       NA      0.150053       9859.97    85.26       \r\n",
      "   3   HLA-DQA10302-DQB10303        LVTLAILTALRLC    1   VTLAILTAL     0.147        Sequence      0.000324    91.86       NA      0.142978      10644.39    87.29       \r\n",
      "   4   HLA-DQA10302-DQB10303       LLVTLAILTALRLC    4   LAILTALRL     0.333        Sequence      0.000297    92.88       NA      0.142647      10682.58    87.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.000257    90.85       NA      0.129250      12348.90    86.43       \r\n",
      "   2   HLA-DQA10302-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.567        Sequence      0.000692    70.34       NA      0.157801       9067.09    76.24       \r\n",
      "   3   HLA-DQA10302-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.493        Sequence      0.000360    85.57       NA      0.143130      10626.90    81.82       \r\n",
      "   4   HLA-DQA10302-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.487        Sequence      0.000363    85.41       NA      0.137816      11255.81    83.69       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10301-DPB141401 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB12901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB12901        VFLLVTLAILTAL    2   LLVTLAILT     0.473        Sequence      0.000920    95.00       NA      0.239145       3760.41    44.10       \r\n",
      "   2   HLA-DPA10202-DPB12901        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.006517    59.37       NA      0.251722       3281.98    38.90       \r\n",
      "   3   HLA-DPA10202-DPB12901        LVTLAILTALRLC    3   LAILTALRL     0.713        Sequence      0.004320    72.54       NA      0.216562       4801.22    53.48       \r\n",
      "   4   HLA-DPA10202-DPB12901       LLVTLAILTALRLC    4   LAILTALRL     0.640        Sequence      0.005034    67.93       NA      0.249165       3374.05    39.95       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10301-DPB146301 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB18001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB18001        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000950    85.82       NA      0.185947       6686.68    45.75       \r\n",
      "   2   HLA-DPA10103-DPB18001        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.000977    85.35       NA      0.216799       4788.92    35.03       \r\n",
      "   3   HLA-DPA10103-DPB18001        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.000397    95.00       NA      0.183982       6830.37    46.48       \r\n",
      "   4   HLA-DPA10103-DPB18001       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000554    92.82       NA      0.203954       5502.96    39.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10105-DQB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10105-DQB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.000397    90.04       NA      0.177268       7345.02    54.97       \r\n",
      "   2   HLA-DQA10105-DQB10501        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.001629    53.06       NA      0.236905       3852.66    35.66       \r\n",
      "   3   HLA-DQA10105-DQB10501        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.000984    69.53       NA      0.196010       5996.87    48.46       \r\n",
      "   4   HLA-DQA10105-DQB10501       LLVTLAILTALRLC    4   LAILTALRL     0.760        Sequence      0.000943    70.79       NA      0.202618       5583.08    46.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10301        VFLLVTLAILTAL    1   FLLVTLAIL     0.153        Sequence      0.002610    94.88       NA      0.202531       5588.34    52.76       \r\n",
      "   2   HLA-DPA10301-DPB10301        LLVTLAILTALRL    4   LAILTALRL     0.473        Sequence      0.009094    77.18       NA      0.250944       3309.72    34.69       \r\n",
      "   3   HLA-DPA10301-DPB10301        LVTLAILTALRLC    3   LAILTALRL     0.653        Sequence      0.007374    81.77       NA      0.213288       4974.35    48.46       \r\n",
      "   4   HLA-DPA10301-DPB10301       LLVTLAILTALRLC    4   LAILTALRL     0.513        Sequence      0.005210    87.77       NA      0.237922       3810.50    39.30       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB14001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB14001        VFLLVTLAILTAL    3   LVTLAILTA     0.553        Sequence      0.006636    82.01       NA      0.342498       1229.09    45.86       \r\n",
      "   2   HLA-DPA10301-DPB14001        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.023281    43.16       NA      0.410363        589.78    30.56       \r\n",
      "   3   HLA-DPA10301-DPB14001        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.009560    73.03       NA      0.333782       1350.64    47.88       \r\n",
      "   4   HLA-DPA10301-DPB14001       LLVTLAILTALRLC    4   LAILTALRL     0.727        Sequence      0.010797    69.43       NA      0.368105        931.66    39.88       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10501        VFLLVTLAILTAL    2   LLVTLAILT     0.407        Sequence      0.003311    95.00       NA      0.197822       5880.45    62.90       \r\n",
      "   2   HLA-DPA10103-DPB10501        LLVTLAILTALRL    4   LAILTALRL     0.500        Sequence      0.003635    95.00       NA      0.255572       3148.07    43.13       \r\n",
      "   3   HLA-DPA10103-DPB10501        LVTLAILTALRLC    3   LAILTALRL     0.553        Sequence      0.001973    95.00       NA      0.221193       4566.58    54.85       \r\n",
      "   4   HLA-DPA10103-DPB10501       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.001798    95.00       NA      0.247198       3446.62    46.01       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB110601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10202-DPB110601        VFLLVTLAILTAL    1   FLLVTLAIL     0.200        Sequence      0.005015    93.15       NA      0.298754       1973.03    49.46       \r\n",
      "   2  HLA-DPA10202-DPB110601        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.014857    74.85       NA      0.347618       1162.85    34.60       \r\n",
      "   3  HLA-DPA10202-DPB110601        LVTLAILTALRLC    3   LAILTALRL     0.760        Sequence      0.011128    81.63       NA      0.308597       1773.71    46.44       \r\n",
      "   4  HLA-DPA10202-DPB110601       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.011757    80.47       NA      0.344157       1207.22    35.66       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11001        VFLLVTLAILTAL    3   LVTLAILTA     0.220        Sequence      0.000339    95.00       NA      0.148171      10062.80    49.90       \r\n",
      "   2   HLA-DPA10201-DPB11001        LLVTLAILTALRL    4   LAILTALRL     0.607        Sequence      0.001338    84.34       NA      0.169781       7964.79    39.35       \r\n",
      "   3   HLA-DPA10201-DPB11001        LVTLAILTALRLC    3   LAILTALRL     0.687        Sequence      0.001007    89.22       NA      0.141169      10854.78    53.65       \r\n",
      "   4   HLA-DPA10201-DPB11001       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.001126    87.46       NA      0.160010       8852.95    43.96       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB141401 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0802\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0802        VFLLVTLAILTAL    3   LVTLAILTA     0.633        Sequence      0.000768    92.82       NA      0.185524       6717.35    69.32       \r\n",
      "   2     DRB1_0802        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.000931    91.18       NA      0.246695       3465.43    49.88       \r\n",
      "   3     DRB1_0802        LVTLAILTALRLC    3   LAILTALRL     0.793        Sequence      0.000725    93.25       NA      0.218113       4721.32    58.94       \r\n",
      "   4     DRB1_0802       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.000616    94.35       NA      0.235958       3892.34    53.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10104-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10104-DPB10101        VFLLVTLAILTAL    1   FLLVTLAIL     0.213        Sequence      0.005744    95.00       NA      0.281147       2387.09    57.34       \r\n",
      "   2   HLA-DPA10104-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.010692    92.98       NA      0.340886       1250.71    39.61       \r\n",
      "   3   HLA-DPA10104-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.006002    95.00       NA      0.298946       1968.94    52.04       \r\n",
      "   4   HLA-DPA10104-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.005666    95.00       NA      0.323137       1515.51    44.91       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10609\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10609        VFLLVTLAILTAL    1   FLLVTLAIL     0.260        Sequence      0.003642    91.71       NA      0.196838       5943.39    58.14       \r\n",
      "   2   HLA-DQA10101-DQB10609        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.003691    91.56       NA      0.201916       5625.65    55.43       \r\n",
      "   3   HLA-DQA10101-DQB10609        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.002805    94.22       NA      0.183612       6857.77    65.07       \r\n",
      "   4   HLA-DQA10101-DQB10609       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.002553    94.97       NA      0.197048       5929.90    58.02       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0301        VFLLVTLAILTAL    3   LVTLAILTA     0.600        Sequence      0.000382    91.27       NA      0.100431      16867.38    86.01       \r\n",
      "   2     DRB1_0301        LLVTLAILTALRL    4   LAILTALRL     0.640        Sequence      0.000797    83.15       NA      0.168935       8038.03    58.70       \r\n",
      "   3     DRB1_0301        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.000954    80.52       NA      0.145798      10324.51    68.48       \r\n",
      "   4     DRB1_0301       LLVTLAILTALRLC    4   LAILTALRL     0.720        Sequence      0.000666    85.42       NA      0.149349       9935.36    66.97       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB15701 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10301        VFLLVTLAILTAL    1   FLLVTLAIL     0.133        Sequence      0.006509    79.50       NA      0.269796       2699.02    46.18       \r\n",
      "   2   HLA-DPA10401-DPB10301        LLVTLAILTALRL    3   TLAILTALR     0.220        Sequence      0.004819    85.00       NA      0.280800       2396.07    42.40       \r\n",
      "   3   HLA-DPA10401-DPB10301        LVTLAILTALRLC    2   TLAILTALR     0.207        Sequence      0.002894    91.55       NA      0.244492       3549.03    55.09       \r\n",
      "   4   HLA-DPA10401-DPB10301       LLVTLAILTALRLC    3   TLAILTALR     0.173        Sequence      0.003486    89.46       NA      0.282616       2349.45    41.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB113301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10201-DPB113301        VFLLVTLAILTAL    2   LLVTLAILT     0.467        Sequence      0.001880    87.67       NA      0.229916       4155.29    32.63       \r\n",
      "   2  HLA-DPA10201-DPB113301        LLVTLAILTALRL    4   LAILTALRL     0.800        Sequence      0.006203    56.95       NA      0.259081       3030.79    23.59       \r\n",
      "   3  HLA-DPA10201-DPB113301        LVTLAILTALRLC    3   LAILTALRL     0.800        Sequence      0.003160    77.03       NA      0.205589       5406.47    41.51       \r\n",
      "   4  HLA-DPA10201-DPB113301       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.003031    78.08       NA      0.236816       3856.37    30.34       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB110501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10103-DPB110501        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000950    85.82       NA      0.185947       6686.68    45.75       \r\n",
      "   2  HLA-DPA10103-DPB110501        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.000977    85.35       NA      0.216799       4788.92    35.03       \r\n",
      "   3  HLA-DPA10103-DPB110501        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.000397    95.00       NA      0.183982       6830.37    46.48       \r\n",
      "   4  HLA-DPA10103-DPB110501       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000554    92.82       NA      0.203954       5502.96    39.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0410\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0410        VFLLVTLAILTAL    3   LVTLAILTA     0.793        Sequence      0.005642    64.53       NA      0.225054       4379.74    69.00       \r\n",
      "   2     DRB1_0410        LLVTLAILTALRL    4   LAILTALRL     0.827        Sequence      0.008018    57.61       NA      0.333055       1361.30    38.46       \r\n",
      "   3     DRB1_0410        LVTLAILTALRLC    3   LAILTALRL     0.973        Sequence      0.005474    65.07       NA      0.290332       2161.27    50.50       \r\n",
      "   4     DRB1_0410       LLVTLAILTALRLC    4   LAILTALRL     0.880        Sequence      0.004730    68.05       NA      0.310724       1733.35    44.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0411\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0411        VFLLVTLAILTAL    3   LVTLAILTA     0.760        Sequence      0.003228    81.69       NA      0.146822      10210.75    75.26       \r\n",
      "   2     DRB1_0411        LLVTLAILTALRL    4   LAILTALRL     0.787        Sequence      0.004523    76.44       NA      0.183134       6893.33    57.63       \r\n",
      "   3     DRB1_0411        LVTLAILTALRLC    3   LAILTALRL     0.967        Sequence      0.002864    83.37       NA      0.158123       9035.55    70.11       \r\n",
      "   4     DRB1_0411       LLVTLAILTALRLC    4   LAILTALRL     0.853        Sequence      0.002140    87.08       NA      0.180414       7099.21    59.01       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB11801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB11801        VFLLVTLAILTAL    3   LVTLAILTA     0.607        Sequence      0.012622    77.61       NA      0.346384       1178.48    48.30       \r\n",
      "   2   HLA-DPA10301-DPB11801        LLVTLAILTALRL    4   LAILTALRL     0.680        Sequence      0.024826    58.16       NA      0.426069        497.60    29.08       \r\n",
      "   3   HLA-DPA10301-DPB11801        LVTLAILTALRLC    3   LAILTALRL     0.793        Sequence      0.016712    70.46       NA      0.373089        882.75    41.77       \r\n",
      "   4   HLA-DPA10301-DPB11801       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.017863    68.56       NA      0.402111        644.86    34.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10401        VFLLVTLAILTAL    1   FLLVTLAIL     0.313        Sequence      0.006801    79.06       NA      0.386223        765.81    29.62       \r\n",
      "   2   HLA-DPA10301-DPB10401        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.024710    44.05       NA      0.454982        363.93    18.41       \r\n",
      "   3   HLA-DPA10301-DPB10401        LVTLAILTALRLC    3   LAILTALRL     0.833        Sequence      0.011187    67.53       NA      0.380633        813.56    30.66       \r\n",
      "   4   HLA-DPA10301-DPB10401       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.011799    66.08       NA      0.411267        584.03    25.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB11401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB11401        VFLLVTLAILTAL    2   LLVTLAILT     0.580        Sequence      0.008896    91.22       NA      0.275557       2535.92    47.82       \r\n",
      "   2   HLA-DPA10401-DPB11401        LLVTLAILTALRL    4   LAILTALRL     0.533        Sequence      0.006710    93.86       NA      0.298776       1972.56    39.65       \r\n",
      "   3   HLA-DPA10401-DPB11401        LVTLAILTALRLC    3   LAILTALRL     0.633        Sequence      0.003818    95.00       NA      0.257701       3076.39    54.30       \r\n",
      "   4   HLA-DPA10401-DPB11401       LLVTLAILTALRLC    4   LAILTALRL     0.513        Sequence      0.003669    95.00       NA      0.300099       1944.53    39.19       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10901        VFLLVTLAILTAL    3   LVTLAILTA     0.320        Sequence      0.022318    89.30       NA      0.278404       2459.00    29.23       \r\n",
      "   2   HLA-DPA10301-DPB10901        LLVTLAILTALRL    4   LAILTALRL     0.653        Sequence      0.038944    76.24       NA      0.304604       1852.02    21.47       \r\n",
      "   3   HLA-DPA10301-DPB10901        LVTLAILTALRLC    3   LAILTALRL     0.687        Sequence      0.025462    86.99       NA      0.266998       2781.98    33.07       \r\n",
      "   4   HLA-DPA10301-DPB10901       LLVTLAILTALRLC    4   LAILTALRL     0.593        Sequence      0.027931    85.17       NA      0.300006       1946.48    22.74       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10101        VFLLVTLAILTAL    3   LVTLAILTA     0.347        Sequence      0.009001    95.00       NA      0.331492       1384.52    62.13       \r\n",
      "   2   HLA-DPA10401-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.019007    89.95       NA      0.387516        755.17    44.91       \r\n",
      "   3   HLA-DPA10401-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.627        Sequence      0.009994    95.00       NA      0.352337       1104.97    55.91       \r\n",
      "   4   HLA-DPA10401-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.580        Sequence      0.009359    95.00       NA      0.374102        873.12    49.12       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10501-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10501-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10501-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB15901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB15901        VFLLVTLAILTAL    3   LVTLAILTA     0.413        Sequence      0.001637    75.09       NA      0.191045       6327.84    59.41       \r\n",
      "   2   HLA-DPA10103-DPB15901        LLVTLAILTALRL    1   LVTLAILTA     0.167        Sequence      0.000966    85.47       NA      0.214912       4887.70    50.95       \r\n",
      "   3   HLA-DPA10103-DPB15901        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.000390    95.00       NA      0.186196       6668.69    61.11       \r\n",
      "   4   HLA-DPA10103-DPB15901       LLVTLAILTALRLC    1   LVTLAILTA     0.160        Sequence      0.000673    90.55       NA      0.207398       5301.67    53.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB113101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10202-DPB113101        VFLLVTLAILTAL    2   LLVTLAILT     0.447        Sequence      0.001512    95.00       NA      0.264681       2852.61    30.81       \r\n",
      "   2  HLA-DPA10202-DPB113101        LLVTLAILTALRL    4   LAILTALRL     0.727        Sequence      0.014212    54.85       NA      0.313313       1685.47    15.88       \r\n",
      "   3  HLA-DPA10202-DPB113101        LVTLAILTALRLC    3   LAILTALRL     0.787        Sequence      0.009690    67.73       NA      0.266063       2810.27    30.29       \r\n",
      "   4  HLA-DPA10202-DPB113101       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.011067    63.50       NA      0.299986       1946.91    19.43       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11601        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.006093    95.00       NA      0.242695       3618.71    56.28       \r\n",
      "   2   HLA-DPA10103-DPB11601        LLVTLAILTALRL    4   LAILTALRL     0.533        Sequence      0.006255    94.94       NA      0.266601       2793.96    48.60       \r\n",
      "   3   HLA-DPA10103-DPB11601        LVTLAILTALRLC    3   LAILTALRL     0.593        Sequence      0.004244    95.00       NA      0.230569       4126.04    60.20       \r\n",
      "   4   HLA-DPA10103-DPB11601       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.004703    95.00       NA      0.268371       2740.96    48.04       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.227        Sequence      0.000589    88.00       NA      0.146569      10238.74    56.66       \r\n",
      "   2   HLA-DPA10201-DPB10301        LLVTLAILTALRL    4   LAILTALRL     0.560        Sequence      0.000725    84.78       NA      0.171924       7782.23    45.11       \r\n",
      "   3   HLA-DPA10201-DPB10301        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000485    90.48       NA      0.146057      10295.62    56.90       \r\n",
      "   4   HLA-DPA10201-DPB10301       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.000479    90.62       NA      0.168442       8081.02    46.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB12601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB12601        VFLLVTLAILTAL    1   FLLVTLAIL     0.213        Sequence      0.000741    88.45       NA      0.187612       6567.30    35.82       \r\n",
      "   2   HLA-DPA10201-DPB12601        LLVTLAILTALRL    4   LAILTALRL     0.753        Sequence      0.001454    74.65       NA      0.236074       3887.45    19.31       \r\n",
      "   3   HLA-DPA10201-DPB12601        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.000683    89.65       NA      0.192793       6209.28    33.78       \r\n",
      "   4   HLA-DPA10201-DPB12601       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.000646    90.40       NA      0.207734       5282.44    28.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10319\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10319        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10505-DQB10319        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10505-DQB10319        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10505-DQB10319       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1418\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1418        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.003546    86.59       NA      0.130501      12182.87    71.84       \r\n",
      "   2     DRB1_1418        LLVTLAILTALRL    4   LAILTALRL     0.800        Sequence      0.025657    55.21       NA      0.185190       6741.67    45.43       \r\n",
      "   3     DRB1_1418        LVTLAILTALRLC    3   LAILTALRL     0.933        Sequence      0.030264    51.73       NA      0.166574       8246.01    54.17       \r\n",
      "   4     DRB1_1418       LLVTLAILTALRLC    4   LAILTALRL     0.887        Sequence      0.026012    54.91       NA      0.189635       6425.11    43.42       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB14501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB14501        VFLLVTLAILTAL    3   LVTLAILTA     0.227        Sequence      0.000589    88.00       NA      0.146569      10238.74    56.66       \r\n",
      "   2   HLA-DPA10201-DPB14501        LLVTLAILTALRL    4   LAILTALRL     0.560        Sequence      0.000725    84.78       NA      0.171924       7782.23    45.11       \r\n",
      "   3   HLA-DPA10201-DPB14501        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000485    90.48       NA      0.146057      10295.62    56.90       \r\n",
      "   4   HLA-DPA10201-DPB14501       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.000479    90.62       NA      0.168442       8081.02    46.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.673        Sequence      0.001731    93.59       NA      0.138666      11152.77    73.90       \r\n",
      "   2   HLA-DQA10201-DQB10202        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.007000    74.18       NA      0.189910       6406.02    49.85       \r\n",
      "   3   HLA-DQA10201-DQB10202        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.003751    84.83       NA      0.150834       9777.00    68.34       \r\n",
      "   4   HLA-DQA10201-DQB10202       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.003106    87.50       NA      0.148207      10058.88    69.56       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10201-DQB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10201-DQB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.673        Sequence      0.001731    93.59       NA      0.138666      11152.77    73.90       \r\n",
      "   2   HLA-DQA10201-DQB10201        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.007000    74.18       NA      0.189910       6406.02    49.85       \r\n",
      "   3   HLA-DQA10201-DQB10201        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.003751    84.83       NA      0.150834       9777.00    68.34       \r\n",
      "   4   HLA-DQA10201-DQB10201       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.003106    87.50       NA      0.148207      10058.88    69.56       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.000249    95.00       NA      0.171047       7856.43    64.22       \r\n",
      "   2   HLA-DQA10101-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.487        Sequence      0.000543    89.78       NA      0.184255       6810.22    55.46       \r\n",
      "   3   HLA-DQA10101-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.613        Sequence      0.000317    95.00       NA      0.169707       7971.17    65.08       \r\n",
      "   4   HLA-DQA10101-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000309    95.00       NA      0.185913       6689.14    54.31       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10104-DPB11501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10104-DPB11501        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.006133    67.25       NA      0.325337       1479.86    53.08       \r\n",
      "   2   HLA-DPA10104-DPB11501        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.022245    25.90       NA      0.406207        616.90    33.32       \r\n",
      "   3   HLA-DPA10104-DPB11501        LVTLAILTALRLC    3   LAILTALRL     0.920        Sequence      0.013916    41.52       NA      0.345849       1185.32    48.00       \r\n",
      "   4   HLA-DPA10104-DPB11501       LLVTLAILTALRLC    4   LAILTALRL     0.840        Sequence      0.014053    41.19       NA      0.372779        885.71    41.33       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10901        VFLLVTLAILTAL    2   LLVTLAILT     0.547        Sequence      0.001111    95.00       NA      0.186996       6611.22    37.73       \r\n",
      "   2   HLA-DPA10201-DPB10901        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.006313    72.45       NA      0.209082       5205.95    29.55       \r\n",
      "   3   HLA-DPA10201-DPB10901        LVTLAILTALRLC    3   LAILTALRL     0.700        Sequence      0.002986    87.16       NA      0.168208       8101.50    45.43       \r\n",
      "   4   HLA-DPA10201-DPB10901       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.002487    89.57       NA      0.200105       5736.97    32.72       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10301-DPB135001 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB10301 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB18501 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.003740    69.95       NA      0.195437       6034.17    55.67       \r\n",
      "   2   HLA-DQA10501-DQB10602        LLVTLAILTALRL    4   LAILTALRL     0.400        Sequence      0.005044    60.78       NA      0.214093       4931.21    45.53       \r\n",
      "   3   HLA-DQA10501-DQB10602        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.002174    83.31       NA      0.196137       5988.64    55.30       \r\n",
      "   4   HLA-DQA10501-DQB10602       LLVTLAILTALRLC    4   LAILTALRL     0.407        Sequence      0.002581    79.53       NA      0.223064       4475.06    40.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB11701\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB11701        VFLLVTLAILTAL    3   LVTLAILTA     0.267        Sequence      0.010045    91.34       NA      0.266717       2790.45    32.00       \r\n",
      "   2   HLA-DPA10301-DPB11701        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.027804    68.22       NA      0.299042       1966.89    22.62       \r\n",
      "   3   HLA-DPA10301-DPB11701        LVTLAILTALRLC    3   LAILTALRL     0.687        Sequence      0.021023    76.93       NA      0.255638       3145.83    35.55       \r\n",
      "   4   HLA-DPA10301-DPB11701       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.018015    80.92       NA      0.293545       2087.42    24.10       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB12301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB12301        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000538    88.66       NA      0.233761       3985.97    38.43       \r\n",
      "   2   HLA-DPA10103-DPB12301        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.001728    64.59       NA      0.282016       2364.75    27.21       \r\n",
      "   3   HLA-DPA10103-DPB12301        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000766    83.00       NA      0.223330       4462.20    41.24       \r\n",
      "   4   HLA-DPA10103-DPB12301       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000700    84.53       NA      0.244981       3530.30    35.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB12101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB12101        VFLLVTLAILTAL    1   FLLVTLAIL     0.220        Sequence      0.002851    94.31       NA      0.289439       2182.25    24.11       \r\n",
      "   2   HLA-DPA10202-DPB12101        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.021527    44.58       NA      0.340973       1249.53    11.35       \r\n",
      "   3   HLA-DPA10202-DPB12101        LVTLAILTALRLC    3   LAILTALRL     0.767        Sequence      0.016959    54.10       NA      0.294691       2061.70    22.55       \r\n",
      "   4   HLA-DPA10202-DPB12101       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.018384    50.87       NA      0.321341       1545.25    15.58       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.587        Sequence      0.004603    95.00       NA      0.185680       6706.03    84.30       \r\n",
      "   2   HLA-DQA10505-DQB10402        LLVTLAILTALRL    4   LAILTALRL     0.447        Sequence      0.002910    95.00       NA      0.187738       6558.35    83.77       \r\n",
      "   3   HLA-DQA10505-DQB10402        LVTLAILTALRLC    3   LAILTALRL     0.660        Sequence      0.002186    95.00       NA      0.175624       7476.84    86.62       \r\n",
      "   4   HLA-DQA10505-DQB10402       LLVTLAILTALRLC    4   LAILTALRL     0.493        Sequence      0.002472    95.00       NA      0.212910       4994.73    77.01       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10502-DQB10319 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.380        Sequence      0.005645    95.00       NA      0.251605       3286.14    66.99       \r\n",
      "   2   HLA-DPA10401-DPB10501        LLVTLAILTALRL    4   LAILTALRL     0.473        Sequence      0.005234    95.00       NA      0.308924       1767.44    48.40       \r\n",
      "   3   HLA-DPA10401-DPB10501        LVTLAILTALRLC    3   LAILTALRL     0.593        Sequence      0.003906    95.00       NA      0.271756       2642.39    60.64       \r\n",
      "   4   HLA-DPA10401-DPB10501       LLVTLAILTALRLC    4   LAILTALRL     0.480        Sequence      0.003837    95.00       NA      0.299832       1950.15    51.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1304\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1304        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001375    93.77       NA      0.190099       6392.94    79.96       \r\n",
      "   2     DRB1_1304        LLVTLAILTALRL    4   LAILTALRL     0.787        Sequence      0.005139    79.84       NA      0.289539       2179.89    53.56       \r\n",
      "   3     DRB1_1304        LVTLAILTALRLC    3   LAILTALRL     0.920        Sequence      0.004789    80.88       NA      0.258385       3053.70    62.15       \r\n",
      "   4     DRB1_1304       LLVTLAILTALRLC    4   LAILTALRL     0.887        Sequence      0.004417    82.02       NA      0.283001       2339.68    55.34       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10401-DQB10319\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10401-DQB10319        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000757    95.00       NA      0.188403       6511.33    85.30       \r\n",
      "   2   HLA-DQA10401-DQB10319        LLVTLAILTALRL    4   LAILTALRL     0.480        Sequence      0.001590    94.84       NA      0.223745       4442.21    72.34       \r\n",
      "   3   HLA-DQA10401-DQB10319        LVTLAILTALRLC    3   LAILTALRL     0.507        Sequence      0.000830    95.00       NA      0.219408       4655.63    74.13       \r\n",
      "   4   HLA-DQA10401-DQB10319       LLVTLAILTALRLC    4   LAILTALRL     0.460        Sequence      0.000824    95.00       NA      0.225988       4335.70    71.42       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1301        VFLLVTLAILTAL    2   LLVTLAILT     0.320        Sequence      0.000339    95.00       NA      0.172065       7770.37    77.21       \r\n",
      "   2     DRB1_1301        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.001473    84.05       NA      0.278972       2443.93    48.77       \r\n",
      "   3     DRB1_1301        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.001370    84.93       NA      0.247780       3424.99    56.95       \r\n",
      "   4     DRB1_1301       LLVTLAILTALRLC    4   LAILTALRL     0.827        Sequence      0.001267    86.00       NA      0.273817       2584.12    50.03       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11701\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11701        VFLLVTLAILTAL    2   LLVTLAILT     0.447        Sequence      0.001512    95.00       NA      0.264681       2852.61    30.81       \r\n",
      "   2   HLA-DPA10202-DPB11701        LLVTLAILTALRL    4   LAILTALRL     0.727        Sequence      0.014212    54.85       NA      0.313313       1685.47    15.88       \r\n",
      "   3   HLA-DPA10202-DPB11701        LVTLAILTALRLC    3   LAILTALRL     0.787        Sequence      0.009690    67.73       NA      0.266063       2810.27    30.29       \r\n",
      "   4   HLA-DPA10202-DPB11701       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.011067    63.50       NA      0.299986       1946.91    19.43       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1302        VFLLVTLAILTAL    2   LLVTLAILT     0.540        Sequence      0.000510    95.00       NA      0.216453       4806.89    62.12       \r\n",
      "   2     DRB1_1302        LLVTLAILTALRL    4   LAILTALRL     0.813        Sequence      0.001943    88.50       NA      0.292924       2101.50    40.78       \r\n",
      "   3     DRB1_1302        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.001429    91.59       NA      0.238120       3802.34    55.89       \r\n",
      "   4     DRB1_1302       LLVTLAILTALRLC    4   LAILTALRL     0.840        Sequence      0.001255    92.70       NA      0.256708       3109.62    50.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10502        VFLLVTLAILTAL    3   LVTLAILTA     0.513        Sequence      0.001349    64.57       NA      0.161334       8727.03    63.47       \r\n",
      "   2   HLA-DQA10505-DQB10502        LLVTLAILTALRL    1   LVTLAILTA     0.140        Sequence      0.001357    64.35       NA      0.177012       7365.39    52.92       \r\n",
      "   3   HLA-DQA10505-DQB10502        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.000672    86.11       NA      0.163101       8561.76    62.29       \r\n",
      "   4   HLA-DQA10505-DQB10502       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.001030    74.24       NA      0.178780       7225.84    51.71       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB110001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10202-DPB110001        VFLLVTLAILTAL    1   FLLVTLAIL     0.280        Sequence      0.000450    87.58       NA      0.258521       3049.21    32.58       \r\n",
      "   2  HLA-DPA10202-DPB110001        LLVTLAILTALRL    4   LAILTALRL     0.787        Sequence      0.002766    42.19       NA      0.324342       1495.88    14.85       \r\n",
      "   3  HLA-DPA10202-DPB110001        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.002180    49.26       NA      0.282584       2350.26    25.14       \r\n",
      "   4  HLA-DPA10202-DPB110001       LLVTLAILTALRLC    4   LAILTALRL     0.787        Sequence      0.001788    55.29       NA      0.311034       1727.55    17.77       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10402-DQB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10402-DQB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.613        Sequence      0.002297    95.00       NA      0.194961       6065.33    64.40       \r\n",
      "   2   HLA-DQA10402-DQB10402        LLVTLAILTALRL    4   LAILTALRL     0.440        Sequence      0.001567    95.00       NA      0.200498       5712.63    61.93       \r\n",
      "   3   HLA-DQA10402-DQB10402        LVTLAILTALRLC    3   LAILTALRL     0.607        Sequence      0.001132    95.00       NA      0.189090       6463.11    67.01       \r\n",
      "   4   HLA-DQA10402-DQB10402       LLVTLAILTALRLC    4   LAILTALRL     0.460        Sequence      0.001536    95.00       NA      0.228476       4220.54    48.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10301        VFLLVTLAILTAL    1   FLLVTLAIL     0.120        Sequence      0.004526    74.61       NA      0.186242       6665.37    49.30       \r\n",
      "   2   HLA-DPA10103-DPB10301        LLVTLAILTALRL    3   TLAILTALR     0.293        Sequence      0.003365    80.65       NA      0.203123       5552.66    42.87       \r\n",
      "   3   HLA-DPA10103-DPB10301        LVTLAILTALRLC    2   TLAILTALR     0.293        Sequence      0.002383    86.46       NA      0.174597       7560.39    53.99       \r\n",
      "   4   HLA-DPA10103-DPB10301       LLVTLAILTALRLC    3   TLAILTALR     0.233        Sequence      0.002574    85.27       NA      0.198681       5826.05    44.49       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11301        VFLLVTLAILTAL    1   FLLVTLAIL     0.233        Sequence      0.003490    92.68       NA      0.234270       3964.08    35.07       \r\n",
      "   2   HLA-DPA10103-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.687        Sequence      0.006689    82.27       NA      0.266248       2804.65    24.87       \r\n",
      "   3   HLA-DPA10103-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.003279    93.34       NA      0.217884       4733.03    41.12       \r\n",
      "   4   HLA-DPA10103-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.003535    92.54       NA      0.249316       3368.54    29.99       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB10401 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB116201 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10106-DPB11101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10106-DPB11101        VFLLVTLAILTAL    2   LLVTLAILT     0.413        Sequence      0.001037    79.63       NA      0.223665       4446.06    36.34       \r\n",
      "   2   HLA-DPA10106-DPB11101        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.003397    45.82       NA      0.287298       2233.39    17.84       \r\n",
      "   3   HLA-DPA10106-DPB11101        LVTLAILTALRLC    3   LAILTALRL     0.867        Sequence      0.001838    64.61       NA      0.239825       3732.84    31.01       \r\n",
      "   4   HLA-DPA10106-DPB11101       LLVTLAILTALRLC    4   LAILTALRL     0.820        Sequence      0.001707    66.77       NA      0.260083       2998.11    24.96       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB14501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB14501        VFLLVTLAILTAL    3   LVTLAILTA     0.260        Sequence      0.000745    93.31       NA      0.203264       5544.20    52.73       \r\n",
      "   2   HLA-DPA10202-DPB14501        LLVTLAILTALRL    4   LAILTALRL     0.620        Sequence      0.001729    81.73       NA      0.239743       3736.15    37.97       \r\n",
      "   3   HLA-DPA10202-DPB14501        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.001118    88.84       NA      0.210374       5133.68    49.75       \r\n",
      "   4   HLA-DPA10202-DPB14501       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.001332    86.33       NA      0.234673       3946.83    39.95       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB13101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB13101        VFLLVTLAILTAL    3   LVTLAILTA     0.407        Sequence      0.004643    95.00       NA      0.246240       3482.54    61.14       \r\n",
      "   2   HLA-DPA10103-DPB13101        LLVTLAILTALRL    4   LAILTALRL     0.627        Sequence      0.007635    92.22       NA      0.260864       2972.88    56.48       \r\n",
      "   3   HLA-DPA10103-DPB13101        LVTLAILTALRLC    3   LAILTALRL     0.700        Sequence      0.005990    94.39       NA      0.231702       4075.77    65.65       \r\n",
      "   4   HLA-DPA10103-DPB13101       LLVTLAILTALRLC    4   LAILTALRL     0.567        Sequence      0.005816    94.62       NA      0.251869       3276.76    59.37       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0809\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0809        VFLLVTLAILTAL    3   LVTLAILTA     0.633        Sequence      0.000768    92.82       NA      0.185524       6717.35    69.32       \r\n",
      "   2     DRB1_0809        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.000931    91.18       NA      0.246695       3465.43    49.88       \r\n",
      "   3     DRB1_0809        LVTLAILTALRLC    3   LAILTALRL     0.793        Sequence      0.000725    93.25       NA      0.218113       4721.32    58.94       \r\n",
      "   4     DRB1_0809       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.000616    94.35       NA      0.235958       3892.34    53.27       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB13401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB13401        VFLLVTLAILTAL    3   LVTLAILTA     0.453        Sequence      0.001158    66.94       NA      0.214554       4906.67    57.97       \r\n",
      "   2   HLA-DPA10201-DPB13401        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.002796    36.11       NA      0.231511       4084.20    51.37       \r\n",
      "   3   HLA-DPA10201-DPB13401        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.002575    38.85       NA      0.195382       6037.76    65.29       \r\n",
      "   4   HLA-DPA10201-DPB13401       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.002508    39.78       NA      0.226905       4292.89    53.17       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10105-DPB11801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10105-DPB11801        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.006582    61.43       NA      0.217434       4756.13    61.19       \r\n",
      "   2   HLA-DPA10105-DPB11801        LLVTLAILTALRL    1   LVTLAILTA     0.173        Sequence      0.005819    64.71       NA      0.285422       2279.19    39.07       \r\n",
      "   3   HLA-DPA10105-DPB11801        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.001871    87.31       NA      0.249548       3360.09    50.59       \r\n",
      "   4   HLA-DPA10105-DPB11801       LLVTLAILTALRLC    1   LVTLAILTA     0.160        Sequence      0.003360    77.59       NA      0.282747       2346.12    39.88       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB13401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB13401        VFLLVTLAILTAL    3   LVTLAILTA     0.487        Sequence      0.002206    77.92       NA      0.226946       4290.99    63.63       \r\n",
      "   2   HLA-DPA10103-DPB13401        LLVTLAILTALRL    4   LAILTALRL     0.753        Sequence      0.003902    63.73       NA      0.255811       3139.94    55.10       \r\n",
      "   3   HLA-DPA10103-DPB13401        LVTLAILTALRLC    3   LAILTALRL     0.887        Sequence      0.002676    73.55       NA      0.215972       4831.97    66.89       \r\n",
      "   4   HLA-DPA10103-DPB13401       LLVTLAILTALRLC    4   LAILTALRL     0.733        Sequence      0.003439    67.17       NA      0.244917       3532.75    58.31       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB110501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10201-DPB110501        VFLLVTLAILTAL    1   FLLVTLAIL     0.287        Sequence      0.000455    69.76       NA      0.171374       7828.68    38.17       \r\n",
      "   2  HLA-DPA10201-DPB110501        LLVTLAILTALRL    4   LAILTALRL     0.767        Sequence      0.000479    68.33       NA      0.201256       5665.97    26.46       \r\n",
      "   3  HLA-DPA10201-DPB110501        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000319    79.32       NA      0.167221       8188.48    40.01       \r\n",
      "   4  HLA-DPA10201-DPB110501       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.000338    77.88       NA      0.180280       7109.51    34.36       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10509-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10509-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10509-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10509-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10509-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10401-DQB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10401-DQB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.613        Sequence      0.002297    95.00       NA      0.194961       6065.33    64.40       \r\n",
      "   2   HLA-DQA10401-DQB10402        LLVTLAILTALRL    4   LAILTALRL     0.440        Sequence      0.001567    95.00       NA      0.200498       5712.63    61.93       \r\n",
      "   3   HLA-DQA10401-DQB10402        LVTLAILTALRLC    3   LAILTALRL     0.607        Sequence      0.001132    95.00       NA      0.189090       6463.11    67.01       \r\n",
      "   4   HLA-DQA10401-DQB10402       LLVTLAILTALRLC    4   LAILTALRL     0.460        Sequence      0.001536    95.00       NA      0.228476       4220.54    48.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10302        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.000257    90.85       NA      0.129250      12348.90    86.43       \r\n",
      "   2   HLA-DQA10303-DQB10302        LLVTLAILTALRL    4   LAILTALRL     0.567        Sequence      0.000692    70.34       NA      0.157801       9067.09    76.24       \r\n",
      "   3   HLA-DQA10303-DQB10302        LVTLAILTALRLC    3   LAILTALRL     0.493        Sequence      0.000360    85.57       NA      0.143130      10626.90    81.82       \r\n",
      "   4   HLA-DQA10303-DQB10302       LLVTLAILTALRLC    4   LAILTALRL     0.487        Sequence      0.000363    85.41       NA      0.137816      11255.81    83.69       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10303        VFLLVTLAILTAL    4   VTLAILTAL     0.253        Sequence      0.000351    90.83       NA      0.132050      11980.39    90.30       \r\n",
      "   2   HLA-DQA10303-DQB10303        LLVTLAILTALRL    2   VTLAILTAL     0.180        Sequence      0.000479    85.98       NA      0.150053       9859.97    85.26       \r\n",
      "   3   HLA-DQA10303-DQB10303        LVTLAILTALRLC    1   VTLAILTAL     0.147        Sequence      0.000324    91.86       NA      0.142978      10644.39    87.29       \r\n",
      "   4   HLA-DQA10303-DQB10303       LLVTLAILTALRLC    4   LAILTALRL     0.333        Sequence      0.000297    92.88       NA      0.142647      10682.58    87.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10101        VFLLVTLAILTAL    3   LVTLAILTA     0.447        Sequence      0.001635    93.53       NA      0.238754       3776.35    55.05       \r\n",
      "   2   HLA-DPA10201-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.003970    81.86       NA      0.308834       1769.16    35.22       \r\n",
      "   3   HLA-DPA10201-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.720        Sequence      0.001198    95.00       NA      0.251967       3273.29    51.13       \r\n",
      "   4   HLA-DPA10201-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.673        Sequence      0.001823    92.56       NA      0.270553       2677.01    45.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10301        VFLLVTLAILTAL    4   VTLAILTAL     0.380        Sequence      0.000467    92.35       NA      0.138834      11132.51    93.30       \r\n",
      "   2   HLA-DQA10303-DQB10301        LLVTLAILTALRL    2   VTLAILTAL     0.360        Sequence      0.000733    84.09       NA      0.164948       8392.36    86.62       \r\n",
      "   3   HLA-DQA10303-DQB10301        LVTLAILTALRLC    1   VTLAILTAL     0.253        Sequence      0.000390    94.64       NA      0.157508       9095.88    88.87       \r\n",
      "   4   HLA-DQA10303-DQB10301       LLVTLAILTALRLC    2   VTLAILTAL     0.307        Sequence      0.000558    89.62       NA      0.162200       8645.64    87.45       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10304\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10304        VFLLVTLAILTAL    3   LVTLAILTA     0.313        Sequence      0.000286    95.00       NA      0.142769      10668.49    94.59       \r\n",
      "   2   HLA-DQA10303-DQB10304        LLVTLAILTALRL    4   LAILTALRL     0.513        Sequence      0.000648    91.00       NA      0.172357       7745.86    87.88       \r\n",
      "   3   HLA-DQA10303-DQB10304        LVTLAILTALRLC    3   LAILTALRL     0.500        Sequence      0.000348    95.00       NA      0.158292       9019.05    91.37       \r\n",
      "   4   HLA-DQA10303-DQB10304       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000379    95.00       NA      0.153343       9515.15    92.40       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB11301        VFLLVTLAILTAL    3   LVTLAILTA     0.293        Sequence      0.005365    92.17       NA      0.289500       2180.81    38.17       \r\n",
      "   2   HLA-DPA10401-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.011032    79.07       NA      0.322317       1529.02    27.39       \r\n",
      "   3   HLA-DPA10401-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.700        Sequence      0.006493    89.70       NA      0.278020       2469.23    42.26       \r\n",
      "   4   HLA-DPA10401-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.647        Sequence      0.006620    89.40       NA      0.305324       1837.65    32.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10508-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10508-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10508-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10508-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10508-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11001        VFLLVTLAILTAL    2   LLVTLAILT     0.600        Sequence      0.001306    95.00       NA      0.167885       8129.87    46.59       \r\n",
      "   2   HLA-DPA10103-DPB11001        LLVTLAILTALRL    4   LAILTALRL     0.527        Sequence      0.002036    92.90       NA      0.186898       6618.23    38.09       \r\n",
      "   3   HLA-DPA10103-DPB11001        LVTLAILTALRLC    3   LAILTALRL     0.553        Sequence      0.001092    95.00       NA      0.157199       9126.34    51.87       \r\n",
      "   4   HLA-DPA10103-DPB11001       LLVTLAILTALRLC    4   LAILTALRL     0.487        Sequence      0.000851    95.00       NA      0.187236       6594.07    37.94       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB13001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB13001        VFLLVTLAILTAL    2   LLVTLAILT     0.493        Sequence      0.001096    94.36       NA      0.197975       5870.72    35.22       \r\n",
      "   2   HLA-DPA10201-DPB13001        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.009229    45.72       NA      0.231813       4070.87    23.53       \r\n",
      "   3   HLA-DPA10201-DPB13001        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.006179    59.00       NA      0.190739       6348.82    38.11       \r\n",
      "   4   HLA-DPA10201-DPB13001       LLVTLAILTALRLC    4   LAILTALRL     0.740        Sequence      0.006644    56.67       NA      0.220930       4579.59    27.00       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10301-DPB158401 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0302\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0302        VFLLVTLAILTAL    3   LVTLAILTA     0.467        Sequence      0.000643    95.00       NA      0.069007      23697.81    59.76       \r\n",
      "   2     DRB1_0302        LLVTLAILTALRL    4   LAILTALRL     0.633        Sequence      0.000706    94.95       NA      0.068692      23778.71    60.17       \r\n",
      "   3     DRB1_0302        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000716    94.86       NA      0.062607      25396.95    68.04       \r\n",
      "   4     DRB1_0302       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.000606    95.00       NA      0.076277      21905.17    50.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10506-DQB10303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10506-DQB10303        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.001503    90.60       NA      0.181970       6980.69    83.28       \r\n",
      "   2   HLA-DQA10506-DQB10303        LLVTLAILTALRL    4   LAILTALRL     0.447        Sequence      0.002393    83.41       NA      0.220414       4605.23    71.63       \r\n",
      "   3   HLA-DQA10506-DQB10303        LVTLAILTALRLC    3   LAILTALRL     0.440        Sequence      0.001909    87.25       NA      0.213389       4968.91    73.90       \r\n",
      "   4   HLA-DQA10506-DQB10303       LLVTLAILTALRLC    4   LAILTALRL     0.420        Sequence      0.002137    85.38       NA      0.222672       4494.08    70.90       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10111-DPB10301 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11801        VFLLVTLAILTAL    3   LVTLAILTA     0.580        Sequence      0.006582    61.43       NA      0.217434       4756.13    61.19       \r\n",
      "   2   HLA-DPA10103-DPB11801        LLVTLAILTALRL    1   LVTLAILTA     0.173        Sequence      0.005819    64.71       NA      0.285422       2279.19    39.07       \r\n",
      "   3   HLA-DPA10103-DPB11801        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.001871    87.31       NA      0.249548       3360.09    50.59       \r\n",
      "   4   HLA-DPA10103-DPB11801       LLVTLAILTALRLC    1   LVTLAILTA     0.160        Sequence      0.003360    77.59       NA      0.282747       2346.12    39.88       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1114\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1114        VFLLVTLAILTAL    2   LLVTLAILT     0.540        Sequence      0.000510    95.00       NA      0.216453       4806.89    62.12       \r\n",
      "   2     DRB1_1114        LLVTLAILTALRL    4   LAILTALRL     0.813        Sequence      0.001943    88.50       NA      0.292924       2101.50    40.78       \r\n",
      "   3     DRB1_1114        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.001429    91.59       NA      0.238120       3802.34    55.89       \r\n",
      "   4     DRB1_1114       LLVTLAILTALRLC    4   LAILTALRL     0.840        Sequence      0.001255    92.70       NA      0.256708       3109.62    50.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11901        VFLLVTLAILTAL    1   FLLVTLAIL     0.267        Sequence      0.001686    95.00       NA      0.216851       4786.23    54.80       \r\n",
      "   2   HLA-DPA10201-DPB11901        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.004485    85.53       NA      0.238272       3796.09    47.97       \r\n",
      "   3   HLA-DPA10201-DPB11901        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.002101    94.63       NA      0.204984       5441.97    58.63       \r\n",
      "   4   HLA-DPA10201-DPB11901       LLVTLAILTALRLC    4   LAILTALRL     0.740        Sequence      0.002149    94.45       NA      0.229538       4172.32    50.68       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB16501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB16501        VFLLVTLAILTAL    3   LVTLAILTA     0.440        Sequence      0.001767    94.69       NA      0.273969       2579.87    51.64       \r\n",
      "   2   HLA-DPA10202-DPB16501        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.009344    66.89       NA      0.328721       1426.66    34.25       \r\n",
      "   3   HLA-DPA10202-DPB16501        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.005001    81.66       NA      0.286956       2241.67    47.46       \r\n",
      "   4   HLA-DPA10202-DPB16501       LLVTLAILTALRLC    4   LAILTALRL     0.693        Sequence      0.004224    84.68       NA      0.315636       1643.64    38.22       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1503\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1503        VFLLVTLAILTAL    3   LVTLAILTA     0.487        Sequence      0.000665    95.00       NA      0.193235       6179.66    59.84       \r\n",
      "   2     DRB1_1503        LLVTLAILTALRL    4   LAILTALRL     0.987        Sequence      0.022964    42.90       NA      0.337764       1293.68    21.13       \r\n",
      "   3     DRB1_1503        LVTLAILTALRLC    3   LAILTALRL     1.000        Sequence      0.023681    42.28       NA      0.313041       1690.44    26.27       \r\n",
      "   4     DRB1_1503       LLVTLAILTALRLC    4   LAILTALRL     0.987        Sequence      0.017455    48.67       NA      0.340498       1255.97    20.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1502        VFLLVTLAILTAL    1   FLLVTLAIL     0.333        Sequence      0.000385    95.00       NA      0.191271       6312.38    62.00       \r\n",
      "   2     DRB1_1502        LLVTLAILTALRL    4   LAILTALRL     0.947        Sequence      0.007289    55.69       NA      0.308609       1773.48    27.40       \r\n",
      "   3     DRB1_1502        LVTLAILTALRLC    3   LAILTALRL     0.973        Sequence      0.007035    56.47       NA      0.271988       2635.76    36.97       \r\n",
      "   4     DRB1_1502       LLVTLAILTALRLC    4   LAILTALRL     0.960        Sequence      0.005275    62.38       NA      0.297326       2003.75    30.17       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1501        VFLLVTLAILTAL    2   LLVTLAILT     0.453        Sequence      0.000261    95.00       NA      0.174714       7550.82    71.22       \r\n",
      "   2     DRB1_1501        LLVTLAILTALRL    4   LAILTALRL     0.953        Sequence      0.011256    42.19       NA      0.344666       1200.59    29.06       \r\n",
      "   3     DRB1_1501        LVTLAILTALRLC    3   LAILTALRL     0.993        Sequence      0.008947    46.38       NA      0.315296       1649.69    35.23       \r\n",
      "   4     DRB1_1501       LLVTLAILTALRLC    4   LAILTALRL     0.987        Sequence      0.007257    50.47       NA      0.345218       1193.44    28.95       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB13901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB13901        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000538    88.66       NA      0.233761       3985.97    38.43       \r\n",
      "   2   HLA-DPA10103-DPB13901        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.001728    64.59       NA      0.282016       2364.75    27.21       \r\n",
      "   3   HLA-DPA10103-DPB13901        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000766    83.00       NA      0.223330       4462.20    41.24       \r\n",
      "   4   HLA-DPA10103-DPB13901       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000700    84.53       NA      0.244981       3530.30    35.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB14801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB14801        VFLLVTLAILTAL    2   LLVTLAILT     0.313        Sequence      0.001408    90.19       NA      0.266507       2796.80    40.49       \r\n",
      "   2   HLA-DPA10103-DPB14801        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.001737    87.32       NA      0.271345       2654.17    39.24       \r\n",
      "   3   HLA-DPA10103-DPB14801        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.001255    91.58       NA      0.231716       4075.15    49.85       \r\n",
      "   4   HLA-DPA10103-DPB14801       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.001199    92.08       NA      0.261556       2950.71    41.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10105-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10105-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.727        Sequence      0.001003    93.15       NA      0.132519      11919.75    77.93       \r\n",
      "   2   HLA-DQA10105-DQB10602        LLVTLAILTALRL    2   VTLAILTAL     0.247        Sequence      0.001007    93.11       NA      0.154396       9407.36    67.75       \r\n",
      "   3   HLA-DQA10105-DQB10602        LVTLAILTALRLC    2   TLAILTALR     0.187        Sequence      0.000607    95.00       NA      0.149011       9971.76    70.37       \r\n",
      "   4   HLA-DQA10105-DQB10602       LLVTLAILTALRLC    2   VTLAILTAL     0.140        Sequence      0.000831    94.82       NA      0.152527       9599.54    68.67       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11801\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11801        VFLLVTLAILTAL    3   LVTLAILTA     0.600        Sequence      0.000748    87.56       NA      0.291442       2135.47    42.17       \r\n",
      "   2   HLA-DPA10202-DPB11801        LLVTLAILTALRL    4   LAILTALRL     0.767        Sequence      0.004994    36.42       NA      0.369378        918.91    18.68       \r\n",
      "   3   HLA-DPA10202-DPB11801        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.004254    41.41       NA      0.324859       1487.54    31.13       \r\n",
      "   4   HLA-DPA10202-DPB11801       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.003775    45.27       NA      0.353001       1097.06    22.93       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10104-DPB11001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10104-DPB11001        VFLLVTLAILTAL    2   LLVTLAILT     0.600        Sequence      0.001306    95.00       NA      0.167885       8129.87    46.59       \r\n",
      "   2   HLA-DPA10104-DPB11001        LLVTLAILTALRL    4   LAILTALRL     0.527        Sequence      0.002036    92.90       NA      0.186898       6618.23    38.09       \r\n",
      "   3   HLA-DPA10104-DPB11001        LVTLAILTALRLC    3   LAILTALRL     0.553        Sequence      0.001092    95.00       NA      0.157199       9126.34    51.87       \r\n",
      "   4   HLA-DPA10104-DPB11001       LLVTLAILTALRLC    4   LAILTALRL     0.487        Sequence      0.000851    95.00       NA      0.187236       6594.07    37.94       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10101        VFLLVTLAILTAL    3   LVTLAILTA     0.440        Sequence      0.003698    92.95       NA      0.327143       1451.23    50.51       \r\n",
      "   2   HLA-DPA10202-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.013773    69.26       NA      0.412897        573.82    26.35       \r\n",
      "   3   HLA-DPA10202-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.006371    86.32       NA      0.361948        995.83    40.43       \r\n",
      "   4   HLA-DPA10202-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.700        Sequence      0.005542    88.39       NA      0.375419        860.77    36.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10201        VFLLVTLAILTAL    1   FLLVTLAIL     0.280        Sequence      0.001012    91.63       NA      0.328853       1424.62    29.11       \r\n",
      "   2   HLA-DPA10202-DPB10201        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.004885    55.17       NA      0.357602       1043.78    21.48       \r\n",
      "   3   HLA-DPA10202-DPB10201        LVTLAILTALRLC    3   LAILTALRL     0.847        Sequence      0.003503    65.81       NA      0.319513       1576.12    31.76       \r\n",
      "   4   HLA-DPA10202-DPB10201       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.003258    68.02       NA      0.354688       1077.21    22.21       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0901        VFLLVTLAILTAL    2   LLVTLAILT     0.460        Sequence      0.001059    94.65       NA      0.211073       5095.00    71.05       \r\n",
      "   2     DRB1_0901        LLVTLAILTALRL    4   LAILTALRL     0.887        Sequence      0.005912    74.42       NA      0.266867       2785.93    55.88       \r\n",
      "   3     DRB1_0901        LVTLAILTALRLC    3   LAILTALRL     0.933        Sequence      0.002711    85.99       NA      0.227168       4280.70    66.76       \r\n",
      "   4     DRB1_0901       LLVTLAILTALRLC    4   LAILTALRL     0.887        Sequence      0.001578    91.70       NA      0.241449       3667.82    62.90       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10202        VFLLVTLAILTAL    2   LLVTLAILT     0.260        Sequence      0.002624    79.12       NA      0.338872       1278.26    29.63       \r\n",
      "   2   HLA-DPA10202-DPB10202        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.008623    43.85       NA      0.382499        797.29    18.91       \r\n",
      "   3   HLA-DPA10202-DPB10202        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.007814    47.06       NA      0.339397       1271.02    29.48       \r\n",
      "   4   HLA-DPA10202-DPB10202       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.007714    47.50       NA      0.369645        916.26    21.84       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB19101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB19101        VFLLVTLAILTAL    2   LLVTLAILT     0.580        Sequence      0.000699    90.99       NA      0.164401       8442.18    51.37       \r\n",
      "   2   HLA-DPA10201-DPB19101        LLVTLAILTALRL    4   LAILTALRL     0.580        Sequence      0.001570    78.41       NA      0.193594       6155.70    38.54       \r\n",
      "   3   HLA-DPA10201-DPB19101        LVTLAILTALRLC    3   LAILTALRL     0.713        Sequence      0.000958    87.01       NA      0.165231       8366.71    50.98       \r\n",
      "   4   HLA-DPA10201-DPB19101       LLVTLAILTALRLC    4   LAILTALRL     0.600        Sequence      0.000956    87.04       NA      0.189887       6407.62    40.12       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11601        VFLLVTLAILTAL    3   LVTLAILTA     0.400        Sequence      0.001529    95.00       NA      0.224803       4391.65    47.87       \r\n",
      "   2   HLA-DPA10201-DPB11601        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.005198    79.39       NA      0.238461       3788.34    43.38       \r\n",
      "   3   HLA-DPA10201-DPB11601        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.003636    85.89       NA      0.204007       5499.80    55.03       \r\n",
      "   4   HLA-DPA10201-DPB11601       LLVTLAILTALRLC    4   LAILTALRL     0.700        Sequence      0.004201    83.47       NA      0.233362       4003.22    45.02       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001974    89.58       NA      0.173620       7640.73    74.29       \r\n",
      "   2   HLA-DQA10505-DQB10202        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.007163    68.86       NA      0.238041       3805.59    51.46       \r\n",
      "   3   HLA-DQA10505-DQB10202        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.004627    77.22       NA      0.193753       6145.12    67.41       \r\n",
      "   4   HLA-DQA10505-DQB10202       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.003865    80.13       NA      0.187206       6596.21    69.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10505-DQB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10505-DQB10201        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.001974    89.58       NA      0.173620       7640.73    74.29       \r\n",
      "   2   HLA-DQA10505-DQB10201        LLVTLAILTALRL    4   LAILTALRL     0.833        Sequence      0.007163    68.86       NA      0.238041       3805.59    51.46       \r\n",
      "   3   HLA-DQA10505-DQB10201        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.004627    77.22       NA      0.193753       6145.12    67.41       \r\n",
      "   4   HLA-DQA10505-DQB10201       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.003865    80.13       NA      0.187206       6596.21    69.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10603\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10603        VFLLVTLAILTAL    3   LVTLAILTA     0.673        Sequence      0.000861    92.17       NA      0.106963      15716.44    80.48       \r\n",
      "   2   HLA-DQA10101-DQB10603        LLVTLAILTALRL    1   LVTLAILTA     0.220        Sequence      0.000766    93.35       NA      0.131032      12113.08    65.61       \r\n",
      "   3   HLA-DQA10101-DQB10603        LVTLAILTALRLC    4   AILTALRLC     0.193        Sequence      0.000247    95.00       NA      0.120111      13632.39    72.78       \r\n",
      "   4   HLA-DQA10101-DQB10603       LLVTLAILTALRLC    1   LVTLAILTA     0.207        Sequence      0.000461    95.00       NA      0.126094      12777.86    68.92       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.727        Sequence      0.001003    93.15       NA      0.132519      11919.75    77.93       \r\n",
      "   2   HLA-DQA10101-DQB10602        LLVTLAILTALRL    2   VTLAILTAL     0.247        Sequence      0.001007    93.11       NA      0.154396       9407.36    67.75       \r\n",
      "   3   HLA-DQA10101-DQB10602        LVTLAILTALRLC    2   TLAILTALR     0.187        Sequence      0.000607    95.00       NA      0.149011       9971.76    70.37       \r\n",
      "   4   HLA-DQA10101-DQB10602       LLVTLAILTALRLC    2   VTLAILTAL     0.140        Sequence      0.000831    94.82       NA      0.152527       9599.54    68.67       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10101-DQB10601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10101-DQB10601        VFLLVTLAILTAL    3   LVTLAILTA     0.467        Sequence      0.003192    95.00       NA      0.174118       7599.67    87.48       \r\n",
      "   2   HLA-DQA10101-DQB10601        LLVTLAILTALRL    2   VTLAILTAL     0.360        Sequence      0.003905    95.00       NA      0.189360       6444.26    82.89       \r\n",
      "   3   HLA-DQA10101-DQB10601        LVTLAILTALRLC    3   LAILTALRL     0.393        Sequence      0.001769    95.00       NA      0.174283       7586.11    87.43       \r\n",
      "   4   HLA-DQA10101-DQB10601       LLVTLAILTALRLC    2   VTLAILTAL     0.247        Sequence      0.002500    95.00       NA      0.191714       6282.20    82.11       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10502-DQB10603 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB110501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10202-DPB110501        VFLLVTLAILTAL    2   LLVTLAILT     0.247        Sequence      0.000521    84.19       NA      0.243022       3605.93    33.47       \r\n",
      "   2  HLA-DPA10202-DPB110501        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.002267    43.49       NA      0.301117       1923.23    16.16       \r\n",
      "   3  HLA-DPA10202-DPB110501        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.001922    48.73       NA      0.266656       2792.30    25.51       \r\n",
      "   4  HLA-DPA10202-DPB110501       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.001542    55.77       NA      0.285520       2276.77    20.06       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10601        VFLLVTLAILTAL    1   FLLVTLAIL     0.153        Sequence      0.001885    92.47       NA      0.216064       4827.16    43.28       \r\n",
      "   2   HLA-DPA10103-DPB10601        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.002652    88.14       NA      0.214526       4908.16    43.84       \r\n",
      "   3   HLA-DPA10103-DPB10601        LVTLAILTALRLC    2   TLAILTALR     0.287        Sequence      0.002103    91.24       NA      0.180523       7090.84    57.19       \r\n",
      "   4   HLA-DPA10103-DPB10601       LLVTLAILTALRLC    3   TLAILTALR     0.200        Sequence      0.002042    91.59       NA      0.210460       5128.91    45.37       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10207-DPB10101 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10104-DQB10602\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10104-DQB10602        VFLLVTLAILTAL    3   LVTLAILTA     0.727        Sequence      0.001003    93.15       NA      0.132519      11919.75    77.93       \r\n",
      "   2   HLA-DQA10104-DQB10602        LLVTLAILTALRL    2   VTLAILTAL     0.247        Sequence      0.001007    93.11       NA      0.154396       9407.36    67.75       \r\n",
      "   3   HLA-DQA10104-DQB10602        LVTLAILTALRLC    2   TLAILTALR     0.187        Sequence      0.000607    95.00       NA      0.149011       9971.76    70.37       \r\n",
      "   4   HLA-DQA10104-DQB10602       LLVTLAILTALRLC    2   VTLAILTAL     0.140        Sequence      0.000831    94.82       NA      0.152527       9599.54    68.67       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0813\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0813        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.001618    95.00       NA      0.257346       3088.22    61.98       \r\n",
      "   2     DRB1_0813        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.007359    79.50       NA      0.350121       1131.78    31.66       \r\n",
      "   3     DRB1_0813        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.007207    79.81       NA      0.311807       1713.16    43.83       \r\n",
      "   4     DRB1_0813       LLVTLAILTALRLC    4   LAILTALRL     0.720        Sequence      0.005774    83.17       NA      0.322294       1529.40    40.51       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB14901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB14901        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.003440    87.24       NA      0.307276       1799.24    33.68       \r\n",
      "   2   HLA-DPA10301-DPB14901        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.009492    68.02       NA      0.368539        927.29    21.57       \r\n",
      "   3   HLA-DPA10301-DPB14901        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.005468    79.91       NA      0.303836       1867.47    34.45       \r\n",
      "   4   HLA-DPA10301-DPB14901       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.004738    82.46       NA      0.332626       1367.64    28.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB15101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB15101        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000950    85.82       NA      0.185947       6686.68    45.75       \r\n",
      "   2   HLA-DPA10103-DPB15101        LLVTLAILTALRL    4   LAILTALRL     0.700        Sequence      0.000977    85.35       NA      0.216799       4788.92    35.03       \r\n",
      "   3   HLA-DPA10103-DPB15101        LVTLAILTALRLC    3   LAILTALRL     0.740        Sequence      0.000397    95.00       NA      0.183982       6830.37    46.48       \r\n",
      "   4   HLA-DPA10103-DPB15101       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000554    92.82       NA      0.203954       5502.96    39.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1305\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1305        VFLLVTLAILTAL    3   LVTLAILTA     0.753        Sequence      0.000312    92.99       NA      0.155653       9280.28    71.89       \r\n",
      "   2     DRB1_1305        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.000529    87.89       NA      0.239756       3735.63    46.80       \r\n",
      "   3     DRB1_1305        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000654    85.19       NA      0.210519       5125.63    54.99       \r\n",
      "   4     DRB1_1305       LLVTLAILTALRLC    4   LAILTALRL     0.767        Sequence      0.000541    87.63       NA      0.233903       3979.85    48.38       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1402        VFLLVTLAILTAL    2   LLVTLAILT     0.253        Sequence      0.003584    87.20       NA      0.196491       5965.75    70.46       \r\n",
      "   2     DRB1_1402        LLVTLAILTALRL    4   LAILTALRL     0.747        Sequence      0.015703    63.63       NA      0.273496       2593.11    44.49       \r\n",
      "   3     DRB1_1402        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.012631    67.82       NA      0.246399       3476.55    53.64       \r\n",
      "   4     DRB1_1402       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.011005    70.22       NA      0.266436       2798.95    46.81       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10201-DPB124601 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10501        VFLLVTLAILTAL    3   LVTLAILTA     0.387        Sequence      0.000918    95.00       NA      0.173562       7645.53    55.45       \r\n",
      "   2   HLA-DPA10201-DPB10501        LLVTLAILTALRL    4   LAILTALRL     0.607        Sequence      0.001372    91.76       NA      0.220058       4623.00    39.49       \r\n",
      "   3   HLA-DPA10201-DPB10501        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.000611    95.00       NA      0.184498       6792.34    51.53       \r\n",
      "   4   HLA-DPA10201-DPB10501       LLVTLAILTALRLC    4   LAILTALRL     0.647        Sequence      0.000571    95.00       NA      0.204086       5495.11    44.72       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10614\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10614        VFLLVTLAILTAL    3   LVTLAILTA     0.800        Sequence      0.003562    88.55       NA      0.167505       8163.36    64.17       \r\n",
      "   2   HLA-DQA10102-DQB10614        LLVTLAILTALRL    1   LVTLAILTA     0.327        Sequence      0.002143    93.70       NA      0.183531       6863.78    56.36       \r\n",
      "   3   HLA-DQA10102-DQB10614        LVTLAILTALRLC    3   LAILTALRL     0.380        Sequence      0.001016    95.00       NA      0.172472       7736.23    61.77       \r\n",
      "   4   HLA-DQA10102-DQB10614       LLVTLAILTALRLC    1   LVTLAILTA     0.333        Sequence      0.001842    94.82       NA      0.190140       6390.10    53.13       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10611\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10611        VFLLVTLAILTAL    3   LVTLAILTA     0.760        Sequence      0.003284    91.30       NA      0.230581       4125.50    67.79       \r\n",
      "   2   HLA-DQA10102-DQB10611        LLVTLAILTALRL    1   LVTLAILTA     0.253        Sequence      0.002661    93.27       NA      0.275436       2539.24    51.88       \r\n",
      "   3   HLA-DQA10102-DQB10611        LVTLAILTALRLC    2   TLAILTALR     0.213        Sequence      0.001250    95.00       NA      0.262879       2908.77    56.39       \r\n",
      "   4   HLA-DQA10102-DQB10611       LLVTLAILTALRLC    1   LVTLAILTA     0.267        Sequence      0.002206    94.71       NA      0.275230       2544.91    51.95       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10102-DQB10610\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10102-DQB10610        VFLLVTLAILTAL    3   LVTLAILTA     0.707        Sequence      0.003464    92.83       NA      0.183975       6830.89    72.42       \r\n",
      "   2   HLA-DQA10102-DQB10610        LLVTLAILTALRL    1   LVTLAILTA     0.247        Sequence      0.002778    94.51       NA      0.200733       5698.12    65.96       \r\n",
      "   3   HLA-DQA10102-DQB10610        LVTLAILTALRLC    1   VTLAILTAL     0.153        Sequence      0.001219    95.00       NA      0.182682       6927.12    72.91       \r\n",
      "   4   HLA-DQA10102-DQB10610       LLVTLAILTALRLC    1   LVTLAILTA     0.227        Sequence      0.001971    95.00       NA      0.191002       6330.78    69.77       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10202        VFLLVTLAILTAL    3   LVTLAILTA     0.473        Sequence      0.000267    95.00       NA      0.126197      12763.63    82.01       \r\n",
      "   2   HLA-DQA10302-DQB10202        LLVTLAILTALRL    4   LAILTALRL     0.620        Sequence      0.000440    91.94       NA      0.150158       9848.77    69.83       \r\n",
      "   3   HLA-DQA10302-DQB10202        LVTLAILTALRLC    3   LAILTALRL     0.660        Sequence      0.000264    95.00       NA      0.127979      12519.89    81.23       \r\n",
      "   4   HLA-DQA10302-DQB10202       LLVTLAILTALRLC    4   LAILTALRL     0.553        Sequence      0.000252    95.00       NA      0.124177      13045.66    82.89       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB11501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB11501        VFLLVTLAILTAL    3   LVTLAILTA     0.493        Sequence      0.006133    67.25       NA      0.325337       1479.86    53.08       \r\n",
      "   2   HLA-DPA10103-DPB11501        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.022245    25.90       NA      0.406207        616.90    33.32       \r\n",
      "   3   HLA-DPA10103-DPB11501        LVTLAILTALRLC    3   LAILTALRL     0.920        Sequence      0.013916    41.52       NA      0.345849       1185.32    48.00       \r\n",
      "   4   HLA-DPA10103-DPB11501       LLVTLAILTALRLC    4   LAILTALRL     0.840        Sequence      0.014053    41.19       NA      0.372779        885.71    41.33       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11101        VFLLVTLAILTAL    2   LLVTLAILT     0.413        Sequence      0.001037    79.63       NA      0.223665       4446.06    36.34       \r\n",
      "   2   HLA-DPA10201-DPB11101        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.003397    45.82       NA      0.287298       2233.39    17.84       \r\n",
      "   3   HLA-DPA10201-DPB11101        LVTLAILTALRLC    3   LAILTALRL     0.867        Sequence      0.001838    64.61       NA      0.239825       3732.84    31.01       \r\n",
      "   4   HLA-DPA10201-DPB11101       LLVTLAILTALRLC    4   LAILTALRL     0.820        Sequence      0.001707    66.77       NA      0.260083       2998.11    24.96       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB13101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB13101        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.003044    94.01       NA      0.301149       1922.56    49.37       \r\n",
      "   2   HLA-DPA10202-DPB13101        LLVTLAILTALRL    4   LAILTALRL     0.720        Sequence      0.010007    78.00       NA      0.328259       1433.81    40.33       \r\n",
      "   3   HLA-DPA10202-DPB13101        LVTLAILTALRLC    3   LAILTALRL     0.773        Sequence      0.006077    86.79       NA      0.294158       2073.63    51.77       \r\n",
      "   4   HLA-DPA10202-DPB13101       LLVTLAILTALRLC    4   LAILTALRL     0.700        Sequence      0.005653    87.77       NA      0.321753       1538.38    42.45       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10901        VFLLVTLAILTAL    2   LLVTLAILT     0.473        Sequence      0.002611    95.00       NA      0.265219       2836.05    30.30       \r\n",
      "   2   HLA-DPA10202-DPB10901        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.015653    67.45       NA      0.296425       2023.38    20.92       \r\n",
      "   3   HLA-DPA10202-DPB10901        LVTLAILTALRLC    3   LAILTALRL     0.707        Sequence      0.009249    81.41       NA      0.250559       3323.54    35.25       \r\n",
      "   4   HLA-DPA10202-DPB10901       LLVTLAILTALRLC    4   LAILTALRL     0.660        Sequence      0.008825    82.39       NA      0.289412       2182.89    22.84       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10901\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10901        VFLLVTLAILTAL    3   LVTLAILTA     0.240        Sequence      0.006606    92.13       NA      0.208091       5262.07    39.56       \r\n",
      "   2   HLA-DPA10103-DPB10901        LLVTLAILTALRL    4   LAILTALRL     0.560        Sequence      0.009892    86.66       NA      0.226411       4315.90    32.49       \r\n",
      "   3   HLA-DPA10103-DPB10901        LVTLAILTALRLC    3   LAILTALRL     0.567        Sequence      0.005850    93.32       NA      0.185950       6686.46    49.04       \r\n",
      "   4   HLA-DPA10103-DPB10901       LLVTLAILTALRLC    4   LAILTALRL     0.540        Sequence      0.005143    94.44       NA      0.219553       4648.33    35.06       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB113101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10201-DPB113101        VFLLVTLAILTAL    3   LVTLAILTA     0.200        Sequence      0.000755    95.00       NA      0.188696       6490.72    37.57       \r\n",
      "   2  HLA-DPA10201-DPB113101        LLVTLAILTALRL    4   LAILTALRL     0.673        Sequence      0.004175    65.22       NA      0.209120       5203.81    29.46       \r\n",
      "   3  HLA-DPA10201-DPB113101        LVTLAILTALRLC    3   LAILTALRL     0.747        Sequence      0.001963    83.18       NA      0.170009       7945.16    45.89       \r\n",
      "   4  HLA-DPA10201-DPB113101       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.002659    76.84       NA      0.199285       5788.10    33.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB18001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB18001        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.003440    87.24       NA      0.307276       1799.24    33.68       \r\n",
      "   2   HLA-DPA10301-DPB18001        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.009492    68.02       NA      0.368539        927.29    21.57       \r\n",
      "   3   HLA-DPA10301-DPB18001        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.005468    79.91       NA      0.303836       1867.47    34.45       \r\n",
      "   4   HLA-DPA10301-DPB18001       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.004738    82.46       NA      0.332626       1367.64    28.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.003440    87.24       NA      0.307276       1799.24    33.68       \r\n",
      "   2   HLA-DPA10301-DPB10402        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.009492    68.02       NA      0.368539        927.29    21.57       \r\n",
      "   3   HLA-DPA10301-DPB10402        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.005468    79.91       NA      0.303836       1867.47    34.45       \r\n",
      "   4   HLA-DPA10301-DPB10402       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.004738    82.46       NA      0.332626       1367.64    28.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0101        VFLLVTLAILTAL    2   LLVTLAILT     0.647        Sequence      0.000421    91.86       NA      0.275935       2525.57    65.69       \r\n",
      "   2     DRB1_0101        LLVTLAILTALRL    4   LAILTALRL     0.993        Sequence      0.069134    13.31       NA      0.495122        235.73    22.08       \r\n",
      "   3     DRB1_0101        LVTLAILTALRLC    3   LAILTALRL     1.000        Sequence      0.122252     9.40       NA      0.485306        262.14    23.61       \r\n",
      "   4     DRB1_0101       LLVTLAILTALRLC    4   LAILTALRL     1.000        Sequence      0.085906    11.72       NA      0.522086        176.08    18.15       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0102\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0102        VFLLVTLAILTAL    2   LLVTLAILT     0.633        Sequence      0.001108    83.82       NA      0.233229       4008.98    65.90       \r\n",
      "   2     DRB1_0102        LLVTLAILTALRL    4   LAILTALRL     1.000        Sequence      0.117278    13.58       NA      0.388826        744.54    22.95       \r\n",
      "   3     DRB1_0102        LVTLAILTALRLC    3   LAILTALRL     1.000        Sequence      0.194369     9.50       NA      0.386895        760.26    23.35       \r\n",
      "   4     DRB1_0102       LLVTLAILTALRLC    4   LAILTALRL     1.000        Sequence      0.144936    11.75       NA      0.422263        518.52    16.64       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0103\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0103        VFLLVTLAILTAL    2   LLVTLAILT     0.653        Sequence      0.000919    95.00       NA      0.156521       9193.53    66.03       \r\n",
      "   2     DRB1_0103        LLVTLAILTALRL    4   LAILTALRL     0.973        Sequence      0.017136    42.50       NA      0.198855       5815.09    45.66       \r\n",
      "   3     DRB1_0103        LVTLAILTALRLC    3   LAILTALRL     0.993        Sequence      0.025462    33.80       NA      0.192564       6224.69    48.47       \r\n",
      "   4     DRB1_0103       LLVTLAILTALRLC    4   LAILTALRL     0.993        Sequence      0.018990    40.16       NA      0.222608       4497.19    35.84       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1454\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1454        VFLLVTLAILTAL    3   LVTLAILTA     0.347        Sequence      0.000857    94.18       NA      0.197446       5904.42    67.89       \r\n",
      "   2     DRB1_1454        LLVTLAILTALRL    4   LAILTALRL     0.927        Sequence      0.011933    55.20       NA      0.305204       1840.03    30.76       \r\n",
      "   3     DRB1_1454        LVTLAILTALRLC    3   LAILTALRL     0.940        Sequence      0.008102    62.95       NA      0.262874       2908.93    44.40       \r\n",
      "   4     DRB1_1454       LLVTLAILTALRLC    4   LAILTALRL     0.940        Sequence      0.006363    67.65       NA      0.278915       2445.44    38.94       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10202-DPB154701 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB17501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB17501        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.004323    86.50       NA      0.301607       1913.06    34.98       \r\n",
      "   2   HLA-DPA10301-DPB17501        LLVTLAILTALRL    4   LAILTALRL     0.740        Sequence      0.014775    59.04       NA      0.361331       1002.50    21.44       \r\n",
      "   3   HLA-DPA10301-DPB17501        LVTLAILTALRLC    3   LAILTALRL     0.780        Sequence      0.010704    68.34       NA      0.308105       1783.17    33.29       \r\n",
      "   4   HLA-DPA10301-DPB17501       LLVTLAILTALRLC    4   LAILTALRL     0.680        Sequence      0.010009    70.06       NA      0.336492       1311.61    26.59       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10401-DPB10402        VFLLVTLAILTAL    2   LLVTLAILT     0.353        Sequence      0.001072    91.78       NA      0.228190       4233.62    54.00       \r\n",
      "   2   HLA-DPA10401-DPB10402        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.001227    90.23       NA      0.275519       2536.97    37.60       \r\n",
      "   3   HLA-DPA10401-DPB10402        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000718    95.00       NA      0.241902       3649.89    49.10       \r\n",
      "   4   HLA-DPA10401-DPB10402       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000688    95.00       NA      0.266381       2800.62    40.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB146301 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10201\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10201        VFLLVTLAILTAL    1   FLLVTLAIL     0.307        Sequence      0.000555    90.35       NA      0.241806       3653.68    36.13       \r\n",
      "   2   HLA-DPA10201-DPB10201        LLVTLAILTALRL    4   LAILTALRL     0.713        Sequence      0.001064    77.74       NA      0.253345       3224.85    32.58       \r\n",
      "   3   HLA-DPA10201-DPB10201        LVTLAILTALRLC    3   LAILTALRL     0.773        Sequence      0.000663    87.58       NA      0.218447       4704.29    44.11       \r\n",
      "   4   HLA-DPA10201-DPB10201       LLVTLAILTALRLC    4   LAILTALRL     0.700        Sequence      0.000572    89.92       NA      0.239975       3726.79    36.73       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB10202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB10202        VFLLVTLAILTAL    2   LLVTLAILT     0.333        Sequence      0.000786    87.21       NA      0.250127       3339.11    36.01       \r\n",
      "   2   HLA-DPA10201-DPB10202        LLVTLAILTALRL    4   LAILTALRL     0.753        Sequence      0.002425    58.66       NA      0.277068       2494.80    28.27       \r\n",
      "   3   HLA-DPA10201-DPB10202        LVTLAILTALRLC    3   LAILTALRL     0.813        Sequence      0.001266    77.32       NA      0.238642       3780.93    39.71       \r\n",
      "   4   HLA-DPA10201-DPB10202       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.001407    74.58       NA      0.264638       2853.93    31.70       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB18501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB18501        VFLLVTLAILTAL    2   LLVTLAILT     0.413        Sequence      0.001486    94.07       NA      0.292902       2102.00    24.38       \r\n",
      "   2   HLA-DPA10202-DPB18501        LLVTLAILTALRL    4   LAILTALRL     0.780        Sequence      0.009618    56.55       NA      0.363304        981.33     7.86       \r\n",
      "   3   HLA-DPA10202-DPB18501        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.006272    69.58       NA      0.314576       1662.60    18.05       \r\n",
      "   4   HLA-DPA10202-DPB18501       LLVTLAILTALRLC    4   LAILTALRL     0.773        Sequence      0.005284    74.15       NA      0.342152       1233.70    11.72       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0408\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0408        VFLLVTLAILTAL    3   LVTLAILTA     0.807        Sequence      0.000809    83.73       NA      0.205396       5417.77    71.11       \r\n",
      "   2     DRB1_0408        LLVTLAILTALRL    4   LAILTALRL     0.827        Sequence      0.001556    74.29       NA      0.307760       1789.84    41.40       \r\n",
      "   3     DRB1_0408        LVTLAILTALRLC    3   LAILTALRL     0.960        Sequence      0.001151    78.87       NA      0.266074       2809.93    53.41       \r\n",
      "   4     DRB1_0408       LLVTLAILTALRLC    4   LAILTALRL     0.847        Sequence      0.000922    82.02       NA      0.286832       2244.68    47.33       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0401        VFLLVTLAILTAL    3   LVTLAILTA     0.767        Sequence      0.000417    90.99       NA      0.162666       8602.16    80.90       \r\n",
      "   2     DRB1_0401        LLVTLAILTALRL    4   LAILTALRL     0.740        Sequence      0.000456    90.11       NA      0.219269       4662.64    63.93       \r\n",
      "   3     DRB1_0401        LVTLAILTALRLC    3   LAILTALRL     0.920        Sequence      0.000286    93.94       NA      0.185582       6713.14    74.37       \r\n",
      "   4     DRB1_0401       LLVTLAILTALRLC    4   LAILTALRL     0.807        Sequence      0.000237    95.00       NA      0.202343       5599.72    69.21       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB10101        VFLLVTLAILTAL    1   FLLVTLAIL     0.213        Sequence      0.005744    95.00       NA      0.281147       2387.09    57.34       \r\n",
      "   2   HLA-DPA10103-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.010692    92.98       NA      0.340886       1250.71    39.61       \r\n",
      "   3   HLA-DPA10103-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.006002    95.00       NA      0.298946       1968.94    52.04       \r\n",
      "   4   HLA-DPA10103-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.005666    95.00       NA      0.323137       1515.51    44.91       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10103-DQB10614\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10103-DQB10614        VFLLVTLAILTAL    3   LVTLAILTA     0.840        Sequence      0.004476    90.04       NA      0.148233      10056.05    67.74       \r\n",
      "   2   HLA-DQA10103-DQB10614        LLVTLAILTALRL    1   LVTLAILTA     0.353        Sequence      0.002015    95.00       NA      0.153573       9491.51    65.25       \r\n",
      "   3   HLA-DQA10103-DQB10614        LVTLAILTALRLC    3   LAILTALRL     0.473        Sequence      0.000776    95.00       NA      0.133974      11733.57    74.28       \r\n",
      "   4   HLA-DQA10103-DQB10614       LLVTLAILTALRLC    1   LVTLAILTA     0.347        Sequence      0.001550    95.00       NA      0.166119       8286.70    59.15       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0402        VFLLVTLAILTAL    3   LVTLAILTA     0.840        Sequence      0.000655    93.77       NA      0.139793      11017.60    76.13       \r\n",
      "   2     DRB1_0402        LLVTLAILTALRL    4   LAILTALRL     0.633        Sequence      0.001455    86.31       NA      0.179984       7132.32    58.42       \r\n",
      "   3     DRB1_0402        LVTLAILTALRLC    3   LAILTALRL     0.873        Sequence      0.000774    92.55       NA      0.158403       9008.22    68.13       \r\n",
      "   4     DRB1_0402       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.000717    93.13       NA      0.183839       6840.94    56.68       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0405\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0405        VFLLVTLAILTAL    3   LVTLAILTA     0.700        Sequence      0.001307    82.61       NA      0.193843       6139.14    72.37       \r\n",
      "   2     DRB1_0405        LLVTLAILTALRL    4   LAILTALRL     0.893        Sequence      0.002008    75.89       NA      0.285299       2282.23    45.61       \r\n",
      "   3     DRB1_0405        LVTLAILTALRLC    3   LAILTALRL     0.947        Sequence      0.001086    84.99       NA      0.244903       3533.28    57.48       \r\n",
      "   4     DRB1_0405       LLVTLAILTALRLC    4   LAILTALRL     0.893        Sequence      0.000977    86.43       NA      0.257123       3095.68    53.82       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0404\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0404        VFLLVTLAILTAL    3   LVTLAILTA     0.887        Sequence      0.002658    68.35       NA      0.233472       3998.45    61.92       \r\n",
      "   2     DRB1_0404        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.003792    62.69       NA      0.348609       1150.45    31.41       \r\n",
      "   3     DRB1_0404        LVTLAILTALRLC    3   LAILTALRL     0.967        Sequence      0.003189    65.35       NA      0.303599       1872.27    42.58       \r\n",
      "   4     DRB1_0404       LLVTLAILTALRLC    4   LAILTALRL     0.847        Sequence      0.002599    68.69       NA      0.326209       1465.97    36.83       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0407\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0407        VFLLVTLAILTAL    3   LVTLAILTA     0.727        Sequence      0.000591    91.33       NA      0.132890      11872.00    78.46       \r\n",
      "   2     DRB1_0407        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.001080    84.99       NA      0.183486       6867.12    55.77       \r\n",
      "   3     DRB1_0407        LVTLAILTALRLC    3   LAILTALRL     0.893        Sequence      0.000632    90.71       NA      0.159000       8950.22    67.05       \r\n",
      "   4     DRB1_0407       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000480    93.04       NA      0.175237       7508.21    59.57       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_0406\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_0406        VFLLVTLAILTAL    3   LVTLAILTA     0.867        Sequence      0.001867    82.27       NA      0.145262      10384.56    73.23       \r\n",
      "   2     DRB1_0406        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.002993    75.61       NA      0.193876       6136.95    49.71       \r\n",
      "   3     DRB1_0406        LVTLAILTALRLC    3   LAILTALRL     0.947        Sequence      0.001512    84.72       NA      0.166872       8219.46    62.86       \r\n",
      "   4     DRB1_0406       LLVTLAILTALRLC    4   LAILTALRL     0.827        Sequence      0.001276    86.72       NA      0.189954       6402.98    51.63       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10501-DQB10319\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10501-DQB10319        VFLLVTLAILTAL    3   LVTLAILTA     0.427        Sequence      0.000699    95.00       NA      0.145602      10346.43    91.25       \r\n",
      "   2   HLA-DQA10501-DQB10319        LLVTLAILTALRL    4   LAILTALRL     0.507        Sequence      0.001976    89.27       NA      0.177336       7339.62    82.10       \r\n",
      "   3   HLA-DQA10501-DQB10319        LVTLAILTALRLC    3   LAILTALRL     0.580        Sequence      0.001003    95.00       NA      0.167866       8131.54    85.19       \r\n",
      "   4   HLA-DQA10501-DQB10319       LLVTLAILTALRLC    4   LAILTALRL     0.473        Sequence      0.000806    95.00       NA      0.179419       7176.05    81.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB12001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB12001        VFLLVTLAILTAL    1   FLLVTLAIL     0.127        Sequence      0.005282    80.42       NA      0.207426       5300.07    45.88       \r\n",
      "   2   HLA-DPA10103-DPB12001        LLVTLAILTALRL    3   TLAILTALR     0.300        Sequence      0.006662    75.77       NA      0.225725       4348.05    39.16       \r\n",
      "   3   HLA-DPA10103-DPB12001        LVTLAILTALRLC    2   TLAILTALR     0.273        Sequence      0.004269    84.16       NA      0.192778       6210.29    51.57       \r\n",
      "   4   HLA-DPA10103-DPB12001       LLVTLAILTALRLC    3   TLAILTALR     0.220        Sequence      0.004479    83.39       NA      0.219095       4671.42    41.52       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10104-DQB10502\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10104-DQB10502        VFLLVTLAILTAL    3   LVTLAILTA     0.553        Sequence      0.000189    95.00       NA      0.084717      19993.43    77.11       \r\n",
      "   2   HLA-DQA10104-DQB10502        LLVTLAILTALRL    4   LAILTALRL     0.587        Sequence      0.000287    93.57       NA      0.096132      17670.49    69.27       \r\n",
      "   3   HLA-DQA10104-DQB10502        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000219    95.00       NA      0.086433      19625.64    75.97       \r\n",
      "   4   HLA-DQA10104-DQB10502       LLVTLAILTALRLC    4   LAILTALRL     0.607        Sequence      0.000234    95.00       NA      0.093966      18089.50    70.78       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10110-DQB10603 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1202\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1202        VFLLVTLAILTAL    3   LVTLAILTA     0.867        Sequence      0.005653    79.74       NA      0.201528       5649.32    54.29       \r\n",
      "   2     DRB1_1202        LLVTLAILTALRL    4   LAILTALRL     0.987        Sequence      0.059339    29.03       NA      0.302644       1891.71    22.31       \r\n",
      "   3     DRB1_1202        LVTLAILTALRLC    3   LAILTALRL     1.000        Sequence      0.070180    25.83       NA      0.287914       2218.56    26.02       \r\n",
      "   4     DRB1_1202       LLVTLAILTALRLC    4   LAILTALRL     0.993        Sequence      0.061468    28.38       NA      0.315536       1645.42    19.40       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10402        VFLLVTLAILTAL    2   LLVTLAILT     0.247        Sequence      0.000521    84.19       NA      0.243022       3605.93    33.47       \r\n",
      "   2   HLA-DPA10202-DPB10402        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.002267    43.49       NA      0.301117       1923.23    16.16       \r\n",
      "   3   HLA-DPA10202-DPB10402        LVTLAILTALRLC    3   LAILTALRL     0.840        Sequence      0.001922    48.73       NA      0.266656       2792.30    25.51       \r\n",
      "   4   HLA-DPA10202-DPB10402       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.001542    55.77       NA      0.285520       2276.77    20.06       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB10401        VFLLVTLAILTAL    1   FLLVTLAIL     0.320        Sequence      0.000571    83.74       NA      0.293815       2081.34    30.45       \r\n",
      "   2   HLA-DPA10202-DPB10401        LLVTLAILTALRL    4   LAILTALRL     0.820        Sequence      0.001674    53.42       NA      0.353817       1087.41    15.46       \r\n",
      "   3   HLA-DPA10202-DPB10401        LVTLAILTALRLC    3   LAILTALRL     0.880        Sequence      0.001556    55.79       NA      0.302839       1887.72    27.84       \r\n",
      "   4   HLA-DPA10202-DPB10401       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.001199    64.09       NA      0.326848       1455.86    21.44       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11301        VFLLVTLAILTAL    2   LLVTLAILT     0.467        Sequence      0.001880    87.67       NA      0.229916       4155.29    32.63       \r\n",
      "   2   HLA-DPA10201-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.800        Sequence      0.006203    56.95       NA      0.259081       3030.79    23.59       \r\n",
      "   3   HLA-DPA10201-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.800        Sequence      0.003160    77.03       NA      0.205589       5406.47    41.51       \r\n",
      "   4   HLA-DPA10201-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.003031    78.08       NA      0.236816       3856.37    30.34       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB135001 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10302-DQB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10302-DQB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.000224    95.00       NA      0.099618      17016.41    91.50       \r\n",
      "   2   HLA-DQA10302-DQB10402        LLVTLAILTALRL    4   LAILTALRL     0.487        Sequence      0.000229    95.00       NA      0.093328      18214.81    93.24       \r\n",
      "   3   HLA-DQA10302-DQB10402        LVTLAILTALRLC    3   LAILTALRL     0.653        Sequence      0.000137    95.00       NA      0.083665      20222.31    95.00       \r\n",
      "   4   HLA-DQA10302-DQB10402       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000165    95.00       NA      0.104638      16116.81    90.11       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB110501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10301-DPB110501        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.003440    87.24       NA      0.307276       1799.24    33.68       \r\n",
      "   2  HLA-DPA10301-DPB110501        LLVTLAILTALRL    4   LAILTALRL     0.760        Sequence      0.009492    68.02       NA      0.368539        927.29    21.57       \r\n",
      "   3  HLA-DPA10301-DPB110501        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.005468    79.91       NA      0.303836       1867.47    34.45       \r\n",
      "   4  HLA-DPA10301-DPB110501       LLVTLAILTALRLC    4   LAILTALRL     0.747        Sequence      0.004738    82.46       NA      0.332626       1367.64    28.26       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB110501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10401-DPB110501        VFLLVTLAILTAL    2   LLVTLAILT     0.353        Sequence      0.001072    91.78       NA      0.228190       4233.62    54.00       \r\n",
      "   2  HLA-DPA10401-DPB110501        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.001227    90.23       NA      0.275519       2536.97    37.60       \r\n",
      "   3  HLA-DPA10401-DPB110501        LVTLAILTALRLC    3   LAILTALRL     0.693        Sequence      0.000718    95.00       NA      0.241902       3649.89    49.10       \r\n",
      "   4  HLA-DPA10401-DPB110501       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000688    95.00       NA      0.266381       2800.62    40.62       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10401-DPB113301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10401-DPB113301        VFLLVTLAILTAL    3   LVTLAILTA     0.293        Sequence      0.005365    92.17       NA      0.289500       2180.81    38.17       \r\n",
      "   2  HLA-DPA10401-DPB113301        LLVTLAILTALRL    4   LAILTALRL     0.647        Sequence      0.011032    79.07       NA      0.322317       1529.02    27.39       \r\n",
      "   3  HLA-DPA10401-DPB113301        LVTLAILTALRLC    3   LAILTALRL     0.700        Sequence      0.006493    89.70       NA      0.278020       2469.23    42.26       \r\n",
      "   4  HLA-DPA10401-DPB113301       LLVTLAILTALRLC    4   LAILTALRL     0.647        Sequence      0.006620    89.40       NA      0.305324       1837.65    32.79       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB14001\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB14001        VFLLVTLAILTAL    3   LVTLAILTA     0.507        Sequence      0.001514    79.08       NA      0.212868       4997.00    56.24       \r\n",
      "   2   HLA-DPA10103-DPB14001        LLVTLAILTALRL    4   LAILTALRL     0.773        Sequence      0.003798    56.40       NA      0.264160       2868.73    41.17       \r\n",
      "   3   HLA-DPA10103-DPB14001        LVTLAILTALRLC    3   LAILTALRL     0.880        Sequence      0.001596    78.03       NA      0.217419       4756.91    54.85       \r\n",
      "   4   HLA-DPA10103-DPB14001       LLVTLAILTALRLC    4   LAILTALRL     0.787        Sequence      0.001751    76.05       NA      0.243201       3598.95    47.13       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DQA10102-DQB10620 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele HLA-DPA10202-DPB141401 - refer to the list of available molecules\r\n",
      "ERROR: Could not find allele DRB1_04201 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB112401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10103-DPB112401        VFLLVTLAILTAL    1   FLLVTLAIL     0.120        Sequence      0.004526    74.61       NA      0.186242       6665.37    49.30       \r\n",
      "   2  HLA-DPA10103-DPB112401        LLVTLAILTALRL    3   TLAILTALR     0.293        Sequence      0.003365    80.65       NA      0.203123       5552.66    42.87       \r\n",
      "   3  HLA-DPA10103-DPB112401        LVTLAILTALRLC    2   TLAILTALR     0.293        Sequence      0.002383    86.46       NA      0.174597       7560.39    53.99       \r\n",
      "   4  HLA-DPA10103-DPB112401       LLVTLAILTALRLC    3   TLAILTALR     0.233        Sequence      0.002574    85.27       NA      0.198681       5826.05    44.49       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: DRB1_1303\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1     DRB1_1303        VFLLVTLAILTAL    3   LVTLAILTA     0.460        Sequence      0.000215    95.00       NA      0.141072      10866.18    85.18       \r\n",
      "   2     DRB1_1303        LLVTLAILTALRL    4   LAILTALRL     0.740        Sequence      0.000327    95.00       NA      0.176941       7371.05    73.63       \r\n",
      "   3     DRB1_1303        LVTLAILTALRLC    3   LAILTALRL     0.827        Sequence      0.000320    95.00       NA      0.150475       9815.05    82.28       \r\n",
      "   4     DRB1_1303       LLVTLAILTALRLC    4   LAILTALRL     0.800        Sequence      0.000261    95.00       NA      0.165750       8319.85    77.41       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10401        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.000224    95.00       NA      0.099618      17016.41    91.50       \r\n",
      "   2   HLA-DQA10303-DQB10401        LLVTLAILTALRL    4   LAILTALRL     0.487        Sequence      0.000229    95.00       NA      0.093328      18214.81    93.24       \r\n",
      "   3   HLA-DQA10303-DQB10401        LVTLAILTALRLC    3   LAILTALRL     0.653        Sequence      0.000137    95.00       NA      0.083665      20222.31    95.00       \r\n",
      "   4   HLA-DQA10303-DQB10401       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000165    95.00       NA      0.104638      16116.81    90.11       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10303-DQB10402\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10303-DQB10402        VFLLVTLAILTAL    3   LVTLAILTA     0.527        Sequence      0.000224    95.00       NA      0.099618      17016.41    91.50       \r\n",
      "   2   HLA-DQA10303-DQB10402        LLVTLAILTALRL    4   LAILTALRL     0.487        Sequence      0.000229    95.00       NA      0.093328      18214.81    93.24       \r\n",
      "   3   HLA-DQA10303-DQB10402        LVTLAILTALRLC    3   LAILTALRL     0.653        Sequence      0.000137    95.00       NA      0.083665      20222.31    95.00       \r\n",
      "   4   HLA-DQA10303-DQB10402       LLVTLAILTALRLC    4   LAILTALRL     0.500        Sequence      0.000165    95.00       NA      0.104638      16116.81    90.11       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10105-DPB10101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10105-DPB10101        VFLLVTLAILTAL    1   FLLVTLAIL     0.213        Sequence      0.005744    95.00       NA      0.281147       2387.09    57.34       \r\n",
      "   2   HLA-DPA10105-DPB10101        LLVTLAILTALRL    4   LAILTALRL     0.693        Sequence      0.010692    92.98       NA      0.340886       1250.71    39.61       \r\n",
      "   3   HLA-DPA10105-DPB10101        LVTLAILTALRLC    3   LAILTALRL     0.733        Sequence      0.006002    95.00       NA      0.298946       1968.94    52.04       \r\n",
      "   4   HLA-DPA10105-DPB10101       LLVTLAILTALRLC    4   LAILTALRL     0.627        Sequence      0.005666    95.00       NA      0.323137       1515.51    44.91       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB11401\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10201-DPB11401        VFLLVTLAILTAL    2   LLVTLAILT     0.693        Sequence      0.000735    91.76       NA      0.167737       8142.90    45.02       \r\n",
      "   2   HLA-DPA10201-DPB11401        LLVTLAILTALRL    4   LAILTALRL     0.613        Sequence      0.001153    86.62       NA      0.193699       6148.71    34.98       \r\n",
      "   3   HLA-DPA10201-DPB11401        LVTLAILTALRLC    3   LAILTALRL     0.727        Sequence      0.000709    92.09       NA      0.156131       9232.41    49.97       \r\n",
      "   4   HLA-DPA10201-DPB11401       LLVTLAILTALRLC    4   LAILTALRL     0.653        Sequence      0.000603    93.43       NA      0.178397       7255.84    40.68       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "ERROR: Could not find allele HLA-DPA10103-DPB142201 - refer to the list of available molecules\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10201-DPB110601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10201-DPB110601        VFLLVTLAILTAL    1   FLLVTLAIL     0.267        Sequence      0.001686    95.00       NA      0.216851       4786.23    54.80       \r\n",
      "   2  HLA-DPA10201-DPB110601        LLVTLAILTALRL    4   LAILTALRL     0.707        Sequence      0.004485    85.53       NA      0.238272       3796.09    47.97       \r\n",
      "   3  HLA-DPA10201-DPB110601        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.002101    94.63       NA      0.204984       5441.97    58.63       \r\n",
      "   4  HLA-DPA10201-DPB110601       LLVTLAILTALRLC    4   LAILTALRL     0.740        Sequence      0.002149    94.45       NA      0.229538       4172.32    50.68       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DQA10401-DQB10301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DQA10401-DQB10301        VFLLVTLAILTAL    3   LVTLAILTA     0.373        Sequence      0.000757    95.00       NA      0.188403       6511.33    85.30       \r\n",
      "   2   HLA-DQA10401-DQB10301        LLVTLAILTALRL    4   LAILTALRL     0.480        Sequence      0.001590    94.84       NA      0.223745       4442.21    72.34       \r\n",
      "   3   HLA-DQA10401-DQB10301        LVTLAILTALRLC    3   LAILTALRL     0.507        Sequence      0.000830    95.00       NA      0.219408       4655.63    74.13       \r\n",
      "   4   HLA-DQA10401-DQB10301       LLVTLAILTALRLC    4   LAILTALRL     0.460        Sequence      0.000824    95.00       NA      0.225988       4335.70    71.42       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB112601\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1  HLA-DPA10103-DPB112601        VFLLVTLAILTAL    3   LVTLAILTA     0.333        Sequence      0.000538    88.66       NA      0.233761       3985.97    38.43       \r\n",
      "   2  HLA-DPA10103-DPB112601        LLVTLAILTALRL    4   LAILTALRL     0.860        Sequence      0.001728    64.59       NA      0.282016       2364.75    27.21       \r\n",
      "   3  HLA-DPA10103-DPB112601        LVTLAILTALRLC    3   LAILTALRL     0.853        Sequence      0.000766    83.00       NA      0.223330       4462.20    41.24       \r\n",
      "   4  HLA-DPA10103-DPB112601       LLVTLAILTALRLC    4   LAILTALRL     0.780        Sequence      0.000700    84.53       NA      0.244981       3530.30    35.55       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10301-DPB16501\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10301-DPB16501        VFLLVTLAILTAL    3   LVTLAILTA     0.367        Sequence      0.019034    92.52       NA      0.333000       1362.11    46.85       \r\n",
      "   2   HLA-DPA10301-DPB16501        LLVTLAILTALRL    4   LAILTALRL     0.733        Sequence      0.045082    75.65       NA      0.376435        851.36    34.76       \r\n",
      "   3   HLA-DPA10301-DPB16501        LVTLAILTALRLC    3   LAILTALRL     0.753        Sequence      0.029170    86.29       NA      0.328046       1437.12    48.24       \r\n",
      "   4   HLA-DPA10301-DPB16501       LLVTLAILTALRLC    4   LAILTALRL     0.667        Sequence      0.025762    88.52       NA      0.360179       1015.08    39.19       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10202-DPB11301\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10202-DPB11301        VFLLVTLAILTAL    2   LLVTLAILT     0.467        Sequence      0.003320    90.93       NA      0.318137       1599.76    24.39       \r\n",
      "   2   HLA-DPA10202-DPB11301        LLVTLAILTALRL    4   LAILTALRL     0.793        Sequence      0.026265    29.15       NA      0.367894        933.79    12.21       \r\n",
      "   3   HLA-DPA10202-DPB11301        LVTLAILTALRLC    3   LAILTALRL     0.820        Sequence      0.017063    46.10       NA      0.303852       1867.15    28.73       \r\n",
      "   4   HLA-DPA10202-DPB11301       LLVTLAILTALRLC    4   LAILTALRL     0.753        Sequence      0.017878    44.23       NA      0.339985       1262.96    18.43       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "# NetMHCIIpan version 4.1\r\n",
      "\r\n",
      "# Input is in PEPTIDE format\r\n",
      "\r\n",
      "# Prediction Mode: EL+BA\r\n",
      "\r\n",
      "# Threshold for Strong binding peptides (%Rank)\t1%\r\n",
      "# Threshold for Weak binding peptides (%Rank)\t5%\r\n",
      "\r\n",
      "# Allele: HLA-DPA10103-DPB14101\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      " Pos                     MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "   1   HLA-DPA10103-DPB14101        VFLLVTLAILTAL    3   LVTLAILTA     0.367        Sequence      0.001458    92.10       NA      0.233253       4007.94    55.87       \r\n",
      "   2   HLA-DPA10103-DPB14101        LLVTLAILTALRL    4   LAILTALRL     0.660        Sequence      0.001770    89.61       NA      0.223720       4443.41    59.39       \r\n",
      "   3   HLA-DPA10103-DPB14101        LVTLAILTALRLC    3   LAILTALRL     0.807        Sequence      0.001533    91.50       NA      0.199212       5792.67    68.28       \r\n",
      "   4   HLA-DPA10103-DPB14101       LLVTLAILTALRLC    4   LAILTALRL     0.693        Sequence      0.001529    91.53       NA      0.232213       4053.29    56.25       \r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n",
      "Number of strong binders: 0 Number of weak binders: 0\r\n",
      "--------------------------------------------------------------------------------------------------------------------------------------------\r\n"
     ]
    }
   ],
   "source": [
    "# for MHC class II\n",
    "hla2_alleles = pd.read_csv('test/MHC2_allele_marry.txt', names=['allele'])\n",
    "for allele in hla2_alleles['allele']:\n",
    "    outfile = allele.replace(':', '') + '_preds.xls' \n",
    "    ! ~/netMHCIIpan-4.1/netMHCIIpan -inptype 1 -f {peptidesII_file} -a {allele} -BA -xls -xlsfile test/{outfile}"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:09:27.824326Z",
     "start_time": "2023-10-11T15:07:46.590952Z"
    }
   },
   "id": "a94238c598af9c0c"
  },
  {
   "cell_type": "code",
   "execution_count": 367,
   "outputs": [
    {
     "data": {
      "text/plain": "Locus               DRB1                                                    \\\nAllele         DRB1_0101 DRB1_0102 DRB1_0103 DRB1_0301 DRB1_0302 DRB1_0401   \nPeptide                                                                      \nLLVTLAILTALRL   0.495122  0.388826  0.198855  0.168935  0.068692  0.219269   \nLLVTLAILTALRLC  0.522086  0.422263  0.222608  0.149349  0.076277  0.202343   \nLVTLAILTALRLC   0.485306  0.386895  0.192564  0.145798  0.062607  0.185582   \nVFLLVTLAILTAL   0.275935  0.233229  0.156521  0.100431  0.069007  0.162666   \n\nLocus                                                   ...  \\\nAllele         DRB1_0402 DRB1_0403 DRB1_0404 DRB1_0405  ...   \nPeptide                                                 ...   \nLLVTLAILTALRL   0.179984  0.193876  0.348609  0.285299  ...   \nLLVTLAILTALRLC  0.183839  0.189954  0.326209  0.257123  ...   \nLVTLAILTALRLC   0.158403  0.166872  0.303599  0.244903  ...   \nVFLLVTLAILTAL   0.139793  0.145262  0.233472  0.193843  ...   \n\nLocus                         HLA-DQ                        \\\nAllele         HLA-DQA10505-DQB10302 HLA-DQA10505-DQB10309   \nPeptide                                                      \nLLVTLAILTALRL               0.233057              0.177336   \nLLVTLAILTALRLC              0.208564              0.179419   \nLVTLAILTALRLC               0.215915              0.167866   \nVFLLVTLAILTAL               0.179439              0.145602   \n\nLocus                                                       \\\nAllele         HLA-DQA10505-DQB10319 HLA-DQA10505-DQB10402   \nPeptide                                                      \nLLVTLAILTALRL               0.177336              0.187738   \nLLVTLAILTALRLC              0.179419              0.212910   \nLVTLAILTALRLC               0.167866              0.175624   \nVFLLVTLAILTAL               0.145602              0.185680   \n\nLocus                                                       \\\nAllele         HLA-DQA10505-DQB10501 HLA-DQA10505-DQB10502   \nPeptide                                                      \nLLVTLAILTALRL               0.289216              0.177012   \nLLVTLAILTALRLC              0.287084              0.178780   \nLVTLAILTALRLC               0.266871              0.163101   \nVFLLVTLAILTAL               0.267939              0.161334   \n\nLocus                                                       \\\nAllele         HLA-DQA10506-DQB10303 HLA-DQA10508-DQB10301   \nPeptide                                                      \nLLVTLAILTALRL               0.220414              0.177336   \nLLVTLAILTALRLC              0.222672              0.179419   \nLVTLAILTALRLC               0.213389              0.167866   \nVFLLVTLAILTAL               0.181970              0.145602   \n\nLocus                                                       \nAllele         HLA-DQA10509-DQB10301 HLA-DQA10601-DQB10301  \nPeptide                                                     \nLLVTLAILTALRL               0.177336              0.187643  \nLLVTLAILTALRLC              0.179419              0.189232  \nLVTLAILTALRLC               0.167866              0.175945  \nVFLLVTLAILTAL               0.145602              0.154648  \n\n[4 rows x 280 columns]",
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead tr th {\n        text-align: left;\n    }\n\n    .dataframe thead tr:last-of-type th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr>\n      <th>Locus</th>\n      <th colspan=\"10\" halign=\"left\">DRB1</th>\n      <th>...</th>\n      <th colspan=\"10\" halign=\"left\">HLA-DQ</th>\n    </tr>\n    <tr>\n      <th>Allele</th>\n      <th>DRB1_0101</th>\n      <th>DRB1_0102</th>\n      <th>DRB1_0103</th>\n      <th>DRB1_0301</th>\n      <th>DRB1_0302</th>\n      <th>DRB1_0401</th>\n      <th>DRB1_0402</th>\n      <th>DRB1_0403</th>\n      <th>DRB1_0404</th>\n      <th>DRB1_0405</th>\n      <th>...</th>\n      <th>HLA-DQA10505-DQB10302</th>\n      <th>HLA-DQA10505-DQB10309</th>\n      <th>HLA-DQA10505-DQB10319</th>\n      <th>HLA-DQA10505-DQB10402</th>\n      <th>HLA-DQA10505-DQB10501</th>\n      <th>HLA-DQA10505-DQB10502</th>\n      <th>HLA-DQA10506-DQB10303</th>\n      <th>HLA-DQA10508-DQB10301</th>\n      <th>HLA-DQA10509-DQB10301</th>\n      <th>HLA-DQA10601-DQB10301</th>\n    </tr>\n    <tr>\n      <th>Peptide</th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n      <th></th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>LLVTLAILTALRL</th>\n      <td>0.495122</td>\n      <td>0.388826</td>\n      <td>0.198855</td>\n      <td>0.168935</td>\n      <td>0.068692</td>\n      <td>0.219269</td>\n      <td>0.179984</td>\n      <td>0.193876</td>\n      <td>0.348609</td>\n      <td>0.285299</td>\n      <td>...</td>\n      <td>0.233057</td>\n      <td>0.177336</td>\n      <td>0.177336</td>\n      <td>0.187738</td>\n      <td>0.289216</td>\n      <td>0.177012</td>\n      <td>0.220414</td>\n      <td>0.177336</td>\n      <td>0.177336</td>\n      <td>0.187643</td>\n    </tr>\n    <tr>\n      <th>LLVTLAILTALRLC</th>\n      <td>0.522086</td>\n      <td>0.422263</td>\n      <td>0.222608</td>\n      <td>0.149349</td>\n      <td>0.076277</td>\n      <td>0.202343</td>\n      <td>0.183839</td>\n      <td>0.189954</td>\n      <td>0.326209</td>\n      <td>0.257123</td>\n      <td>...</td>\n      <td>0.208564</td>\n      <td>0.179419</td>\n      <td>0.179419</td>\n      <td>0.212910</td>\n      <td>0.287084</td>\n      <td>0.178780</td>\n      <td>0.222672</td>\n      <td>0.179419</td>\n      <td>0.179419</td>\n      <td>0.189232</td>\n    </tr>\n    <tr>\n      <th>LVTLAILTALRLC</th>\n      <td>0.485306</td>\n      <td>0.386895</td>\n      <td>0.192564</td>\n      <td>0.145798</td>\n      <td>0.062607</td>\n      <td>0.185582</td>\n      <td>0.158403</td>\n      <td>0.166872</td>\n      <td>0.303599</td>\n      <td>0.244903</td>\n      <td>...</td>\n      <td>0.215915</td>\n      <td>0.167866</td>\n      <td>0.167866</td>\n      <td>0.175624</td>\n      <td>0.266871</td>\n      <td>0.163101</td>\n      <td>0.213389</td>\n      <td>0.167866</td>\n      <td>0.167866</td>\n      <td>0.175945</td>\n    </tr>\n    <tr>\n      <th>VFLLVTLAILTAL</th>\n      <td>0.275935</td>\n      <td>0.233229</td>\n      <td>0.156521</td>\n      <td>0.100431</td>\n      <td>0.069007</td>\n      <td>0.162666</td>\n      <td>0.139793</td>\n      <td>0.145262</td>\n      <td>0.233472</td>\n      <td>0.193843</td>\n      <td>...</td>\n      <td>0.179439</td>\n      <td>0.145602</td>\n      <td>0.145602</td>\n      <td>0.185680</td>\n      <td>0.267939</td>\n      <td>0.161334</td>\n      <td>0.181970</td>\n      <td>0.145602</td>\n      <td>0.145602</td>\n      <td>0.154648</td>\n    </tr>\n  </tbody>\n</table>\n<p>4 rows × 280 columns</p>\n</div>"
     },
     "execution_count": 367,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfs = []\n",
    "for allele in hla2_alleles['allele']:\n",
    "    try:\n",
    "        df = pd.read_csv(\n",
    "            'test/' + allele.replace(':', '') + '_preds.xls',\n",
    "            delimiter='\\t',\n",
    "            skiprows=[0],\n",
    "        )\n",
    "    except:\n",
    "        continue\n",
    "    df['Allele'] = allele\n",
    "    df = df.drop(columns=['Pos', 'ID', 'Ave', 'NB'])\n",
    "    dfs.append(df)\n",
    "\n",
    "netmhcII_data = pd.concat(dfs)\n",
    "netmhcII_data['sequence_length'] = [len(x) for x in netmhcII_data['Peptide'].values]\n",
    "netmhcII_data['Locus'] = [x[:4] if x[:3] == 'DRB' else x[:6] for x in netmhcII_data['Allele'].values]\n",
    "\n",
    "data_pivot = netmhcII_data.pivot_table(\n",
    "    index='Peptide',\n",
    "    columns=['Locus', 'Allele'],\n",
    "    values='Score_BA',\n",
    ")\n",
    "\n",
    "data_pivot"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:22:22.749128Z",
     "start_time": "2023-10-11T15:22:22.377377Z"
    }
   },
   "id": "59f61ef5bfc4d196"
  },
  {
   "cell_type": "code",
   "execution_count": 368,
   "outputs": [],
   "source": [
    "data_pivot.to_pickle('test/netmhcIIpan_pred.pkl.gz', protocol=2)"
   ],
   "metadata": {
    "collapsed": false,
    "ExecuteTime": {
     "end_time": "2023-10-11T15:22:29.046017Z",
     "start_time": "2023-10-11T15:22:29.007959Z"
    }
   },
   "id": "2113d9a440a4b05"
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
